<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>4.9.1 DNA dervived data | The OBIS manual</title>
  <meta name="description" content="4.9.1 DNA dervived data | The OBIS manual" />
  <meta name="generator" content="bookdown 0.34 and GitBook 2.6.7" />

  <meta property="og:title" content="4.9.1 DNA dervived data | The OBIS manual" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="4.9.1 DNA dervived data | The OBIS manual" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="examples.html"/>
<link rel="next" href="other_data_types.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-JBW0JFMNBJ"></script>
<script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'G-JBW0JFMNBJ');
</script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="/">The OBIS manual</a></li>
<li><a href="https://obis.org/">Return to OBIS.org</a></li>

<li class="divider"></li>
<li class="part"><span><b>Overview</b></span></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#guidelines-on-the-sharing-and-use-of-data-in-obis"><i class="fa fa-check"></i><b>1.1</b> Guidelines on the sharing and use of data in OBIS</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i><b>1.2</b> Acknowledgements</a></li>
<li class="chapter" data-level="1.3" data-path="policy.html"><a href="policy.html"><i class="fa fa-check"></i><b>1.3</b> Data Policy</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="policy.html"><a href="policy.html#guidelines-on-the-sharing-and-use-of-data-in-obis-1"><i class="fa fa-check"></i><b>1.3.1</b> Guidelines on the sharing and use of data in OBIS</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="gethelp.html"><a href="gethelp.html"><i class="fa fa-check"></i><b>1.4</b> Getting Help in OBIS</a></li>
<li class="chapter" data-level="1.5" data-path="FAQ.html"><a href="FAQ.html"><i class="fa fa-check"></i><b>1.5</b> Frequency Asked Questions</a></li>
</ul></li>
<li class="part"><span><b>Contributing data to OBIS</b></span></li>
<li class="chapter" data-level="2" data-path="contribute.html"><a href="contribute.html"><i class="fa fa-check"></i><b>2</b> What can you contribute and how?</a>
<ul>
<li class="chapter" data-level="2.1" data-path="contribute.html"><a href="contribute.html#why-publish-data-to-obis"><i class="fa fa-check"></i><b>2.1</b> Why publish data to OBIS</a></li>
<li class="chapter" data-level="2.2" data-path="contribute.html"><a href="contribute.html#how-to-handle-sensitive-data"><i class="fa fa-check"></i><b>2.2</b> How to handle sensitive data</a></li>
<li class="chapter" data-level="2.3" data-path="data_standards.html"><a href="data_standards.html"><i class="fa fa-check"></i><b>2.3</b> OBIS Data Life Cycle</a></li>
<li class="chapter" data-level="2.4" data-path="data_standards.html"><a href="data_standards.html#biodiversity-data-standards"><i class="fa fa-check"></i><b>2.4</b> Biodiversity data standards</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="darwin_core.html"><a href="darwin_core.html"><i class="fa fa-check"></i><b>2.4.1</b> Darwin Core</a></li>
<li class="chapter" data-level="2.4.2" data-path="data_format.html"><a href="data_format.html"><i class="fa fa-check"></i><b>2.4.2</b> Darwin Core Archive</a></li>
<li class="chapter" data-level="2.4.3" data-path="relational_db.html"><a href="relational_db.html"><i class="fa fa-check"></i><b>2.4.3</b> Relational databases: the underlying framework of OBIS</a></li>
<li class="chapter" data-level="2.4.4" data-path="eml.html"><a href="eml.html"><i class="fa fa-check"></i><b>2.4.4</b> Ecological Metadata Language</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="nodes.html"><a href="nodes.html"><i class="fa fa-check"></i><b>2.5</b> OBIS nodes</a>
<ul>
<li class="chapter" data-level="2.5.1" data-path="nodes.html"><a href="nodes.html#terms-of-reference-of-obis-nodes"><i class="fa fa-check"></i><b>2.5.1</b> Terms of Reference of OBIS nodes</a></li>
<li class="chapter" data-level="2.5.2" data-path="nodes.html"><a href="nodes.html#how-to-become-an-obis-node"><i class="fa fa-check"></i><b>2.5.2</b> How to become an OBIS node</a></li>
<li class="chapter" data-level="2.5.3" data-path="nodes.html"><a href="nodes.html#obis-node-health-status-check-and-transition-strategy"><i class="fa fa-check"></i><b>2.5.3</b> OBIS Node Health Status Check and Transition Strategy</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Data Formatting</b></span></li>
<li class="chapter" data-level="3" data-path="formatting.html"><a href="formatting.html"><i class="fa fa-check"></i><b>3</b> Dataset structure</a>
<ul>
<li class="chapter" data-level="3.1" data-path="formatting.html"><a href="formatting.html#when-to-use-event-core"><i class="fa fa-check"></i><b>3.1</b> When to use Event Core</a></li>
<li class="chapter" data-level="3.2" data-path="formatting.html"><a href="formatting.html#when-to-use-occurrence-core"><i class="fa fa-check"></i><b>3.2</b> When to use Occurrence Core</a></li>
<li class="chapter" data-level="3.3" data-path="formatting.html"><a href="formatting.html#extensions-in-obis"><i class="fa fa-check"></i><b>3.3</b> Extensions in OBIS</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="formatting.html"><a href="formatting.html#how-are-extensions-linked-to-core-tables-in-obis"><i class="fa fa-check"></i><b>3.3.1</b> How are extensions linked to core tables in OBIS?</a></li>
<li class="chapter" data-level="3.3.2" data-path="formatting.html"><a href="formatting.html#differences-between-identifiers"><i class="fa fa-check"></i><b>3.3.2</b> Differences between identifiers</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="formatting.html"><a href="formatting.html#data-formatting-tools"><i class="fa fa-check"></i><b>3.4</b> Data formatting tools</a></li>
<li class="chapter" data-level="3.5" data-path="identifiers.html"><a href="identifiers.html"><i class="fa fa-check"></i><b>3.5</b> Constructing and using indentifier codes</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="identifiers.html"><a href="identifiers.html#occurrenceid"><i class="fa fa-check"></i><b>3.5.1</b> occurrenceID</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="checklist.html"><a href="checklist.html"><i class="fa fa-check"></i><b>4</b> Formatting data tables</a>
<ul>
<li class="chapter" data-level="4.1" data-path="checklist.html"><a href="checklist.html#darwin-core-term-checklist-for-obis"><i class="fa fa-check"></i><b>4.1</b> Darwin Core Term Checklist for OBIS</a></li>
<li class="chapter" data-level="4.2" data-path="name_matching.html"><a href="name_matching.html"><i class="fa fa-check"></i><b>4.2</b> Name Matching Strategy for taxonomic quality control</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="name_matching.html"><a href="name_matching.html#taxon-matching-workflow"><i class="fa fa-check"></i><b>4.2.1</b> Taxon Matching Workflow</a></li>
<li class="chapter" data-level="4.2.2" data-path="name_matching.html"><a href="name_matching.html#r-packages-for-taxon-matching"><i class="fa fa-check"></i><b>4.2.2</b> R packages for taxon matching</a></li>
<li class="chapter" data-level="4.2.3" data-path="name_matching.html"><a href="name_matching.html#taxon-match-tools-overview"><i class="fa fa-check"></i><b>4.2.3</b> Taxon Match Tools Overview</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="format_occurrence.html"><a href="format_occurrence.html"><i class="fa fa-check"></i><b>4.3</b> How to format Occurrence tables</a></li>
<li class="chapter" data-level="4.4" data-path="format_event.html"><a href="format_event.html"><i class="fa fa-check"></i><b>4.4</b> How to format Event tables</a></li>
<li class="chapter" data-level="4.5" data-path="format_emof.html"><a href="format_emof.html"><i class="fa fa-check"></i><b>4.5</b> How to format extendedMeasurementOrFact tables</a></li>
<li class="chapter" data-level="4.6" data-path="vocabulary.html"><a href="vocabulary.html"><i class="fa fa-check"></i><b>4.6</b> Choosing Vocabularies for your dataset</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="vocabulary.html"><a href="vocabulary.html#map-your-data-with-dwc-vocabulary"><i class="fa fa-check"></i><b>4.6.1</b> Map your data with DwC vocabulary</a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="vocabulary.html"><a href="vocabulary.html#how-to-map-emof-terms-to-preferred-bodc-vocabulary"><i class="fa fa-check"></i><b>4.7</b> How to map eMoF terms to preferred BODC vocabulary</a>
<ul>
<li class="chapter" data-level="4.7.1" data-path="vocabulary.html"><a href="vocabulary.html#measurementorfact-vocabularies"><i class="fa fa-check"></i><b>4.7.1</b> MeasurementOrFact vocabularies</a></li>
<li class="chapter" data-level="4.7.2" data-path="vocabulary.html"><a href="vocabulary.html#requesting-new-vocabulary-terms"><i class="fa fa-check"></i><b>4.7.2</b> Requesting new vocabulary terms</a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="common_formatissues.html"><a href="common_formatissues.html"><i class="fa fa-check"></i><b>4.8</b> Common Data formatting issues</a>
<ul>
<li class="chapter" data-level="4.8.1" data-path="common_formatissues.html"><a href="common_formatissues.html#missing-required-fields"><i class="fa fa-check"></i><b>4.8.1</b> Missing required fields</a></li>
<li class="chapter" data-level="4.8.2" data-path="common_formatissues.html"><a href="common_formatissues.html#temporal-dates-and-times"><i class="fa fa-check"></i><b>4.8.2</b> Temporal: Dates and times</a></li>
<li class="chapter" data-level="4.8.3" data-path="common_formatissues.html"><a href="common_formatissues.html#historical-data"><i class="fa fa-check"></i><b>4.8.3</b> Historical data</a></li>
<li class="chapter" data-level="4.8.4" data-path="common_formatissues.html"><a href="common_formatissues.html#spatial"><i class="fa fa-check"></i><b>4.8.4</b> Spatial</a></li>
</ul></li>
<li class="chapter" data-level="4.9" data-path="examples.html"><a href="examples.html"><i class="fa fa-check"></i><b>4.9</b> Examples: ENV-DATA and DNA derived data</a>
<ul>
<li class="chapter" data-level="4.9.1" data-path="dna_data.html"><a href="dna_data.html"><i class="fa fa-check"></i><b>4.9.1</b> DNA dervived data</a></li>
<li class="chapter" data-level="4.9.2" data-path="other_data_types.html"><a href="other_data_types.html"><i class="fa fa-check"></i><b>4.9.2</b> Multimedia data (Acoustic, Imaging)</a></li>
<li class="chapter" data-level="4.9.3" data-path="other_data_types.html"><a href="other_data_types.html#habitat-data"><i class="fa fa-check"></i><b>4.9.3</b> Habitat data</a></li>
<li class="chapter" data-level="4.9.4" data-path="other_data_types.html"><a href="other_data_types.html#tracking-data"><i class="fa fa-check"></i><b>4.9.4</b> Tracking data</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Ensuring Data Quality</b></span></li>
<li class="chapter" data-level="5" data-path="data_qc.html"><a href="data_qc.html"><i class="fa fa-check"></i><b>5</b> Data quality control</a>
<ul>
<li class="chapter" data-level="5.1" data-path="data_qc.html"><a href="data_qc.html#why-are-records-dropped"><i class="fa fa-check"></i><b>5.1</b> Why are records dropped?</a></li>
<li class="chapter" data-level="5.2" data-path="data_qc.html"><a href="data_qc.html#how-to-conduct-quality-control"><i class="fa fa-check"></i><b>5.2</b> How to conduct Quality Control</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="data_qc.html"><a href="data_qc.html#conducting-qc-with-obistools"><i class="fa fa-check"></i><b>5.2.1</b> Conducting QC with obistools</a></li>
<li class="chapter" data-level="5.2.2" data-path="data_qc.html"><a href="data_qc.html#qc-with-r-package-hmisc"><i class="fa fa-check"></i><b>5.2.2</b> QC with R package Hmisc</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="dataquality.html"><a href="dataquality.html"><i class="fa fa-check"></i><b>5.3</b> Data quality flags</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="dataquality.html"><a href="dataquality.html#qc-flags-in-downloaded-data"><i class="fa fa-check"></i><b>5.3.1</b> QC Flags in downloaded data</a></li>
<li class="chapter" data-level="5.3.2" data-path="dataquality.html"><a href="dataquality.html#inspecting-qc-flags-with-r"><i class="fa fa-check"></i><b>5.3.2</b> Inspecting QC flags with R</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="mofreports.html"><a href="mofreports.html"><i class="fa fa-check"></i><b>5.4</b> How To Use MoF Report and Tool</a></li>
<li class="chapter" data-level="5.5" data-path="lifewatch_qc.html"><a href="lifewatch_qc.html"><i class="fa fa-check"></i><b>5.5</b> Geographic and data format quality control</a></li>
<li class="chapter" data-level="5.6" data-path="common_qc.html"><a href="common_qc.html"><i class="fa fa-check"></i><b>5.6</b> Common Quality Control issues</a>
<ul>
<li class="chapter" data-level="5.6.1" data-path="common_qc.html"><a href="common_qc.html#uncertain-temporal-range"><i class="fa fa-check"></i><b>5.6.1</b> Uncertain temporal range</a></li>
<li class="chapter" data-level="5.6.2" data-path="common_qc.html"><a href="common_qc.html#uncertain-geolocation"><i class="fa fa-check"></i><b>5.6.2</b> Uncertain geolocation</a></li>
<li class="chapter" data-level="5.6.3" data-path="common_qc.html"><a href="common_qc.html#how-to-use-obis-map-tool"><i class="fa fa-check"></i><b>5.6.3</b> How to use OBIS Map Tool</a></li>
<li class="chapter" data-level="5.6.4" data-path="common_qc.html"><a href="common_qc.html#using-getty-thesaurus-google-maps-to-obtain-locality-coordinates"><i class="fa fa-check"></i><b>5.6.4</b> Using Getty Thesaurus &amp; Google Maps to obtain locality coordinates</a></li>
<li class="chapter" data-level="5.6.5" data-path="common_qc.html"><a href="common_qc.html#how-to-use-marine-regions-gazetteer-tool"><i class="fa fa-check"></i><b>5.6.5</b> How to use Marine Regions Gazetteer tool</a></li>
<li class="chapter" data-level="5.6.6" data-path="common_qc.html"><a href="common_qc.html#low-confidence-taxonomic-identification"><i class="fa fa-check"></i><b>5.6.6</b> Low confidence taxonomic identification</a></li>
<li class="chapter" data-level="5.6.7" data-path="common_qc.html"><a href="common_qc.html#uncertain-measurements"><i class="fa fa-check"></i><b>5.6.7</b> Uncertain measurements</a></li>
<li class="chapter" data-level="5.6.8" data-path="common_qc.html"><a href="common_qc.html#non-marine-species"><i class="fa fa-check"></i><b>5.6.8</b> Non-marine species</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Publishing Data</b></span></li>
<li class="chapter" data-level="6" data-path="data_publication.html"><a href="data_publication.html"><i class="fa fa-check"></i><b>6</b> Data publication and sharing</a>
<ul>
<li class="chapter" data-level="6.1" data-path="data_publication.html"><a href="data_publication.html#licenses"><i class="fa fa-check"></i><b>6.1</b> Licenses</a></li>
<li class="chapter" data-level="6.2" data-path="ipt.html"><a href="ipt.html"><i class="fa fa-check"></i><b>6.2</b> IPT: Integrated Publishing Toolkit</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="ipt.html"><a href="ipt.html#introduction-to-the-ipt"><i class="fa fa-check"></i><b>6.2.1</b> Introduction to the IPT</a></li>
<li class="chapter" data-level="6.2.2" data-path="ipt.html"><a href="ipt.html#how-to-access-the-ipt"><i class="fa fa-check"></i><b>6.2.2</b> How to access the IPT</a></li>
<li class="chapter" data-level="6.2.3" data-path="ipt.html"><a href="ipt.html#create-your-resource-on-the-ipt"><i class="fa fa-check"></i><b>6.2.3</b> Create your resource on the IPT</a></li>
<li class="chapter" data-level="6.2.4" data-path="ipt.html"><a href="ipt.html#map-your-data-to-darwin-core"><i class="fa fa-check"></i><b>6.2.4</b> Map your data to Darwin Core</a></li>
<li class="chapter" data-level="6.2.5" data-path="ipt.html"><a href="ipt.html#add-metadata"><i class="fa fa-check"></i><b>6.2.5</b> Add metadata</a></li>
<li class="chapter" data-level="6.2.6" data-path="ipt.html"><a href="ipt.html#publish-on-the-ipt"><i class="fa fa-check"></i><b>6.2.6</b> Publish on the IPT</a></li>
<li class="chapter" data-level="6.2.7" data-path="ipt.html"><a href="ipt.html#downloading-datasets-from-an-ipt"><i class="fa fa-check"></i><b>6.2.7</b> Downloading datasets from an IPT</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="ipt_admin.html"><a href="ipt_admin.html"><i class="fa fa-check"></i><b>6.3</b> IPT Administration Responsibilities</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="ipt_admin.html"><a href="ipt_admin.html#creating-and-installing-ipts"><i class="fa fa-check"></i><b>6.3.1</b> Creating and Installing IPTs</a></li>
<li class="chapter" data-level="6.3.2" data-path="ipt_admin.html"><a href="ipt_admin.html#registration"><i class="fa fa-check"></i><b>6.3.2</b> Registration</a></li>
<li class="chapter" data-level="6.3.3" data-path="ipt_admin.html"><a href="ipt_admin.html#maintaining-the-ipt"><i class="fa fa-check"></i><b>6.3.3</b> Maintaining the IPT</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="data_sharing.html"><a href="data_sharing.html"><i class="fa fa-check"></i><b>6.4</b> Maintaining and sharing published data</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="data_sharing.html"><a href="data_sharing.html#adding-a-doi-to-datasets"><i class="fa fa-check"></i><b>6.4.1</b> Adding a DOI to datasets</a></li>
<li class="chapter" data-level="6.4.2" data-path="data_sharing.html"><a href="data_sharing.html#user-tracking"><i class="fa fa-check"></i><b>6.4.2</b> User tracking</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="data_sharing.html"><a href="data_sharing.html#update-your-data-in-obis"><i class="fa fa-check"></i><b>6.5</b> Update your data in OBIS</a></li>
<li class="chapter" data-level="6.6" data-path="data_sharing.html"><a href="data_sharing.html#simultaneous-publishing-to-gbif"><i class="fa fa-check"></i><b>6.6</b> Simultaneous publishing to GBIF</a></li>
</ul></li>
<li class="part"><span><b>Access Data from OBIS</b></span></li>
<li class="chapter" data-level="7" data-path="access.html"><a href="access.html"><i class="fa fa-check"></i><b>7</b> Data access</a>
<ul>
<li class="chapter" data-level="7.1" data-path="access.html"><a href="access.html#mapper"><i class="fa fa-check"></i><b>7.1</b> Mapper</a></li>
<li class="chapter" data-level="7.2" data-path="access.html"><a href="access.html#r-package"><i class="fa fa-check"></i><b>7.2</b> R package</a></li>
<li class="chapter" data-level="7.3" data-path="access.html"><a href="access.html#api"><i class="fa fa-check"></i><b>7.3</b> API</a></li>
<li class="chapter" data-level="7.4" data-path="access.html"><a href="access.html#obis-homepage-and-dataset-pages"><i class="fa fa-check"></i><b>7.4</b> OBIS Homepage and dataset pages</a></li>
<li class="chapter" data-level="7.5" data-path="access.html"><a href="access.html#full-exports"><i class="fa fa-check"></i><b>7.5</b> Full exports</a></li>
<li class="chapter" data-level="7.6" data-path="access.html"><a href="access.html#finding-your-own-data-in-obis"><i class="fa fa-check"></i><b>7.6</b> Finding your own data in OBIS</a></li>
<li class="chapter" data-level="7.7" data-path="access.html"><a href="access.html#how-to-contact-data-provider"><i class="fa fa-check"></i><b>7.7</b> How to contact data provider</a></li>
<li class="chapter" data-level="7.8" data-path="access.html"><a href="access.html#interpreting-downloaded-files-from-obis"><i class="fa fa-check"></i><b>7.8</b> Interpreting downloaded files from OBIS</a></li>
<li class="chapter" data-level="7.9" data-path="citing.html"><a href="citing.html"><i class="fa fa-check"></i><b>7.9</b> Citing Data from OBIS</a></li>
</ul></li>
<li class="part"><span><b>Data Visualization and Analysis</b></span></li>
<li class="chapter" data-level="8" data-path="dataviz.html"><a href="dataviz.html"><i class="fa fa-check"></i><b>8</b> Data Visualization</a>
<ul>
<li class="chapter" data-level="8.1" data-path="dataviz.html"><a href="dataviz.html#example-notebooks-using-data-from-obis"><i class="fa fa-check"></i><b>8.1</b> Example notebooks using data from OBIS</a></li>
<li class="chapter" data-level="8.2" data-path="dataviz.html"><a href="dataviz.html#obisindicators-calculating-visualizing-spatial-biodiversity-using-data-from-obis"><i class="fa fa-check"></i><b>8.2</b> obisindicators: calculating &amp; visualizing spatial biodiversity using data from OBIS</a></li>
</ul></li>
<li class="part"><span><b>Additional Resources</b></span></li>
<li class="chapter" data-level="9" data-path="other_resources.html"><a href="other_resources.html"><i class="fa fa-check"></i><b>9</b> Other Resources</a>
<ul>
<li class="chapter" data-level="9.1" data-path="other_resources.html"><a href="other_resources.html#mbon-pole-to-pole-tutorial"><i class="fa fa-check"></i><b>9.1</b> MBON Pole to Pole Tutorial</a></li>
<li class="chapter" data-level="9.2" data-path="other_resources.html"><a href="other_resources.html#ioos-darwin-core-guide"><i class="fa fa-check"></i><b>9.2</b> IOOS Darwin Core Guide</a></li>
<li class="chapter" data-level="9.3" data-path="other_resources.html"><a href="other_resources.html#emodnet-biology"><i class="fa fa-check"></i><b>9.3</b> EMODnet Biology</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The OBIS manual</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="dna-dervived-data" class="section level3 hasAnchor" number="4.9.1">
<h3><span class="header-section-number">4.9.1</span> DNA dervived data<a href="dna_data.html#dna-dervived-data" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>Contents:</strong></p>
<ul>
<li><a href="index.html#introduction">Introduction</a></li>
<li><a href="dna_data.html#edna-and-dna-derived-data-example">eDNA &amp; DNA Derived use cases</a></li>
<li><a href="dna_data.html#how-to-find-genetic-data-in-obis">How to find genetic data in OBIS</a></li>
</ul>
<div id="introduction-1" class="section level4 hasAnchor" number="4.9.1.1">
<h4><span class="header-section-number">4.9.1.1</span> Introduction<a href="dna_data.html#introduction-1" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>DNA derived data are increasingly being used to document taxon occurrences. This genetic data may come from a sampling event, an individual organism, may be linked to physical material (or not), or may result from DNA detection methods e.g., metabarcoding or qPCR. Thus genetic data may reflect a single organism, or may include information from bulk samples with many individuals. Still, DNA-derived occurrence data of species should be documented as standardized and as reproducible as possible.</p>
<p>To ensure DNA data are useful to the broadest possible community, GBIF published a guide entitled <a href="https://docs.gbif-uat.org/publishing-dna-derived-data/1.0/en/">Publishing DNA-derived data through biodiversity data platforms</a>. This guide is supported by the <a href="http://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml">DNA derived data extension for Darwin Core</a>, which incorporates MIxS terms into the Darwin Core standard. There are 5 categories for which genetic data could fall into:</p>
<ol style="list-style-type: decimal">
<li>DNA-derived occurrences</li>
<li>Enriched occurrences</li>
<li>Targeted species detection</li>
<li>Name references</li>
<li>Metadata only</li>
</ol>
<p>For a guide and decision tree on determining which category your data falls into, see the <a href="https://docs.gbif.org/publishing-dna-derived-data/1.0/en/#data-packaging-and-mapping">Data packaging and mapping</a> section of the GBIF guide. Refer to the <a href="dna_data.html#edna--dna-derived-data">examples below</a> for use case examples of eDNA and DNA derived data (Category 1).</p>
<blockquote>
<p>Currently, genetic data <strong>must</strong> be published with Occurrence core, not Event core. eDNA and DNA derived data are then linked to the Occurrence core data table with the use of <code>occurrenceID</code> and/or <code>eventID</code>. See below for further guidance on compiling genetic data.</p>
</blockquote>
<div id="how-to-compile-and-publish-genetic-data" class="section level5 hasAnchor" number="4.9.1.1.1">
<h5><span class="header-section-number">4.9.1.1.1</span> How to compile and publish genetic data<a href="dna_data.html#how-to-compile-and-publish-genetic-data" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Broadly speaking, you will need to have information on the taxonomy and sequences for each occurrence record associated with a DNA sample. You should first fill in the <a href="format_occurrence.html">Occurrence core table</a>, and then complete the DNA Derived Data extension (as well as the eMoF extension, if applicable, for any measurements taken).</p>
<p>In addition to the <a href="format_occurrence.html">usual required terms for Occurrence datasets</a>, you should consider the following additional terms:</p>
<p><strong>Occurrence core table highly recommended terms:</strong></p>
<ul>
<li>Class Occurrence | DwC: organismQuantity</li>
<li>Class Occurrence | DwC: OrganismQuantityType</li>
<li>Class Occurrence| DwC: associatedSequences</li>
<li>Class Event | DwC: sampleSizeValue</li>
<li>Class Event | DwC: sampleSizeUnit</li>
<li>Class Event | DwC: samplingProtocol</li>
</ul>
<p>Then, you will need to format the DNADerivedData extension. The following (free-text) terms are required or highly recommended. Note that some terms are more relevant depending on the methodologies used (e.g., metabarcoding vs qPCR).</p>
<p><strong>DNA Derived data extension highly recommended terms:</strong></p>
<ul>
<li>DNA Derived | DwC: DNA_sequence</li>
<li>DNA Derived | DwC: sop</li>
<li>DNA Derived | DwC: target_gene</li>
<li>DNA Derived | DwC: target_subfragment</li>
<li>DNA Derived | DwC: pcr_primer_forward</li>
<li>DNA Derived | DwC: pcr_primer_reverse</li>
<li>DNA Derived | DwC: pcr_primer_name_forward</li>
<li>DNA Derived | DwC: pcr_primer_name_reverse</li>
<li>DNA Derived | DwC: pcr_primer_reference</li>
<li>DNA Derived | DwC: Pcr_cond</li>
<li>DNA Derived | DwC: annealingTemp</li>
<li>DNA Derived | DwC: annealinTempUnit</li>
<li>DNA Derived | DwC: probeReporter</li>
<li>DNA Derived | DwC: probeQuencher</li>
<li>DNA Derived | DwC: ampliconSize</li>
<li>DNA Derived | DwC: thresholdQuantificationCycle</li>
<li>DNA Derived | DwC: baselineValue</li>
</ul>
<p>For a complete list of terms you can map to, see <a href="http://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2021-07-05.xml">the DwC DNA Derived Data extension page</a>. See the <a href="dna_data.html#edna--dna-derived-data">examples below</a> for use case examples. The Marine Biological Data Mobilization Workshop also has a <a href="https://ioos.github.io/bio_mobilization_workshop/edna-extension/#dna-derived-extension">tutorial</a> for this type of data.</p>
<p>When your data tables are formatted and you are ready to publish it on the IPT, it will follow the same process for <a href="data_publication.html">publishing on an IPT</a>. You will upload your source files, and add the Occurrence core Darwin Core mappings, and then the DNA Derived Data Darwin Core mappings. However the extension must first be <a href="data_publication.html#ipt-administration">installed by the IPT administrator</a> (often the node manager). Once the extension is installed, you can add the Darwin Core DNA Derived Data mapping for that file.</p>
</div>
<div id="obis-bioinformatics-pipline" class="section level5 hasAnchor" number="4.9.1.1.2">
<h5><span class="header-section-number">4.9.1.1.2</span> OBIS Bioinformatics Pipline<a href="dna_data.html#obis-bioinformatics-pipline" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>OBIS recognizes the vast amount of data generated from marine DNA sampling, especially from eDNA sequencing. Thus we have been developing a bioinformatics pipeline to facilitate publication of this data into OBIS. The pipeline was initially developed for the <a href="https://pacman.obis.org/">PacMAN project (Pacific Islands Marine Bioinvasions Alert Network)</a>.</p>
<p>Broadly speaking, it creates a framework that receives raw sequence data from eDNA samples, cleans, aligns, classifies sequences, and finally outputs a DwC-compatible table. The pipeline is currently under development and for now only accepts CO1 data. It will be extended to include other genetic markers in the future. More details about the PacMAN pipeline can be found on its <a href="https://github.com/iobis/PacMAN-pipeline">associated GitHub repository</a>. Once fully online, we will provide guidelines on how to use the pipeline.</p>
<p>OBIS is developing guidelines and pipelines to accept other data types, such as:</p>
<ul>
<li><a href="other_data_types.html#multimedia-data">Acoustic</a></li>
<li><a href="other_data_types.html#multimedia-data">Imaging</a></li>
<li><a href="other_data_types.html#tracking-data">Tracking</a></li>
<li><a href="other_data_types.html#habitat-data">Habitat</a></li>
</ul>
</div>
</div>
<div id="edna-and-dna-derived-data-example" class="section level4 hasAnchor" number="4.9.1.2">
<h4><span class="header-section-number">4.9.1.2</span> eDNA and DNA derived data example<a href="dna_data.html#edna-and-dna-derived-data-example" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The following example use cases draw on both the <a href="https://docs.gbif-uat.org/publishing-dna-derived-data/1.0/en/">GBIF guide</a> and the <a href="https://rs.gbif-uat.org/extensions.html#http">DNA derived data extension</a> to illustrate how to incorporate a DNA derived data extension file into a Darwin Core archive. Note: for the purposes of this section, only required Occurrence core terms are shown, in addition to all eDNA &amp; DNA specific terms. For additional Occurrence core terms, refer to <a href="darwin_core.html#occurrence">Occurrence</a>.</p>
<div id="edna-data-from-monterey-bay-california" class="section level5 hasAnchor" number="4.9.1.2.1">
<h5><span class="header-section-number">4.9.1.2.1</span> eDNA data from Monterey Bay, California<a href="dna_data.html#edna-data-from-monterey-bay-california" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>The data for this example is from the use case <a href="https://www1.usgs.gov/obis-usa/ipt/resource?r=18s_monterey_bay_time_series_edna">“18S Monterey Bay Time Series: an eDNA data set from Monterey Bay, California, including years 2006, 2013 - 2016’</a>. The data from this study originate from marine filtered seawater samples that have undergone metabarcoding of the 18S V9 region.</p>
<p><strong>Occurrence core:</strong></p>
<p>We can populate the Occurrence core with all the required and highly recommended fields, as well as considering the eDNA and DNA specific fields. The Occurrence core contain the taxonomic identification of each ASV observed; its number of reads, as well as relevant metadata including the sample collection location, references for the identification procedure, and links to archived sequences.</p>
<p><code>OccurrenceID</code> and <code>basisOfRecord</code> are some of the required Occurrence core terms, in addition to the highly recommended fields of <code>organismQuantity</code> and <code>organismQuantityType</code>. A selection of samples from this plate were included in another publication (Djurhuus et al., 2020),which is recorded in <code>identificationReferences</code> along with the GitHub repository where the data can be found.</p>
<table>
<colgroup>
<col width="22%" />
<col width="13%" />
<col width="14%" />
<col width="17%" />
<col width="31%" />
</colgroup>
<thead>
<tr class="header">
<th>occurrenceID</th>
<th>basisOfRecord</th>
<th>organismQuantity</th>
<th>OrganismQuantityType</th>
<th>associatedSequences</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>11216c01_12_edna_1_S_occ1</td>
<td>MaterialSample</td>
<td>19312</td>
<td>DNA sequence reads</td>
<td>NCBI BioProject acc. nr. PRJNA433203</td>
</tr>
<tr class="even">
<td>11216c01_12_edna_2_S_occ1</td>
<td>MaterialSample</td>
<td>16491</td>
<td>DNA sequence reads</td>
<td>NCBI BioProject acc. nr. PRJNA433203</td>
</tr>
<tr class="odd">
<td>11216c01_12_edna_3_S_occ1</td>
<td>MaterialSample</td>
<td>21670</td>
<td>DNA sequence reads</td>
<td>NCBI BioProject acc. nr. PRJNA433203</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="11%" />
<col width="14%" />
<col width="34%" />
<col width="38%" />
</colgroup>
<thead>
<tr class="header">
<th>sampleSizeValue</th>
<th>sampleSizeUnit</th>
<th>identificationReferences</th>
<th>identificationRemarks</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>147220</td>
<td>DNA sequence reads</td>
<td>GitHub repository <a href="https://doi.org/10.1038/s41467-019-14105-1">Djurhuus et al. 2020</a></td>
<td>unassigned, Genbank nr Release 221 September 20 2017</td>
</tr>
<tr class="even">
<td>121419</td>
<td>DNA sequence reads</td>
<td>GitHub repository <a href="https://doi.org/10.1038/s41467-019-14105-1">Djurhuus et al. 2020</a></td>
<td>unassigned, Genbank nr Release 221 September 20 2017</td>
</tr>
<tr class="odd">
<td>161525</td>
<td>DNA sequence reads</td>
<td>GitHub repository <a href="https://doi.org/10.1038/s41467-019-14105-1">Djurhuus et al. 2020</a></td>
<td>unassigned, Genbank nr Release 221 September 20 2017</td>
</tr>
</tbody>
</table>
<p><strong>DNA Derived Data extension:</strong></p>
<p>Next, we can create the <strong>DNA Derived Data extension</strong> which will be connected to the Occurrence core with the use of <code>occurrenceID</code>. This extension contains the DNA sequences and relevant DNA metadata, including sequencing procedures, primers used and SOP’s. The recommended use of ENVO’s biome classes were applied to describe the environmental system from which the sample was extracted. The samples were collected by CTD rosette and filtered by a peristaltic pump system. Illumina MiSeq metabarcoding was applied for the target_gene 18S and the target_subfragment, V9 region. URL’s are provided for the protocols followed for nucleic acids extraction and amplification.</p>
<p>For a detailed description of the steps taken to process the data, including algorithms used, see the original publication. Adding Operational Taxonomic Unit (OTU) related data are highly recommended and should be as complete as possible, for example:</p>
<table>
<colgroup>
<col width="20%" />
<col width="22%" />
<col width="22%" />
<col width="34%" />
</colgroup>
<thead>
<tr class="header">
<th>occurrenceID</th>
<th>env-broad_scale</th>
<th>env_local_scale</th>
<th>env_medium</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>11216c01_12_edna_1_S_occ1</td>
<td>marine biome (ENVO:00000447)</td>
<td>coastal water (ENVO:00001250)</td>
<td>waterborne particulate matter (ENVO:01000436)</td>
</tr>
<tr class="even">
<td>11216c01_12_edna_2_S_occ1</td>
<td>marine biome (ENVO:00000447)</td>
<td>coastal water (ENVO:00001250)</td>
<td>waterborne particulate matter (ENVO:01000436)</td>
</tr>
<tr class="odd">
<td>11216c01_12_edna_3_S_occ1</td>
<td>marine biome (ENVO:00000447)</td>
<td>coastal water (ENVO:00001250)</td>
<td>waterborne particulate matter (ENVO:01000436)</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="16%" />
<col width="32%" />
<col width="32%" />
<col width="9%" />
<col width="9%" />
</colgroup>
<thead>
<tr class="header">
<th>samp_vol_we_dna_ext</th>
<th>nucl_acid_ext</th>
<th>nucl_acid_amp</th>
<th>lib_layout</th>
<th>target_gene</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1000ml</td>
<td><a href="https://www.protocols.io/view/mbari-environmental-dna-edna-extraction-using-qiag-bp2l647d5vqe/v1">dx.doi.org/10.17504/protocols.io.xjufknw</a></td>
<td><a href="https://www.protocols.io/view/environmental-dna-edna-18s-metabarcoding-illumina-6qpvrydzgmkn/v2">dx.doi.org/10.17504/protocols.io.n2vdge6</a></td>
<td>paired</td>
<td>18S</td>
</tr>
<tr class="even">
<td>1000ml</td>
<td><a href="https://www.protocols.io/view/mbari-environmental-dna-edna-extraction-using-qiag-bp2l647d5vqe/v1">dx.doi.org/10.17504/protocols.io.xjufknw</a></td>
<td><a href="https://www.protocols.io/view/environmental-dna-edna-18s-metabarcoding-illumina-6qpvrydzgmkn/v2">dx.doi.org/10.17504/protocols.io.n2vdge6</a></td>
<td>paired</td>
<td>18S</td>
</tr>
<tr class="odd">
<td>1000ml</td>
<td><a href="https://www.protocols.io/view/mbari-environmental-dna-edna-extraction-using-qiag-bp2l647d5vqe/v1">dx.doi.org/10.17504/protocols.io.xjufknw</a></td>
<td><a href="https://www.protocols.io/view/environmental-dna-edna-18s-metabarcoding-illumina-6qpvrydzgmkn/v2">dx.doi.org/10.17504/protocols.io.n2vdge6</a></td>
<td>paired</td>
<td>18S</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="15%" />
<col width="15%" />
<col width="13%" />
<col width="55%" />
</colgroup>
<thead>
<tr class="header">
<th>target_subfragment</th>
<th>seq_meth</th>
<th>otu_class_appr</th>
<th>otu_seq_comp_appr</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>V9</td>
<td>Illumina MiSeq 2x250</td>
<td>dada2;1.14.0;ASV</td>
<td>blast;2.9.0+;80% identity;e-value cutoff: x MEGAN6;6.18.5;bitscore: 100 :2%</td>
</tr>
<tr class="even">
<td>V9</td>
<td>Illumina MiSeq 2x250</td>
<td>dada2;1.14.0;ASV</td>
<td>blast;2.9.0+;80% identity;e-value cutoff: x MEGAN6;6.18.5;bitscore: 100 :2%</td>
</tr>
<tr class="odd">
<td>V9</td>
<td>Illumina MiSeq 2x250</td>
<td>dada2;1.14.0;ASV</td>
<td>blast;2.9.0+;80% identity;e-value cutoff: x MEGAN6;6.18.5;bitscore: 100 :2%</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="16%" />
<col width="64%" />
<col width="18%" />
</colgroup>
<thead>
<tr class="header">
<th>otu_db</th>
<th>sop</th>
<th>DNA_sequence</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Genbank nr;221</td>
<td><a href="https://www.protocols.io/view/mbari-environmental-dna-edna-extraction-using-qiag-bp2l647d5vqe/v1">dx.doi.org/10.17504/protocols.io.xjufknw</a> or GitHub repository</td>
<td>GCTACTACCGATT…</td>
</tr>
<tr class="even">
<td>Genbank nr;221</td>
<td><a href="https://www.protocols.io/view/mbari-environmental-dna-edna-extraction-using-qiag-bp2l647d5vqe/v1">dx.doi.org/10.17504/protocols.io.xjufknw</a> or GitHub repository</td>
<td>GCTACTACCGATT…</td>
</tr>
<tr class="odd">
<td>Genbank nr;221</td>
<td><a href="https://www.protocols.io/view/mbari-environmental-dna-edna-extraction-using-qiag-bp2l647d5vqe/v1">dx.doi.org/10.17504/protocols.io.xjufknw</a> or GitHub repository</td>
<td>GCTACTACCGATT…</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="16%" />
<col width="20%" />
<col width="20%" />
<col width="20%" />
<col width="22%" />
</colgroup>
<thead>
<tr class="header">
<th>pcr_primer_forward</th>
<th>pcr_primer_reverse</th>
<th>pcr_primer_name_forward</th>
<th>pcr_primer_name_reverse</th>
<th>pcr_primer_reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>GTACACACCGCCCGTC</td>
<td>TGATCCTTCTGCAGGTTCACCTAC</td>
<td>1391f</td>
<td>EukBr</td>
<td>Amaral-Zettler et al. 2009</td>
</tr>
<tr class="even">
<td>GTACACACCGCCCGTC</td>
<td>TGATCCTTCTGCAGGTTCACCTAC</td>
<td>1391f</td>
<td>EukBr</td>
<td>Amaral-Zettler et al. 2009</td>
</tr>
<tr class="odd">
<td>GTACACACCGCCCGTC</td>
<td>TGATCCTTCTGCAGGTTCACCTAC</td>
<td>1391f</td>
<td>EukBr</td>
<td>Amaral-Zettler et al. 2009</td>
</tr>
</tbody>
</table>
</div>
<div id="s-rrna-gene-metabarcoding-data-of-pico--to-mesoplankton" class="section level5 hasAnchor" number="4.9.1.2.2">
<h5><span class="header-section-number">4.9.1.2.2</span> 16S rRNA gene metabarcoding data of Pico- to Mesoplankton<a href="dna_data.html#s-rrna-gene-metabarcoding-data-of-pico--to-mesoplankton" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>DNA derived datasets can also include an extendedMeasurementsOrFact (eMoF) extension file, in addition to the Occurrence and DNA derived extensions. In this example, environmental measurements were provided in an eMoF file, in addition to the DNA derived data and occurrence data. Here we show how to incorporate such measurements in the extensions.</p>
<p>In the publication <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2016.00679/full">“Diversity of Pico- to Mesoplankton along the 2000 km Salinity Gradient of the Baltic Sea”</a>, a dataset with 16S rRNA gene metabarcoding data of surface water microbial communities was created from 21 off-shore stations, following a transect from Kattegat to the Gulf of Bothnia in the Baltic Sea. The full dataset entitled <a href="https://www.gbif.org/dataset/9e29a2fe-d780-48a8-a93f-9ce041f9202f">“Diversity of Pico- to Mesoplankton along the 2000 km Salinity Gradient of the Baltic Sea (Hu et al. 2016)</a> is available from GBIF.</p>
<p><strong>Occurrence core:</strong></p>
<p>The Occurrence core contain information about the organisms in the sample including the taxonomy and quantity of organisms detected, the collection location, references for the identification procedure, and links to the sequences generated.</p>
<p><strong>Important note: even though this dataset uses OTU identifiers for taxonomy (therefore not including <code>scientificNameID</code>) OBIS still recommends using <code>scientificNameID</code>.</strong></p>
<table>
<colgroup>
<col width="15%" />
<col width="48%" />
<col width="16%" />
<col width="19%" />
</colgroup>
<thead>
<tr class="header">
<th>basisOfRecord</th>
<th>occurrenceID</th>
<th>eventID</th>
<th>eventDate</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>MaterialSample</td>
<td>SBDI-ASV-3:16S_1:919a2aa9d306e4cf3fa9ca02a2aa5730</td>
<td>SBDI-ASV-3:16S_1</td>
<td>2013-07-13 07:08:00</td>
</tr>
<tr class="even">
<td>MaterialSample</td>
<td>SBDI-ASV-3:16S_1:43e088977eba5732bfa45e20b1d8cdd2</td>
<td>SBDI-ASV-3:16S_1</td>
<td>2013-07-13 07:08:00</td>
</tr>
<tr class="odd">
<td>MaterialSample</td>
<td>SBDI-ASV-3:16S_1:887bc7033b46d960e893caceb711700b</td>
<td>SBDI-ASV-3:16S_1</td>
<td>2013-07-13 07:08:00</td>
</tr>
</tbody>
</table>
<table style="width:100%;">
<colgroup>
<col width="23%" />
<col width="28%" />
<col width="22%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>organismQuantity</th>
<th>organismQuantityType</th>
<th>sampleSizeValue</th>
<th>sampleSizeUnit</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>2235</td>
<td>DNA sequence reads</td>
<td>12393</td>
<td>DNA sequence reads</td>
</tr>
<tr class="even">
<td>795</td>
<td>DNA sequence reads</td>
<td>12393</td>
<td>DNA sequence reads</td>
</tr>
<tr class="odd">
<td>40</td>
<td>DNA sequence reads</td>
<td>12393</td>
<td>DNA sequence reads</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="34%" />
<col width="12%" />
<col width="20%" />
<col width="31%" />
</colgroup>
<thead>
<tr class="header">
<th>samplingProtocol</th>
<th>associatedSequences</th>
<th>identificationReferences</th>
<th>identificationRemarks</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>200–500 mL seawater were filtered onto 0.22 μm pore-size mixed cellulose ester membrane filters; [<a href="https://doi.org/10.3389/fmicb.2016.00679" class="uri">https://doi.org/10.3389/fmicb.2016.00679</a>]</td>
<td>[<a href="https://www.ebi.ac.uk/ena/browser/view/ERR1202034" class="uri">https://www.ebi.ac.uk/ena/browser/view/ERR1202034</a>]</td>
<td>[<a href="https://docs.biodiversitydata.se/analyse-data/molecular-tools/#taxonomy-annotation" class="uri">https://docs.biodiversitydata.se/analyse-data/molecular-tools/#taxonomy-annotation</a>]</td>
<td>DADA2:assignTaxonomy:addSpecies annotation against sbdi-gtdb=R06-RS202-1; confidence at lowest specified (ASV portal) taxon: 0.5</td>
</tr>
<tr class="even">
<td>200–500 mL seawater were filtered onto 0.22 μm pore-size mixed cellulose ester membrane filters; [<a href="https://doi.org/10.3389/fmicb.2016.00679" class="uri">https://doi.org/10.3389/fmicb.2016.00679</a>]</td>
<td>[<a href="https://www.ebi.ac.uk/ena/browser/view/ERR1202034" class="uri">https://www.ebi.ac.uk/ena/browser/view/ERR1202034</a>]</td>
<td>[<a href="https://docs.biodiversitydata.se/analyse-data/molecular-tools/#taxonomy-annotation" class="uri">https://docs.biodiversitydata.se/analyse-data/molecular-tools/#taxonomy-annotation</a>]</td>
<td>DADA2:assignTaxonomy:addSpecies annotation against sbdi-gtdb=R06-RS202-1; confidence at lowest specified (ASV portal) taxon: 0.56</td>
</tr>
<tr class="odd">
<td>200–500 mL seawater were filtered onto 0.22 μm pore-size mixed cellulose ester membrane filters; [<a href="https://doi.org/10.3389/fmicb.2016.00679" class="uri">https://doi.org/10.3389/fmicb.2016.00679</a>]</td>
<td>[<a href="https://www.ebi.ac.uk/ena/browser/view/ERR1202034" class="uri">https://www.ebi.ac.uk/ena/browser/view/ERR1202034</a>]</td>
<td>[<a href="https://docs.biodiversitydata.se/analyse-data/molecular-tools/#taxonomy-annotation" class="uri">https://docs.biodiversitydata.se/analyse-data/molecular-tools/#taxonomy-annotation</a>]</td>
<td>DADA2:assignTaxonomy:addSpecies annotation against sbdi-gtdb=R06-RS202-1; confidence at lowest specified (ASV portal) taxon: 0.99</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="18%" />
<col width="19%" />
<col width="40%" />
<col width="21%" />
</colgroup>
<thead>
<tr class="header">
<th>decimalLatitude</th>
<th>decimalLongitude</th>
<th>taxonID</th>
<th>scientificName</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>55.185</td>
<td>13.791</td>
<td>ASV:919a2aa9d306e4cf3fa9ca02a2aa5730</td>
<td>UBA6821</td>
</tr>
<tr class="even">
<td>55.185</td>
<td>13.791</td>
<td>ASV:43e088977eba5732bfa45e20b1d8cdd2</td>
<td>Chthoniobacterales</td>
</tr>
<tr class="odd">
<td>55.185</td>
<td>13.791</td>
<td>ASV:887bc7033b46d960e893caceb711700b</td>
<td>BACL27 sp014190055</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="10%" />
<col width="19%" />
<col width="18%" />
<col width="20%" />
<col width="20%" />
<col width="9%" />
</colgroup>
<thead>
<tr class="header">
<th>kingdom</th>
<th>phylum</th>
<th>class</th>
<th>order</th>
<th>family</th>
<th>genus</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Bacteria</td>
<td>Verrucomicrobiota</td>
<td>Verrucomicrobiae</td>
<td>Chthoniobacterales</td>
<td>UBA6821</td>
<td>UBA6821</td>
</tr>
<tr class="even">
<td>Bacteria</td>
<td>Verrucomicrobiota</td>
<td>Verrucomicrobiae</td>
<td>Chthoniobacterales</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>Bacteria</td>
<td>Actinobacteriota</td>
<td>Acidimicrobiia</td>
<td>Acidimicrobiales</td>
<td>Ilumatobacteraceae</td>
<td>BACL27</td>
</tr>
</tbody>
</table>
<p><strong>DNA Derived Data extension:</strong></p>
<p>The DNA Derived Data extension for metabarcoding data contains the DNA sequences and relevant DNA metadata, primers and procedures. This example table contains the highly recommended and recommended fields as populated with the example dataset data. For this dataset, authors additionally provided measurements of of water sample temperature and salinity, which are provided in an <strong>extendedMeasurementOrFact</strong> extension file:</p>
<table>
<colgroup>
<col width="35%" />
<col width="21%" />
<col width="20%" />
<col width="22%" />
</colgroup>
<thead>
<tr class="header">
<th>id</th>
<th>env_broad_scale</th>
<th>env_local_scale</th>
<th>env_medium</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>SBDI-ASV-3:16S_1:919a2aa9d306e4cf3fa9ca02a2aa5730</td>
<td>aquatic biome [ENVO_00002030]</td>
<td>marine biome [ENVO_00000447]</td>
<td>brackish water [ENVO_00002019]</td>
</tr>
<tr class="even">
<td>SBDI-ASV-3:16S_1:43e088977eba5732bfa45e20b1d8cdd2</td>
<td>aquatic biome [ENVO_00002030]</td>
<td>marine biome [ENVO_00000447]</td>
<td>brackish water [ENVO_00002019]</td>
</tr>
<tr class="odd">
<td>SBDI-ASV-3:16S_1:887bc7033b46d960e893caceb711700b</td>
<td>aquatic biome [ENVO_00002030]</td>
<td>marine biome [ENVO_00000447]</td>
<td>brackish water [ENVO_00002019]</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="13%" />
<col width="14%" />
<col width="22%" />
<col width="18%" />
<col width="30%" />
</colgroup>
<thead>
<tr class="header">
<th>lib_layout</th>
<th>target_gene</th>
<th>target_subfragment</th>
<th>seq_meth</th>
<th>sop</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>paired</td>
<td>16S rRNA</td>
<td>V3-V4</td>
<td>Illumina MiSeq</td>
<td><a href="https://nf-co.re/ampliseq" class="uri">https://nf-co.re/ampliseq</a></td>
</tr>
<tr class="even">
<td>paired</td>
<td>16S rRNA</td>
<td>V3-V4</td>
<td>Illumina MiSeq</td>
<td><a href="https://nf-co.re/ampliseq" class="uri">https://nf-co.re/ampliseq</a></td>
</tr>
<tr class="odd">
<td>paired</td>
<td>16S rRNA</td>
<td>V3-V4</td>
<td>Illumina MiSeq</td>
<td><a href="https://nf-co.re/ampliseq" class="uri">https://nf-co.re/ampliseq</a></td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="17%" />
<col width="19%" />
<col width="21%" />
<col width="21%" />
<col width="20%" />
</colgroup>
<thead>
<tr class="header">
<th>pcr_primer_forward</th>
<th>pcr_primer_reverse</th>
<th>pcr_primer_name_forward</th>
<th>pcr_primer_name_reverse</th>
<th>DNA_sequence</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>CCTACGGGNGGCWGCAG</td>
<td>GACTACHVGGGTATCTAATCC</td>
<td>341</td>
<td>805R</td>
<td>TCGAGAATTTTTCACAATG…</td>
</tr>
<tr class="even">
<td>CCTACGGGNGGCWGCAG</td>
<td>GACTACHVGGGTATCTAATCC</td>
<td>341</td>
<td>805R</td>
<td>TCGAGAATTTTTCACAATG…</td>
</tr>
<tr class="odd">
<td>CCTACGGGNGGCWGCAG</td>
<td>GACTACHVGGGTATCTAATCC</td>
<td>341</td>
<td>805R</td>
<td>TGGGGAATCTTGCGCAATG…</td>
</tr>
</tbody>
</table>
<p><strong>extendedMeasurementOrFact (eMoF) extension:</strong></p>
<table>
<colgroup>
<col width="22%" />
<col width="38%" />
<col width="12%" />
<col width="13%" />
<col width="12%" />
</colgroup>
<thead>
<tr class="header">
<th>measurementID</th>
<th>occurrenceID</th>
<th>measurementType</th>
<th>measurementValue</th>
<th>measurementUnit</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>SBDI-ASV-3:16S_1:temperature</td>
<td>SBDI-ASV-3:16S_1:919a2aa9d306e4cf3fa9ca02a2aa5730</td>
<td>temperature</td>
<td>16.9</td>
<td>°C</td>
</tr>
<tr class="even">
<td>SBDI-ASV-3:16S_1:salinity</td>
<td>SBDI-ASV-3:16S_1:919a2aa9d306e4cf3fa9ca02a2aa5730</td>
<td>salinity</td>
<td>7.25</td>
<td>psu</td>
</tr>
<tr class="odd">
<td>SBDI-ASV-3:16S_1:temperature</td>
<td>SBDI-ASV-3:16S_1:1ead98754d34073a4606f7ff1e94126e</td>
<td>temperature</td>
<td>16.9</td>
<td>°C</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="how-to-find-genetic-data-in-obis" class="section level4 hasAnchor" number="4.9.1.3">
<h4><span class="header-section-number">4.9.1.3</span> How to find genetic data in OBIS<a href="dna_data.html#how-to-find-genetic-data-in-obis" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<div id="sequence-search-tool" class="section level5 hasAnchor" number="4.9.1.3.1">
<h5><span class="header-section-number">4.9.1.3.1</span> Sequence Search tool<a href="dna_data.html#sequence-search-tool" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>If you want to search for sequences or related sequences in OBIS, you can do so with the <a href="https://sequence.obis.org/">OBIS Sequence Search</a>.</p>
<ol style="list-style-type: decimal">
<li>Copy your sequence in the provided box (an example sequence is provided for testing as well)</li>
<li>Press the Search button</li>
<li>View results below</li>
<li>You can also change the Minimum Alignment Score slider in the map view to see location of sequences</li>
</ol>
<p>The search result will show you taxonomic information for species sequences that align to your sequence, the alignment score, and a link to the respective datasets.</p>
</div>
<div id="obis-mapper" class="section level5 hasAnchor" number="4.9.1.3.2">
<h5><span class="header-section-number">4.9.1.3.2</span> OBIS Mapper<a href="dna_data.html#obis-mapper" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>If you wish to find records that have the DNADerivedData extension you can add this filter when using the <a href="https://mapper.obis.org/">Mapper</a>.</p>
<ol style="list-style-type: decimal">
<li>Navigate to the Criteria tab</li>
<li>Open the Extensions section</li>
<li>Check the box for DNADerivedData</li>
<li>Click save to create the layer</li>
<li><a href="access.html">Download the data from the layer</a></li>
</ol>

<p><strong>Content</strong></p>
<ul>
<li><a href="other_data_types.html#multimedia-data-acoustic-imaging">Multimedia data</a></li>
<li><a href="other_data_types.html#habitat-data">Habitat data</a></li>
<li><a href="other_data_types.html#tracking-data">Tracking data</a></li>
</ul>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="examples.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="other_data_types.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["manual.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "none"
}
});
});
</script>

</body>

</html>
