<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>4.10 Examples: ENV-DATA and DNA derived data | The OBIS manual</title>
  <meta name="description" content="4.10 Examples: ENV-DATA and DNA derived data | The OBIS manual" />
  <meta name="generator" content="bookdown 0.36 and GitBook 2.6.7" />

  <meta property="og:title" content="4.10 Examples: ENV-DATA and DNA derived data | The OBIS manual" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="4.10 Examples: ENV-DATA and DNA derived data | The OBIS manual" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="common_formatissues.html"/>
<link rel="next" href="other_data_types.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-JBW0JFMNBJ"></script>
<script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'G-JBW0JFMNBJ');
</script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="/">The OBIS manual</a></li>
<li><a href="https://obis.org/">Return to OBIS.org</a></li>

<li class="divider"></li>
<li class="part"><span><b>Overview</b></span></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#guidelines-on-the-sharing-and-use-of-data-in-obis"><i class="fa fa-check"></i><b>1.1</b> Guidelines on the sharing and use of data in OBIS</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i><b>1.2</b> Acknowledgements</a></li>
<li class="chapter" data-level="1.3" data-path="policy.html"><a href="policy.html"><i class="fa fa-check"></i><b>1.3</b> Data Policy</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="policy.html"><a href="policy.html#guidelines-on-the-sharing-and-use-of-data-in-obis-1"><i class="fa fa-check"></i><b>1.3.1</b> Guidelines on the sharing and use of data in OBIS</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="gethelp.html"><a href="gethelp.html"><i class="fa fa-check"></i><b>1.4</b> Getting Help in OBIS</a></li>
<li class="chapter" data-level="1.5" data-path="FAQ.html"><a href="FAQ.html"><i class="fa fa-check"></i><b>1.5</b> Frequently Asked Questions</a></li>
</ul></li>
<li class="part"><span><b>Contributing data to OBIS</b></span></li>
<li class="chapter" data-level="2" data-path="contribute.html"><a href="contribute.html"><i class="fa fa-check"></i><b>2</b> What can you contribute and how?</a>
<ul>
<li class="chapter" data-level="2.1" data-path="contribute.html"><a href="contribute.html#why-publish-data-to-obis"><i class="fa fa-check"></i><b>2.1</b> Why publish data to OBIS</a></li>
<li class="chapter" data-level="2.2" data-path="contribute.html"><a href="contribute.html#how-to-handle-sensitive-data"><i class="fa fa-check"></i><b>2.2</b> How to handle sensitive data</a></li>
<li class="chapter" data-level="2.3" data-path="data_standards.html"><a href="data_standards.html"><i class="fa fa-check"></i><b>2.3</b> OBIS Data Life Cycle</a></li>
<li class="chapter" data-level="2.4" data-path="data_standards.html"><a href="data_standards.html#biodiversity-data-standards"><i class="fa fa-check"></i><b>2.4</b> Biodiversity data standards</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="darwin_core.html"><a href="darwin_core.html"><i class="fa fa-check"></i><b>2.4.1</b> Darwin Core</a></li>
<li class="chapter" data-level="2.4.2" data-path="data_format.html"><a href="data_format.html"><i class="fa fa-check"></i><b>2.4.2</b> Darwin Core Archive</a></li>
<li class="chapter" data-level="2.4.3" data-path="relational_db.html"><a href="relational_db.html"><i class="fa fa-check"></i><b>2.4.3</b> Relational databases: the underlying framework of OBIS</a></li>
<li class="chapter" data-level="2.4.4" data-path="eml.html"><a href="eml.html"><i class="fa fa-check"></i><b>2.4.4</b> Ecological Metadata Language</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="nodes.html"><a href="nodes.html"><i class="fa fa-check"></i><b>2.5</b> OBIS nodes</a>
<ul>
<li class="chapter" data-level="2.5.1" data-path="nodes.html"><a href="nodes.html#terms-of-reference-of-obis-nodes"><i class="fa fa-check"></i><b>2.5.1</b> Terms of Reference of OBIS nodes</a></li>
<li class="chapter" data-level="2.5.2" data-path="nodes.html"><a href="nodes.html#how-to-become-an-obis-node"><i class="fa fa-check"></i><b>2.5.2</b> How to become an OBIS node</a></li>
<li class="chapter" data-level="2.5.3" data-path="nodes.html"><a href="nodes.html#obis-node-health-status-check-and-transition-strategy"><i class="fa fa-check"></i><b>2.5.3</b> OBIS Node Health Status Check and Transition Strategy</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Data Formatting</b></span></li>
<li class="chapter" data-level="3" data-path="formatting.html"><a href="formatting.html"><i class="fa fa-check"></i><b>3</b> Dataset structure</a>
<ul>
<li class="chapter" data-level="3.1" data-path="formatting.html"><a href="formatting.html#when-to-use-event-core"><i class="fa fa-check"></i><b>3.1</b> When to use Event Core</a></li>
<li class="chapter" data-level="3.2" data-path="formatting.html"><a href="formatting.html#when-to-use-occurrence-core"><i class="fa fa-check"></i><b>3.2</b> When to use Occurrence Core</a></li>
<li class="chapter" data-level="3.3" data-path="formatting.html"><a href="formatting.html#extensions-in-obis"><i class="fa fa-check"></i><b>3.3</b> Extensions in OBIS</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="formatting.html"><a href="formatting.html#how-are-extensions-linked-to-core-tables-in-obis"><i class="fa fa-check"></i><b>3.3.1</b> How are extensions linked to core tables in OBIS?</a></li>
<li class="chapter" data-level="3.3.2" data-path="formatting.html"><a href="formatting.html#differences-between-identifiers"><i class="fa fa-check"></i><b>3.3.2</b> Differences between identifiers</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="formatting.html"><a href="formatting.html#data-formatting-tools"><i class="fa fa-check"></i><b>3.4</b> Data formatting tools</a></li>
<li class="chapter" data-level="3.5" data-path="identifiers.html"><a href="identifiers.html"><i class="fa fa-check"></i><b>3.5</b> Constructing and using indentifier codes</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="identifiers.html"><a href="identifiers.html#eventid"><i class="fa fa-check"></i><b>3.5.1</b> eventID</a></li>
<li class="chapter" data-level="3.5.2" data-path="identifiers.html"><a href="identifiers.html#occurrenceid"><i class="fa fa-check"></i><b>3.5.2</b> occurrenceID</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="checklist.html"><a href="checklist.html"><i class="fa fa-check"></i><b>4</b> Formatting data tables</a>
<ul>
<li class="chapter" data-level="4.1" data-path="checklist.html"><a href="checklist.html#darwin-core-term-checklist-for-obis"><i class="fa fa-check"></i><b>4.1</b> Darwin Core Term Checklist for OBIS</a></li>
<li class="chapter" data-level="4.2" data-path="name_matching.html"><a href="name_matching.html"><i class="fa fa-check"></i><b>4.2</b> Name Matching Strategy for taxonomic quality control</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="name_matching.html"><a href="name_matching.html#taxon-matching-workflow"><i class="fa fa-check"></i><b>4.2.1</b> Taxon Matching Workflow</a></li>
<li class="chapter" data-level="4.2.2" data-path="name_matching.html"><a href="name_matching.html#r-packages-for-taxon-matching"><i class="fa fa-check"></i><b>4.2.2</b> R packages for taxon matching</a></li>
<li class="chapter" data-level="4.2.3" data-path="name_matching.html"><a href="name_matching.html#taxon-match-tools-overview"><i class="fa fa-check"></i><b>4.2.3</b> Taxon Match Tools Overview</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="format_occurrence.html"><a href="format_occurrence.html"><i class="fa fa-check"></i><b>4.3</b> How to format Occurrence tables</a></li>
<li class="chapter" data-level="4.4" data-path="format_event.html"><a href="format_event.html"><i class="fa fa-check"></i><b>4.4</b> How to format Event tables</a></li>
<li class="chapter" data-level="4.5" data-path="format_emof.html"><a href="format_emof.html"><i class="fa fa-check"></i><b>4.5</b> How to format extendedMeasurementOrFact tables</a></li>
<li class="chapter" data-level="4.6" data-path="dna_data.html"><a href="dna_data.html"><i class="fa fa-check"></i><b>4.6</b> DNA derived data</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="dna_data.html"><a href="dna_data.html#introduction-to-dna-data"><i class="fa fa-check"></i><b>4.6.1</b> Introduction to DNA data</a></li>
<li class="chapter" data-level="4.6.2" data-path="dna_data.html"><a href="dna_data.html#how-to-find-genetic-data-in-obis"><i class="fa fa-check"></i><b>4.6.2</b> How to find genetic data in OBIS</a></li>
<li class="chapter" data-level="4.6.3" data-path="dna_data.html"><a href="dna_data.html#guidelines-for-compiling-genetic-data-edna-and-metabarcoding-datasets"><i class="fa fa-check"></i><b>4.6.3</b> Guidelines for compiling genetic data: eDNA and metabarcoding datasets</a></li>
<li class="chapter" data-level="4.6.4" data-path="dna_data.html"><a href="dna_data.html#unknown-sequences"><i class="fa fa-check"></i><b>4.6.4</b> Unknown sequences</a></li>
<li class="chapter" data-level="4.6.5" data-path="dna_data.html"><a href="dna_data.html#guidelines-for-compiling-genetic-data-qpcr"><i class="fa fa-check"></i><b>4.6.5</b> Guidelines for compiling genetic data: qPCR</a></li>
<li class="chapter" data-level="4.6.6" data-path="dna_data.html"><a href="dna_data.html#obis-bioinformatics-pipline"><i class="fa fa-check"></i><b>4.6.6</b> OBIS Bioinformatics Pipline</a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="vocabulary.html"><a href="vocabulary.html"><i class="fa fa-check"></i><b>4.7</b> Choosing vocabularies for your dataset</a>
<ul>
<li class="chapter" data-level="4.7.1" data-path="vocabulary.html"><a href="vocabulary.html#map-data-fields-to-darwin-core"><i class="fa fa-check"></i><b>4.7.1</b> Map data fields to Darwin Core</a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="vocabulary.html"><a href="vocabulary.html#map-emof-measurement-identifiers-to-preferred-bodc-vocabulary"><i class="fa fa-check"></i><b>4.8</b> Map eMoF measurement identifiers to preferred BODC vocabulary</a>
<ul>
<li class="chapter" data-level="4.8.1" data-path="vocabulary.html"><a href="vocabulary.html#measurementorfact-vocabulary-background"><i class="fa fa-check"></i><b>4.8.1</b> MeasurementOrFact vocabulary background</a></li>
<li class="chapter" data-level="4.8.2" data-path="vocabulary.html"><a href="vocabulary.html#requesting-new-vocabulary-terms"><i class="fa fa-check"></i><b>4.8.2</b> Requesting new vocabulary terms</a></li>
</ul></li>
<li class="chapter" data-level="4.9" data-path="common_formatissues.html"><a href="common_formatissues.html"><i class="fa fa-check"></i><b>4.9</b> Common Data formatting issues</a>
<ul>
<li class="chapter" data-level="4.9.1" data-path="common_formatissues.html"><a href="common_formatissues.html#missing-required-fields"><i class="fa fa-check"></i><b>4.9.1</b> Missing required fields</a></li>
<li class="chapter" data-level="4.9.2" data-path="common_formatissues.html"><a href="common_formatissues.html#temporal-dates-and-times"><i class="fa fa-check"></i><b>4.9.2</b> Temporal: Dates and times</a></li>
<li class="chapter" data-level="4.9.3" data-path="common_formatissues.html"><a href="common_formatissues.html#historical-data"><i class="fa fa-check"></i><b>4.9.3</b> Historical data</a></li>
<li class="chapter" data-level="4.9.4" data-path="common_formatissues.html"><a href="common_formatissues.html#spatial"><i class="fa fa-check"></i><b>4.9.4</b> Spatial</a></li>
</ul></li>
<li class="chapter" data-level="4.10" data-path="examples.html"><a href="examples.html"><i class="fa fa-check"></i><b>4.10</b> Examples: ENV-DATA and DNA derived data</a>
<ul>
<li class="chapter" data-level="4.10.1" data-path="other_data_types.html"><a href="other_data_types.html"><i class="fa fa-check"></i><b>4.10.1</b> Other data types</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Ensuring Data Quality</b></span></li>
<li class="chapter" data-level="5" data-path="data_qc.html"><a href="data_qc.html"><i class="fa fa-check"></i><b>5</b> Data quality control</a>
<ul>
<li class="chapter" data-level="5.1" data-path="data_qc.html"><a href="data_qc.html#why-are-records-dropped"><i class="fa fa-check"></i><b>5.1</b> Why are records dropped?</a></li>
<li class="chapter" data-level="5.2" data-path="data_qc.html"><a href="data_qc.html#how-to-conduct-quality-control"><i class="fa fa-check"></i><b>5.2</b> How to conduct Quality Control</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="data_qc.html"><a href="data_qc.html#conducting-qc-with-obistools"><i class="fa fa-check"></i><b>5.2.1</b> Conducting QC with obistools</a></li>
<li class="chapter" data-level="5.2.2" data-path="data_qc.html"><a href="data_qc.html#qc-with-r-package-hmisc"><i class="fa fa-check"></i><b>5.2.2</b> QC with R package Hmisc</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="dataquality.html"><a href="dataquality.html"><i class="fa fa-check"></i><b>5.3</b> Data quality flags</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="dataquality.html"><a href="dataquality.html#qc-flags-in-downloaded-data"><i class="fa fa-check"></i><b>5.3.1</b> QC Flags in downloaded data</a></li>
<li class="chapter" data-level="5.3.2" data-path="dataquality.html"><a href="dataquality.html#inspecting-qc-flags-with-r"><i class="fa fa-check"></i><b>5.3.2</b> Inspecting QC flags with R</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="mofreports.html"><a href="mofreports.html"><i class="fa fa-check"></i><b>5.4</b> How To Use MoF Report and Tool</a></li>
<li class="chapter" data-level="5.5" data-path="lifewatch_qc.html"><a href="lifewatch_qc.html"><i class="fa fa-check"></i><b>5.5</b> Geographic and data format quality control</a></li>
<li class="chapter" data-level="5.6" data-path="common_qc.html"><a href="common_qc.html"><i class="fa fa-check"></i><b>5.6</b> Common Quality Control issues</a>
<ul>
<li class="chapter" data-level="5.6.1" data-path="common_qc.html"><a href="common_qc.html#uncertain-temporal-range"><i class="fa fa-check"></i><b>5.6.1</b> Uncertain temporal range</a></li>
<li class="chapter" data-level="5.6.2" data-path="common_qc.html"><a href="common_qc.html#uncertain-geolocation"><i class="fa fa-check"></i><b>5.6.2</b> Uncertain geolocation</a></li>
<li class="chapter" data-level="5.6.3" data-path="common_qc.html"><a href="common_qc.html#how-to-use-obis-map-tool"><i class="fa fa-check"></i><b>5.6.3</b> How to use OBIS Map Tool</a></li>
<li class="chapter" data-level="5.6.4" data-path="common_qc.html"><a href="common_qc.html#using-getty-thesaurus-google-maps-to-obtain-locality-coordinates"><i class="fa fa-check"></i><b>5.6.4</b> Using Getty Thesaurus &amp; Google Maps to obtain locality coordinates</a></li>
<li class="chapter" data-level="5.6.5" data-path="common_qc.html"><a href="common_qc.html#how-to-use-marine-regions-gazetteer-tool"><i class="fa fa-check"></i><b>5.6.5</b> How to use Marine Regions Gazetteer tool</a></li>
<li class="chapter" data-level="5.6.6" data-path="common_qc.html"><a href="common_qc.html#low-confidence-taxonomic-identification"><i class="fa fa-check"></i><b>5.6.6</b> Low confidence taxonomic identification</a></li>
<li class="chapter" data-level="5.6.7" data-path="common_qc.html"><a href="common_qc.html#uncertain-measurements"><i class="fa fa-check"></i><b>5.6.7</b> Uncertain measurements</a></li>
<li class="chapter" data-level="5.6.8" data-path="common_qc.html"><a href="common_qc.html#non-marine-species"><i class="fa fa-check"></i><b>5.6.8</b> Non-marine species</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Publishing Data</b></span></li>
<li class="chapter" data-level="6" data-path="data_publication.html"><a href="data_publication.html"><i class="fa fa-check"></i><b>6</b> Data publication and sharing</a>
<ul>
<li class="chapter" data-level="6.1" data-path="data_publication.html"><a href="data_publication.html#licenses"><i class="fa fa-check"></i><b>6.1</b> Licenses</a></li>
<li class="chapter" data-level="6.2" data-path="ipt.html"><a href="ipt.html"><i class="fa fa-check"></i><b>6.2</b> IPT: Integrated Publishing Toolkit</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="ipt.html"><a href="ipt.html#introduction-to-the-ipt"><i class="fa fa-check"></i><b>6.2.1</b> Introduction to the IPT</a></li>
<li class="chapter" data-level="6.2.2" data-path="ipt.html"><a href="ipt.html#how-to-access-the-ipt"><i class="fa fa-check"></i><b>6.2.2</b> How to access the IPT</a></li>
<li class="chapter" data-level="6.2.3" data-path="ipt.html"><a href="ipt.html#create-your-resource-on-the-ipt"><i class="fa fa-check"></i><b>6.2.3</b> Create your resource on the IPT</a></li>
<li class="chapter" data-level="6.2.4" data-path="ipt.html"><a href="ipt.html#map-your-data-to-darwin-core"><i class="fa fa-check"></i><b>6.2.4</b> Map your data to Darwin Core</a></li>
<li class="chapter" data-level="6.2.5" data-path="ipt.html"><a href="ipt.html#add-metadata"><i class="fa fa-check"></i><b>6.2.5</b> Add metadata</a></li>
<li class="chapter" data-level="6.2.6" data-path="ipt.html"><a href="ipt.html#publish-on-the-ipt"><i class="fa fa-check"></i><b>6.2.6</b> Publish on the IPT</a></li>
<li class="chapter" data-level="6.2.7" data-path="ipt.html"><a href="ipt.html#downloading-datasets-from-an-ipt"><i class="fa fa-check"></i><b>6.2.7</b> Downloading datasets from an IPT</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="ipt_admin.html"><a href="ipt_admin.html"><i class="fa fa-check"></i><b>6.3</b> IPT Administration Responsibilities</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="ipt_admin.html"><a href="ipt_admin.html#creating-and-installing-ipts"><i class="fa fa-check"></i><b>6.3.1</b> Creating and Installing IPTs</a></li>
<li class="chapter" data-level="6.3.2" data-path="ipt_admin.html"><a href="ipt_admin.html#registration"><i class="fa fa-check"></i><b>6.3.2</b> Registration</a></li>
<li class="chapter" data-level="6.3.3" data-path="ipt_admin.html"><a href="ipt_admin.html#maintaining-the-ipt"><i class="fa fa-check"></i><b>6.3.3</b> Maintaining the IPT</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="data_sharing.html"><a href="data_sharing.html"><i class="fa fa-check"></i><b>6.4</b> Maintaining and sharing published data</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="data_sharing.html"><a href="data_sharing.html#adding-a-doi-to-datasets"><i class="fa fa-check"></i><b>6.4.1</b> Adding a DOI to datasets</a></li>
<li class="chapter" data-level="6.4.2" data-path="data_sharing.html"><a href="data_sharing.html#user-tracking"><i class="fa fa-check"></i><b>6.4.2</b> User tracking</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="data_sharing.html"><a href="data_sharing.html#update-your-data-in-obis"><i class="fa fa-check"></i><b>6.5</b> Update your data in OBIS</a></li>
<li class="chapter" data-level="6.6" data-path="data_sharing.html"><a href="data_sharing.html#simultaneous-publishing-to-gbif"><i class="fa fa-check"></i><b>6.6</b> Simultaneous publishing to GBIF</a></li>
</ul></li>
<li class="part"><span><b>Access Data from OBIS</b></span></li>
<li class="chapter" data-level="7" data-path="access.html"><a href="access.html"><i class="fa fa-check"></i><b>7</b> Data access</a>
<ul>
<li class="chapter" data-level="7.1" data-path="access.html"><a href="access.html#obis-homepage-and-dataset-pages"><i class="fa fa-check"></i><b>7.1</b> OBIS Homepage and dataset pages</a></li>
<li class="chapter" data-level="7.2" data-path="access.html"><a href="access.html#mapper"><i class="fa fa-check"></i><b>7.2</b> Mapper</a></li>
<li class="chapter" data-level="7.3" data-path="access.html"><a href="access.html#r-package"><i class="fa fa-check"></i><b>7.3</b> R package</a></li>
<li class="chapter" data-level="7.4" data-path="access.html"><a href="access.html#api"><i class="fa fa-check"></i><b>7.4</b> API</a></li>
<li class="chapter" data-level="7.5" data-path="access.html"><a href="access.html#full-exports"><i class="fa fa-check"></i><b>7.5</b> Full exports</a></li>
<li class="chapter" data-level="7.6" data-path="access.html"><a href="access.html#finding-your-own-data-in-obis"><i class="fa fa-check"></i><b>7.6</b> Finding your own data in OBIS</a></li>
<li class="chapter" data-level="7.7" data-path="access.html"><a href="access.html#how-to-contact-data-provider"><i class="fa fa-check"></i><b>7.7</b> How to contact data provider</a></li>
<li class="chapter" data-level="7.8" data-path="access.html"><a href="access.html#interpreting-downloaded-files-from-obis"><i class="fa fa-check"></i><b>7.8</b> Interpreting downloaded files from OBIS</a></li>
<li class="chapter" data-level="7.9" data-path="citing.html"><a href="citing.html"><i class="fa fa-check"></i><b>7.9</b> Citing Data from OBIS</a></li>
</ul></li>
<li class="part"><span><b>Data Visualization and Analysis</b></span></li>
<li class="chapter" data-level="8" data-path="dataviz.html"><a href="dataviz.html"><i class="fa fa-check"></i><b>8</b> Data Visualization</a>
<ul>
<li class="chapter" data-level="8.1" data-path="dataviz.html"><a href="dataviz.html#example-notebooks-using-data-from-obis"><i class="fa fa-check"></i><b>8.1</b> Example notebooks using data from OBIS</a></li>
<li class="chapter" data-level="8.2" data-path="dataviz.html"><a href="dataviz.html#obisindicators-calculating-visualizing-spatial-biodiversity-using-data-from-obis"><i class="fa fa-check"></i><b>8.2</b> obisindicators: calculating &amp; visualizing spatial biodiversity using data from OBIS</a></li>
</ul></li>
<li class="part"><span><b>Additional Resources</b></span></li>
<li class="chapter" data-level="9" data-path="other_resources.html"><a href="other_resources.html"><i class="fa fa-check"></i><b>9</b> Other Resources</a>
<ul>
<li class="chapter" data-level="9.1" data-path="other_resources.html"><a href="other_resources.html#mbon-pole-to-pole-tutorial"><i class="fa fa-check"></i><b>9.1</b> MBON Pole to Pole Tutorial</a></li>
<li class="chapter" data-level="9.2" data-path="other_resources.html"><a href="other_resources.html#ioos-darwin-core-guide"><i class="fa fa-check"></i><b>9.2</b> IOOS Darwin Core Guide</a></li>
<li class="chapter" data-level="9.3" data-path="other_resources.html"><a href="other_resources.html#emodnet-biology"><i class="fa fa-check"></i><b>9.3</b> EMODnet Biology</a></li>
<li class="chapter" data-level="9.4" data-path="other_resources.html"><a href="other_resources.html#template-generators"><i class="fa fa-check"></i><b>9.4</b> Template Generators</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The OBIS manual</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="examples-env-data-and-dna-derived-data" class="section level2 hasAnchor" number="4.10">
<h2><span class="header-section-number">4.10</span> Examples: ENV-DATA and DNA derived data<a href="examples.html#examples-env-data-and-dna-derived-data" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><strong>Contents</strong></p>
<ul>
<li><a href="examples.html#fish-abundance-and-distribution">Fish abundance &amp; distribution</a></li>
<li><a href="examples.html#hard-coral-cover-and-composition">Hard coral cover &amp; composition</a></li>
<li><a href="examples.html#invertebrates-abundance-and-distribution">Invertebrates abundance &amp; distribution</a></li>
<li><a href="examples.html#macroalgae-canopy-cover-and-composition">Macroalgae canopy cover &amp; composition</a></li>
<li><a href="examples.html#mangroves-cover-and-composition">Mangroves cover &amp; composition</a></li>
<li><a href="examples.html#marine-birds-abundance-and-distribution">Marine birds abundance &amp; distribution</a></li>
<li><a href="examples.html#marine-mammals-abundance-and-distribution">Marine mammals abundance &amp; distribution</a></li>
<li><a href="examples.html#marine-turtles-abundance-and-distribution">Marine turtles abundance &amp; distribution</a>
<ul>
<li><a href="examples.html#survey-and-sighting-data">Survey &amp; sighting data</a></li>
</ul></li>
<li><a href="examples.html#microbes-biomass-and-diversity">Microbes biomass &amp; diversity</a></li>
<li><a href="examples.html#phytoplankton-biomass-and-diversity">Phytoplankton biomass &amp; diversity</a></li>
<li><a href="examples.html#seagrass-cover-and-composition">Seagrass cover &amp; composition</a></li>
<li><a href="examples.html#zooplankton-biomass-and-diversity">Zooplankton biomass &amp; diversity</a></li>
</ul>
<p><strong>Special data types:</strong></p>
<ul>
<li><a href="dna_data.html#edna-dna-derived-data">eDNA &amp; DNA derived data</a>
<ul>
<li><a href="dna_data.html#edna-data-from-monterey-bay-california">eDNA data from Monterey Bay, California</a></li>
<li><a href="dna_data#s-rrna-gene-metabarcoding-data-of-pico--to-mesoplankton">16S rRNA gene metabarcoding data of Pico- to Mesoplankton</a></li>
</ul></li>
<li><a href="other_data_types.html#multimedia-data">Acoustic, imaging, or other multimedia data</a></li>
<li><a href="other_data_types.html#tracking-data.html">Tracking data</a></li>
<li><a href="other_data_types.html#habitat-data">Habitat</a></li>
</ul>
<div id="fish-abundance-and-distribution" class="section level4 hasAnchor" number="4.10.0.1">
<h4><span class="header-section-number">4.10.0.1</span> Fish abundance and distribution<a href="examples.html#fish-abundance-and-distribution" class="anchor-section" aria-label="Anchor link to header"></a></h4>
(example coming soon)
<p>
 
</p>
</div>
<div id="hard-coral-cover-and-composition" class="section level4 hasAnchor" number="4.10.0.2">
<h4><span class="header-section-number">4.10.0.2</span> Hard coral cover and composition<a href="examples.html#hard-coral-cover-and-composition" class="anchor-section" aria-label="Anchor link to header"></a></h4>
(example coming soon)
<p>
 
</p>
</div>
<div id="invertebrates-abundance-and-distribution" class="section level4 hasAnchor" number="4.10.0.3">
<h4><span class="header-section-number">4.10.0.3</span> Invertebrates abundance and distribution<a href="examples.html#invertebrates-abundance-and-distribution" class="anchor-section" aria-label="Anchor link to header"></a></h4>
(example coming soon)
<p>
 
</p>
</div>
<div id="macroalgae-canopy-cover-and-composition" class="section level4 hasAnchor" number="4.10.0.4">
<h4><span class="header-section-number">4.10.0.4</span> Macroalgae canopy cover and composition<a href="examples.html#macroalgae-canopy-cover-and-composition" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>In this section we will encode a fictional macroalgal survey dataset into Darwin Core using the ENV-DATA approach, i.e. using an Event core with an Occurrence extension and an extendedMeasurementOrFact extension.</p>
<img src="images/dwca_macroalgae_survey.png" class="img-responsive img-responsive-50"/>
<p class="caption-70">
Figure: A fictional macroalgae survey with a single site, multiple zones, quadrats, and different types of transects.
</p>
<p><strong>Event core:</strong></p>
<p>First we can create the Event core table by extracting all events in a broad sense and populating attributes such as time, location, and depth at the appropriate level. The events at the different levels are linked together using <code>eventID</code> and <code>parentEventID</code>. As the survey sites has a fixed location we can populate <code>decimalLongitude</code> and <code>decimalLatitude</code> at the top level event. The zones have different depths, so <code>minimumDepthInMeters</code> and <code>maximumDepthInMeters</code> are populated at the zone level. Finally, as not all sampling was done on the same day, <code>eventDate</code> is populated at the quadrat and transect level.</p>
<table style="width:100%;">
<colgroup>
<col width="14%" />
<col width="14%" />
<col width="14%" />
<col width="14%" />
<col width="14%" />
<col width="14%" />
<col width="14%" />
</colgroup>
<thead>
<tr class="header">
<th>eventID</th>
<th>parentEventID</th>
<th>eventDate</th>
<th>decimalLongitude</th>
<th>decimalLatitude</th>
<th>minimumDepthInMeters</th>
<th>maximumDepthInMeters</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>site_1</td>
<td></td>
<td></td>
<td>54.7943</td>
<td>16.9425</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>zone_1</td>
<td>site_1</td>
<td></td>
<td></td>
<td></td>
<td>0</td>
<td>0</td>
</tr>
<tr class="odd">
<td>zone_2</td>
<td>site_1</td>
<td></td>
<td></td>
<td></td>
<td>0</td>
<td>5</td>
</tr>
<tr class="even">
<td>zone_3</td>
<td>site_1</td>
<td></td>
<td></td>
<td></td>
<td>5</td>
<td>10</td>
</tr>
<tr class="odd">
<td><span class="marker-green">quadrat_1</span></td>
<td>zone_1</td>
<td>2019-01-02</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>transect_1</td>
<td>zone_2</td>
<td>2019-01-03</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>transect_2</td>
<td>zone_3</td>
<td>2019-01-04</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
<p><strong>Occurrence extension:</strong></p>
<p>Next we can construct the Occurrence extension table. This table has the scientific names and links to the World Register of Marine Species in <code>scientificNameID</code>. The first column of the table references the events in the core table (see <code>quadrat_1</code> for example highlighted in green).</p>
<table>
<colgroup>
<col width="25%" />
<col width="25%" />
<col width="25%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>id</th>
<th>occurrenceID</th>
<th>scientificName</th>
<th>scientificNameID</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><span class="marker-green">quadrat_1</span></td>
<td><span class="marker-blue">occ_1</span></td>
<td>Ulva rigida</td>
<td><a href="urn:lsid:marinespecies.org:taxname:145990">urn:lsid:marinespecies.org:taxname:145990</a></td>
</tr>
<tr class="even">
<td><span class="marker-green">quadrat_1</span></td>
<td><span class="marker-orange">occ_2</span></td>
<td>Ulva lactuca</td>
<td><a href="urn:lsid:marinespecies.org:taxname:145984">urn:lsid:marinespecies.org:taxname:145984</a></td>
</tr>
<tr class="odd">
<td>transect_1</td>
<td>occ_3</td>
<td>Plantae</td>
<td><a href="urn:lsid:marinespecies.org:taxname:3">urn:lsid:marinespecies.org:taxname:3</a></td>
</tr>
<tr class="even">
<td>transect_1</td>
<td>occ_4</td>
<td>Plantae</td>
<td><a href="urn:lsid:marinespecies.org:taxname:3">urn:lsid:marinespecies.org:taxname:3</a></td>
</tr>
<tr class="odd">
<td>transect_2</td>
<td>occ_5</td>
<td>Gracilaria</td>
<td><a href="urn:lsid:marinespecies.org:taxname:144188">urn:lsid:marinespecies.org:taxname:144188</a></td>
</tr>
<tr class="even">
<td>transect_2</td>
<td>occ_6</td>
<td>Laurencia</td>
<td><a href="urn:lsid:marinespecies.org:taxname:143914">urn:lsid:marinespecies.org:taxname:143914</a></td>
</tr>
</tbody>
</table>
<p><strong>extendedMeasurementOrFact (eMoF) extension:</strong></p>
<p>And finally there is the MeasurementOrFact extension table, which has attributes of the zones (shore height), the quadrats (surface area), the transects (surface area and length), and the occurrences (percentage cover and functional group). Attributes of occurrences point to the Occurrence extension table using the <code>occurrenceID</code> column (see <code>occ_1</code> and <code>occ_2</code> highlighted in blue and orange). Note that besides NERC vocabulary terms we are also referencing the CATAMI vocabulary for macroalgal functional groups.</p>
<table>
<colgroup>
<col width="12%" />
<col width="12%" />
<col width="12%" />
<col width="12%" />
<col width="12%" />
<col width="12%" />
<col width="12%" />
<col width="12%" />
</colgroup>
<thead>
<tr class="header">
<th>id</th>
<th>occurrenceID</th>
<th>measurementType</th>
<th>measurementTypeID</th>
<th>measurementValue</th>
<th>measurementValueID</th>
<th>measurementUnit</th>
<th>measurementUnitID</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>zone_1</td>
<td></td>
<td>shore height</td>
<td>?</td>
<td>high</td>
<td>?</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><span class="marker-green">quadrat_1</span></td>
<td></td>
<td>surface area</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/AREABEDS">P01/current/AREABEDS</a></td>
<td>0.25</td>
<td></td>
<td>m2</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UMSQ/">P06/current/UMSQ</a></td>
</tr>
<tr class="odd">
<td>quadrat_1</td>
<td><span class="marker-blue">occ_1</span></td>
<td>cover</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL10/">P01/current/SDBIOL10</a></td>
<td>24</td>
<td></td>
<td>percent</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UPCT">P06/current/UPCT</a></td>
</tr>
<tr class="even">
<td>quadrat_1</td>
<td><span class="marker-orange">occ_2</span></td>
<td>cover</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL10/">P01/current/SDBIOL10</a></td>
<td>56</td>
<td></td>
<td>percent</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UPCT">P06/current/UPCT</a></td>
</tr>
<tr class="odd">
<td>transect_1</td>
<td></td>
<td>surface area</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/AREABEDS">P01/current/AREABEDS</a></td>
<td>60</td>
<td></td>
<td>m2</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UMSQ/">P06/current/UMSQ</a></td>
</tr>
<tr class="even">
<td>transect_1</td>
<td></td>
<td>length</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/LENTRACK">P01/current/LENTRACK</a></td>
<td>30</td>
<td></td>
<td>m</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/ULAA/">P06/current/ULAA</a></td>
</tr>
<tr class="odd">
<td>transect_1</td>
<td>occ_3</td>
<td>functional group</td>
<td>?</td>
<td>sheet-like red</td>
<td>CATAMI:80300925</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>transect_1</td>
<td>occ_4</td>
<td>functional group</td>
<td>?</td>
<td>filamentous brown</td>
<td>CATAMI:80300931</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>transect_1</td>
<td>occ_3</td>
<td>cover</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL10/">P01/current/SDBIOL10</a></td>
<td>8</td>
<td></td>
<td>percent</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UPCT">P06/current/UPCT</a></td>
</tr>
<tr class="even">
<td>transect_1</td>
<td>occ_4</td>
<td>cover</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL10/">P01/current/SDBIOL10</a></td>
<td>24</td>
<td></td>
<td>percent</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UPCT">P06/current/UPCT</a></td>
</tr>
<tr class="odd">
<td>transect_2</td>
<td>occ_5</td>
<td>cover</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL10/">P01/current/SDBIOL10</a></td>
<td>4</td>
<td></td>
<td>percent</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UPCT">P06/current/UPCT</a></td>
</tr>
<tr class="even">
<td>transect_2</td>
<td>occ_6</td>
<td>cover</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL10/">P01/current/SDBIOL10</a></td>
<td>16</td>
<td></td>
<td>percent</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UPCT">P06/current/UPCT</a></td>
</tr>
</tbody>
</table>
</div>
<div id="mangroves-cover-and-composition" class="section level4 hasAnchor" number="4.10.0.5">
<h4><span class="header-section-number">4.10.0.5</span> Mangroves cover and composition<a href="examples.html#mangroves-cover-and-composition" class="anchor-section" aria-label="Anchor link to header"></a></h4>
(example coming soon)
<p>
 
</p>
</div>
<div id="marine-birds-abundance-and-distribution" class="section level4 hasAnchor" number="4.10.0.6">
<h4><span class="header-section-number">4.10.0.6</span> Marine birds abundance and distribution<a href="examples.html#marine-birds-abundance-and-distribution" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The example for ENV-DATA collected with marine bird sightings/occurrences is based on the dataset <a href="https://www.marine.csiro.au/ipt/resource?r=in2017_v02_wov">“RV Investigator Voyage IN2017_V02 Seabird Observations, Australia (2017)”</a>. In this dataset, seabird sightings were recorded continuously during daylight hours during a voyage to recover and redeploy moorings at the SOTS site, southwest of Tasmania, Australia, in March 2017. Observations were made from c.30 minutes before sunrise to c.30 minutes after sunset, extending to 300m in the forward quadrant with the best viewing conditions. There were 1200 observations from 38 species of birds along with 3 cetacean species and one seal. This example will focus on the ENV-DATA associated with the bird sightings. The most frequently sighted bird species were <em>Puﬃnus tenuirostris</em> (Short-tailed Shearwater) and <em>Pachyptila turtur</em> (Fairy Prion).</p>
<p>For this dataset, human observation recorded individual bird sightings (thus, each specimen is a single occurrence). The dataset contains abiotic measurements (ENV-DATA) which are related to each individual sighting, instead of an entire sample. Therefore, we can create an Occurrence core with an eMoF extension that contain the abiotic environmental measurements or facts.</p>
<p><strong>Occurrence core:</strong></p>
<p>The Occurrence core is populated with the occurrence records of seabirds sighted during the RV voyages. Occurrence details and scientific names are provided here. All birds were observed above sea level, all <code>minimumDepthInMeters</code> and <code>maximumDepthInMeters</code> values equal zero.</p>
<table>
<colgroup>
<col width="8%" />
<col width="9%" />
<col width="8%" />
<col width="4%" />
<col width="6%" />
<col width="4%" />
<col width="6%" />
<col width="8%" />
<col width="6%" />
<col width="5%" />
<col width="6%" />
<col width="8%" />
<col width="6%" />
<col width="8%" />
</colgroup>
<thead>
<tr class="header">
<th>occurrenceID</th>
<th>eventDate</th>
<th>institutionCode</th>
<th>collectionCode</th>
<th>basisOfRecord</th>
<th>recordedBy</th>
<th>organismQuantity</th>
<th>organismQuantityType</th>
<th>occurrenceStatus</th>
<th>decimalLatitude</th>
<th>decimalLongitude</th>
<th>coordinateUncertaintyInMeters</th>
<th>coordinatePrecision</th>
<th>footprintWKT</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>in2017_v02_00998</td>
<td>2017-03-17 01:07:00</td>
<td>Australasian Seabird Group, BirdLife Australia</td>
<td>in2017_v02_wov</td>
<td>HumanObservation</td>
<td>EJW+CRC+TAH</td>
<td>2</td>
<td>individuals</td>
<td>present</td>
<td>-43.40741</td>
<td>147.45576</td>
<td>200</td>
<td>0.0018</td>
<td>POINT (147.45576 -43.40741)</td>
</tr>
<tr class="even">
<td>in2017_v02_01380</td>
<td>2017-03-19 22:26:00</td>
<td>Australasian Seabird Group, BirdLife Australia</td>
<td>in2017_v02_wov</td>
<td>HumanObservation</td>
<td>EJW+CRC+TAH</td>
<td>1</td>
<td>individuals</td>
<td>present</td>
<td>-45.98644</td>
<td>142.1445</td>
<td>200</td>
<td>0.0018</td>
<td>POINT (142.14450 -45.98644)</td>
</tr>
<tr class="odd">
<td>in2017_v02_01012</td>
<td>2017-03-17 02:38:00</td>
<td>Australasian Seabird Group, BirdLife Australia</td>
<td>in2017_v02_wov</td>
<td>HumanObservation</td>
<td>EJW+CRC+TAH</td>
<td>1</td>
<td>individuals</td>
<td>present</td>
<td>-43.40728</td>
<td>147.45549</td>
<td>200</td>
<td>0.0018</td>
<td>POINT (147.45549 -43.40728)</td>
</tr>
</tbody>
</table>
<p><strong>extendedMeasurementOrFact (eMoF) extension:</strong></p>
<p>As shown in previous examples, the MeasurementOrFact extension table contains abiotic measurements or facts corresponding to an occurrence / sighting. Individual sightings and abiotic measurements are linked with <code>occurrenceID</code>. In the example dataset, the ENV-DATA consist of measurements taken during the moorings deployment at the SOTS site, at the time of the marine bird sightings. In addition to NERC vocabulary terms, authors also referenced the Australian Ocean Data Network (AODN) Discovery Parameter Vocabulary for <em>Sea-floor depth (m)</em> and <em>Sea Surface Temperature</em> as <code>measurementType</code>. NERC equivalents to the AODN terms are added as additional MeasurementOrFact (MoF) records.</p>
<table>
<colgroup>
<col width="5%" />
<col width="11%" />
<col width="10%" />
<col width="20%" />
<col width="6%" />
<col width="16%" />
<col width="10%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th>occurrenceID</th>
<th>measurementID</th>
<th>measurementType</th>
<th>measurementTypeID</th>
<th>measurementValue</th>
<th>measurementValueID</th>
<th>measurementUnit</th>
<th>measurementUnitID</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>in2017_v02_00998</td>
<td>in2017_v02_00998-depth</td>
<td>Sea-floor depth (m)</td>
<td><a href="http://vocab.aodn.org.au/def/discovery_parameter/entity/574" class="uri">http://vocab.aodn.org.au/def/discovery_parameter/entity/574</a></td>
<td>73.0313</td>
<td>NA</td>
<td>Metres</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/ULAA" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/ULAA</a></td>
</tr>
<tr class="even">
<td>in2017_v02_00998</td>
<td>in2017_v02_00998-depth</td>
<td>Sea-floor depth</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/MBANZZZZ/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/MBANZZZZ/</a></td>
<td>73.0313</td>
<td>NA</td>
<td>Metres</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/ULAA" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/ULAA</a></td>
</tr>
<tr class="odd">
<td>in2017_v02_00998</td>
<td>in2017_v02_00998-air_pressure</td>
<td>Air Pressure (hPa)</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/CAPHZZ01" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/CAPHZZ01</a></td>
<td>1024.91385</td>
<td>NA</td>
<td>hPa</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/HPAX" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/HPAX</a></td>
</tr>
<tr class="even">
<td>in2017_v02_00998</td>
<td>in2017_v02_00998-air_temp</td>
<td>Atmospheric temperature (deg C)</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/CTMPZZ01" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/CTMPZZ01</a></td>
<td>15.3</td>
<td>NA</td>
<td>degrees Celsius</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UPAA" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UPAA</a></td>
</tr>
<tr class="odd">
<td>in2017_v02_00998</td>
<td>in2017_v02_00998-wov_sea_state</td>
<td>Sea state</td>
<td><a href="http://vocab.nerc.ac.uk/collection/C39/current/" class="uri">http://vocab.nerc.ac.uk/collection/C39/current/</a></td>
<td>moderate 1.25 - 2.5 m</td>
<td><a href="http://vocab.nerc.ac.uk/collection/C39/current/4/" class="uri">http://vocab.nerc.ac.uk/collection/C39/current/4/</a></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>in2017_v02_00998</td>
<td>in2017_v02_00998-sea_surface_temp</td>
<td>Sea surface temperature</td>
<td><a href="http://vocab.aodn.org.au/def/discovery_parameter/entity/97" class="uri">http://vocab.aodn.org.au/def/discovery_parameter/entity/97</a></td>
<td>17.32</td>
<td>NA</td>
<td>degrees Celsius</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UPAA" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UPAA</a></td>
</tr>
<tr class="odd">
<td>in2017_v02_00998</td>
<td>in2017_v02_00998-sea_surface_temp</td>
<td>Sea surface temperature</td>
<td><a href="http://vocab.nerc.ac.uk/standard_name/sea_surface_temperature/" class="uri">http://vocab.nerc.ac.uk/standard_name/sea_surface_temperature/</a></td>
<td>17.32</td>
<td>NA</td>
<td>degrees Celsius</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UPAA" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UPAA</a></td>
</tr>
<tr class="even">
<td>in2017_v02_00998</td>
<td>in2017_v02_00998-wind_direction</td>
<td>Wind direction (deg)</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/EWDAZZ01" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/EWDAZZ01</a></td>
<td>283</td>
<td>NA</td>
<td>degrees</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UABB" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UABB</a></td>
</tr>
<tr class="odd">
<td>in2017_v02_00998</td>
<td>in2017_v02_00998-wind_speed</td>
<td>Wind Speed (knt)</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/ESSAZZ01" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/ESSAZZ01</a></td>
<td>5.49</td>
<td>NA</td>
<td>Knots (nautical miles per hour)</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UKNT" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UKNT</a></td>
</tr>
</tbody>
</table>
</div>
<div id="marine-mammals-abundance-and-distribution" class="section level4 hasAnchor" number="4.10.0.7">
<h4><span class="header-section-number">4.10.0.7</span> Marine mammals abundance and distribution<a href="examples.html#marine-mammals-abundance-and-distribution" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>In this section we will explore how to encode a survey data set into Darwin Core using the ENV-DATA approach. As an example, sections of the actual data set of <a href="http://ipt.vliz.be/eurobis/resource?r=cetus_cetaceans">CETUS: Cetacean monitoring surveys in the Eastern North Atlantic</a>, is used.</p>
<img src="images/ENV_example_CETUS.png" class="img-responsive img-responsive-50"/>
Figure: A representation of the observation events of <a href="http://ipt.vliz.be/eurobis/resource?r=cetus_cetaceans">CETUS: Cetacean monitoring surveys in the Eastern North Atlantic</a>, presenting the route <strong>Madeira</strong> as a site with three cruises (zones). Each <strong>Cruise</strong> is divided into different <strong>Transects</strong> and each transect contains a number of <strong>Positions</strong>.
</p>
<p><strong>Event core:</strong></p>
<p>Create the Event core table by extracting all events and populating attributes. As in the previous example, the events at the different levels are linked together using <code>eventID</code> and <code>parentEventID</code>. As the survey observations were made at locations of cetacean sightings instead of fixed locations, we can populate <code>footprintWKT</code> and <code>footprintSRS</code> as location information. Not all sampling was done on the same day, therefore <code>eventDate</code> is populated at the transect level.</p>
<table>
<colgroup>
<col width="11%" />
<col width="8%" />
<col width="9%" />
<col width="62%" />
<col width="7%" />
</colgroup>
<thead>
<tr class="header">
<th>eventID</th>
<th>parentEventID</th>
<th>eventDate</th>
<th>footprintWKT</th>
<th>footprintSRS</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Madeira</td>
<td></td>
<td>2012-07/2017-09</td>
<td>POLYGON ((-16.74 31.49, -16.74 41.23, -8.70 41.23, -8.70 31.49, -16.74 31.49))</td>
<td>EPSG:4326</td>
</tr>
<tr class="even">
<td><span class="marker-green">Madeira:Cruise-001</span></td>
<td>Madeira</td>
<td>2012-07</td>
<td>MULTIPOINT ( (-8.7 41.19), (-9.15 38.7))</td>
<td>EPSG:4326</td>
</tr>
<tr class="odd">
<td>Madeira:Cruise-002</td>
<td>Madeira</td>
<td>2012-07</td>
<td>MULTIPOINT ( (-9.15 38.7), (-16.73 32.74))</td>
<td>EPSG:4326</td>
</tr>
<tr class="even">
<td>Madeira:Cruise-003</td>
<td>Madeira</td>
<td>2012-07</td>
<td>MULTIPOINT ( (-16.73 32.74), (-9.15 38.7))</td>
<td>EPSG:4326</td>
</tr>
</tbody>
</table>
<p><strong>Occurrence extension:</strong></p>
<p>Construct the Occurrence extension table with the scientific names and links to the World Register of Marine Species in <code>scientificNameID</code>. The first column of the table references the events in the core table (see <code>Madeira:Cruise-001</code> highlighted in green).The <code>occurrenceID</code> corresponds to the Position of the observation (see <code>Transect-01:Pos-0001</code> and <code>CIIMAR-CETUS-0001</code> highlighted in blue, or <code>Transect-01:Pos-0002</code> and <code>CIIMAR-CETUS-0002</code> highlighted in orange).</p>
<table>
<colgroup>
<col width="35%" />
<col width="16%" />
<col width="35%" />
<col width="13%" />
</colgroup>
<thead>
<tr class="header">
<th>id</th>
<th>occurrenceID</th>
<th>scientificNameID</th>
<th>scientificName</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><span class="marker-green">Madeira:Cruise-001</span>:<span class="marker-blue">Transect-01:Pos-0001</span></td>
<td><span class="marker-blue">CIIMAR-CETUS-0001</span></td>
<td><a href="urn:lsid:marinespecies.org:taxname:2688" class="uri">urn:lsid:marinespecies.org:taxname:2688</a></td>
<td>Cetacea</td>
</tr>
<tr class="even">
<td><span class="marker-green">Madeira:Cruise-001</span>:<span class="marker-orange">Transect-01:Pos-0002</span></td>
<td><span class="marker-orange">CIIMAR-CETUS-0002</span></td>
<td><a href="urn:lsid:marinespecies.org:taxname:2688" class="uri">urn:lsid:marinespecies.org:taxname:2688</a></td>
<td>Cetacea</td>
</tr>
<tr class="odd">
<td><span class="marker-green">Madeira:Cruise-001</span>:Transect-01:Pos-0003</td>
<td>CIIMAR-CETUS-0003</td>
<td><a href="urn:lsid:marinespecies.org:taxname:2688">urn:lsid:marinespecies.org:taxname:2688</a></td>
<td>Cetacea</td>
</tr>
<tr class="even">
<td><span class="marker-green">Madeira:Cruise-001</span>:Transect-02:Pos-0004</td>
<td>CIIMAR-CETUS-0004</td>
<td><a href="urn:lsid:marinespecies.org:taxname:2688">urn:lsid:marinespecies.org:taxname:2688</a></td>
<td>Cetacea</td>
</tr>
<tr class="odd">
<td><span class="marker-green">Madeira:Cruise-001</span>:Transect-02:Pos-0005</td>
<td>CIIMAR-CETUS-0005</td>
<td><a href="urn:lsid:marinespecies.org:taxname:2688">urn:lsid:marinespecies.org:taxname:2688</a></td>
<td>Cetacea</td>
</tr>
<tr class="even">
<td><span class="marker-green">Madeira:Cruise-001</span>:Transect-02:Pos-0006</td>
<td>CIIMAR-CETUS-0006</td>
<td><a href="urn:lsid:marinespecies.org:taxname:2688">urn:lsid:marinespecies.org:taxname:2688</a></td>
<td>Cetacea</td>
</tr>
<tr class="odd">
<td><span class="marker-green">Madeira:Cruise-001</span>:Transect-02:Pos-0007</td>
<td>CIIMAR-CETUS-0007</td>
<td><a href="urn:lsid:marinespecies.org:taxname:2688">urn:lsid:marinespecies.org:taxname:2688</a></td>
<td>Cetacea</td>
</tr>
</tbody>
</table>
<p><strong>extendedMeasurementOrFact (eMoF) extension:</strong></p>
<p>And finally, the extendedMeasurementOrFact extension table has attributes of the zones (such as Vessel speed and Vessel Heading), the Transects (such as Wave height and Wind speed), and the Positions (such as Visibility and the Number of smaal/big ships &gt;20m). Attributes of Positions point to the Occurrence extension table using the <code>occurrenceID</code> column (see <code>Transect-01:Pos-0001</code> and <code>Transect-01:Pos-0002</code> highlighted in blue and orange, respectively).</p>
<table>
<colgroup>
<col width="10%" />
<col width="12%" />
<col width="11%" />
<col width="24%" />
<col width="13%" />
<col width="17%" />
<col width="11%" />
</colgroup>
<thead>
<tr class="header">
<th>id</th>
<th>occurrenceID</th>
<th>measurementType</th>
<th>measurementTypeID</th>
<th>measurementValue</th>
<th>measurementUnit</th>
<th>measurementUnitID</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><span class="marker-green">Madeira:Cruise-001</span></td>
<td></td>
<td>Vessel name</td>
<td><a href="http://vocab.nerc.ac.uk/collection/Q01/current/Q0100001/">Q01/current/Q0100001</a></td>
<td>Monte da Guia</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>Madeira:Cruise-001:Transect-01</td>
<td></td>
<td>Length of the track</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/DSRNCV01/">P01/current/DSRNCV01</a></td>
<td>39.75</td>
<td>km</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/ULKM/">P06/current/ULKM</a></td>
</tr>
<tr class="odd">
<td>Madeira:Cruise-001:Transect-01:Pos-0001</td>
<td><span class="marker-blue">CIIMAR-CETUS-0001</span></td>
<td>Visibility</td>
<td></td>
<td>2000-4000</td>
<td>Meters</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/ULAA/">P06/current/ULAA</a></td>
</tr>
<tr class="even">
<td>Madeira:Cruise-001:Transect-01:Pos-0001</td>
<td><span class="marker-blue">CIIMAR-CETUS-0001</span></td>
<td>Wind speed</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/WMOCWFBF/">P01/current/WMOCWFBF</a></td>
<td>1</td>
<td>Beaufort scale</td>
<td></td>
</tr>
<tr class="odd">
<td>Madeira:Cruise-001:Transect-01:Pos-0001</td>
<td><span class="marker-blue">CIIMAR-CETUS-0001</span></td>
<td>Wave height</td>
<td></td>
<td>2</td>
<td>Douglas scale</td>
<td></td>
</tr>
<tr class="even">
<td>Madeira:Cruise-001:Transect-01:Pos-0001</td>
<td><span class="marker-blue">CIIMAR-CETUS-0001</span></td>
<td>Number of big ships (&gt;20m)</td>
<td></td>
<td>3</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>Madeira:Cruise-001:Transect-01:Pos-0001</td>
<td><span class="marker-blue">CIIMAR-CETUS-0001</span></td>
<td>Vessel heading</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/HDNGGP01/">P01/current/HDNGGP01</a></td>
<td>206</td>
<td>Degrees</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UAAA/">P06/current/UAAA</a></td>
</tr>
<tr class="even">
<td>Madeira:Cruise-001:Transect-01:Pos-0001</td>
<td><span class="marker-blue">CIIMAR-CETUS-0001</span></td>
<td>Number of small ships (&lt;20m)</td>
<td></td>
<td>0</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>Madeira:Cruise-001:Transect-01:Pos-0001</td>
<td><span class="marker-blue">CIIMAR-CETUS-0001</span></td>
<td>Vessel speed</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/APSAGP01/">P01/current/APSAGP01</a></td>
<td>16</td>
<td>Knots (nautical miles per hour)</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UKNT/">P06/current/UKNT</a></td>
</tr>
</tbody>
</table>
</div>
<div id="marine-turtles-abundance-and-distribution" class="section level4 hasAnchor" number="4.10.0.8">
<h4><span class="header-section-number">4.10.0.8</span> Marine turtles abundance and distribution<a href="examples.html#marine-turtles-abundance-and-distribution" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<div id="survey-and-sighting-data" class="section level5 hasAnchor" number="4.10.0.8.1">
<h5><span class="header-section-number">4.10.0.8.1</span> Survey and sighting data<a href="examples.html#survey-and-sighting-data" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>This section deals with encoding survey and/ or sighting data of sea turtles into Darwin Core using the ENV-DATA approach. Extracts from the actual data set of <a href="https://ipt.vliz.be/eurobis/resource?r=cc_flt_cbar_13-17#anchor-description">Presence of sea turtles collected through Fixed-Line-Transect monitoring across the Western Mediterranean Sea (Civitavecchia-Barcelona route) between 2013 and 2017</a>, are used as an example.</p>
<p><strong>Event core:</strong></p>
<p>The Event core is created by extracting all sighting events and populating the attributes at each event. The events at the different levels are linked together using <code>eventID</code> and <code>parentEventID</code>. In the example dataset, turtle sightings have been recorded since 2007, along a ferry route between Italy and Spain, as part of the monitoring project FLT Med Net (Fixed Line Transect Mediterranean monitoring Network). Turtle sighting locations can be given by populating the fields <code>footprintWKT</code> and <code>footprintSRS</code> with location information. Sightings were recorded at different dates, therefore <code>eventDate</code> is populated at the transect level.</p>
<table style="width:100%;">
<colgroup>
<col width="5%" />
<col width="5%" />
<col width="9%" />
<col width="9%" />
<col width="5%" />
<col width="4%" />
<col width="8%" />
<col width="2%" />
<col width="5%" />
<col width="5%" />
<col width="4%" />
<col width="4%" />
<col width="2%" />
<col width="7%" />
<col width="15%" />
<col width="3%" />
</colgroup>
<thead>
<tr class="header">
<th>id</th>
<th>modified</th>
<th>datasetID</th>
<th>datasetName</th>
<th>eventID</th>
<th>parentEventID</th>
<th>eventDate</th>
<th>eventRemarks</th>
<th>minimumDepthInMeters</th>
<th>maximumDepthInMeters</th>
<th>decimalLatitude</th>
<th>decimalLongitude</th>
<th>geodeticDatum</th>
<th>coordinateUncertaintyInMeters</th>
<th>footprintWKT</th>
<th>footprintSRS</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>TURTLE_CBAR_0043</td>
<td>2021-05-05 07:59:08</td>
<td><a href="https://marineinfo.org/id/dataset/6403" class="uri">https://marineinfo.org/id/dataset/6403</a></td>
<td>Presence of sea turtles collected through Fixed-Line-Transect monitoring across the Western Mediterranean Sea</td>
<td>TURTLE_CBAR_0043</td>
<td></td>
<td>2013-04-03T05:30:00+02:00/2013-04-03T16:00:00+02:00</td>
<td>transect</td>
<td>0</td>
<td>0</td>
<td>41.26179967</td>
<td>4.933265167</td>
<td>EPSG:4326</td>
<td>222970.2874</td>
<td>LINESTRING (7.602633333333 41.24378333333, 2.263897 41.279816)</td>
<td>EPSG:4326</td>
</tr>
<tr class="even">
<td>TURTLE_CBAR_0045</td>
<td>2021-05-05 07:59:08</td>
<td><a href="https://marineinfo.org/id/dataset/6403" class="uri">https://marineinfo.org/id/dataset/6403</a></td>
<td>Presence of sea turtles collected through Fixed-Line-Transect monitoring across the Western Mediterranean Sea</td>
<td>TURTLE_CBAR_0045</td>
<td></td>
<td>2013-04-18T05:22:00+02:00/2013-04-18T15:53:00+02:00</td>
<td>transect</td>
<td>0</td>
<td>0</td>
<td>41.30371367</td>
<td>4.936571167</td>
<td>EPSG:4326</td>
<td>225420.0359</td>
<td>LINESTRING (7.636983333333 41.32418333333, 2.236159 41.283244)</td>
<td>EPSG:4326</td>
</tr>
<tr class="odd">
<td>TURTLE_CBAR_0045_0001</td>
<td>2021-05-05 07:59:08</td>
<td><a href="https://marineinfo.org/id/dataset/6403" class="uri">https://marineinfo.org/id/dataset/6403</a></td>
<td>Presence of sea turtles collected through Fixed-Line-Transect monitoring across the Western Mediterranean Sea</td>
<td>TURTLE_CBAR_0045_0001</td>
<td>TURTLE_CBAR_0045</td>
<td>2013-04-18T05:55:00+02:00</td>
<td>sample</td>
<td>0</td>
<td>0</td>
<td>41.3228</td>
<td>7.4984</td>
<td>EPSG:4326</td>
<td></td>
<td>POINT</td>
<td>EPSG:4326</td>
</tr>
<tr class="even">
<td>TURTLE_CBAR_0045_0002</td>
<td>2021-05-05 07:59:08</td>
<td><a href="https://marineinfo.org/id/dataset/6403" class="uri">https://marineinfo.org/id/dataset/6403</a></td>
<td>Presence of sea turtles collected through Fixed-Line-Transect monitoring across the Western Mediterranean Sea</td>
<td>TURTLE_CBAR_0045_0002</td>
<td>TURTLE_CBAR_0045</td>
<td>2013-04-18T08:35:00+02:00</td>
<td>sample</td>
<td>0</td>
<td>0</td>
<td>41.322845</td>
<td>5.995345</td>
<td>EPSG:4326</td>
<td></td>
<td>POINT</td>
<td>EPSG:4326</td>
</tr>
</tbody>
</table>
<p><strong>Occurrence extension:</strong></p>
<p>The Occurrence extension contain details regarding the sighted animals and include <code>scientificName</code> and the links to the World Register of Marine Species in <code>scientificNameID</code>. The <code>EventID</code> references the events as in the Event core. This table further provides information on the <code>basisOfRecord</code> and <code>occurrenceStatus</code>.</p>
<table>
<colgroup>
<col width="6%" />
<col width="7%" />
<col width="9%" />
<col width="6%" />
<col width="8%" />
<col width="7%" />
<col width="9%" />
<col width="8%" />
<col width="14%" />
<col width="7%" />
<col width="4%" />
<col width="8%" />
</colgroup>
<thead>
<tr class="header">
<th>EventID</th>
<th>occurrenceID</th>
<th>datasetID</th>
<th>collectionCode</th>
<th>basisOfRecord</th>
<th>catalogNumber</th>
<th>recordedBy</th>
<th>occurrenceStatus</th>
<th>scientificNameID</th>
<th>scientificName</th>
<th>kingdom</th>
<th>scientificNameAuthorship</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>TURTLE_CBAR_0043</td>
<td>AdL_TURTLE_CBAR_0001</td>
<td><a href="https://marineinfo.org/id/dataset/6403" class="uri">https://marineinfo.org/id/dataset/6403</a></td>
<td>TURTLE_CBAR_13-17</td>
<td>HumanObservation</td>
<td>AdL_TURTLE_CBAR_0001</td>
<td>Ilaria Campana | Miriam Paraboschi | Erica Ercoli | Erica</td>
<td>absent</td>
<td><a href="urn:lsid:marinespecies.org:taxname:136999" class="uri">urn:lsid:marinespecies.org:taxname:136999</a></td>
<td>Cheloniidae</td>
<td>Animalia</td>
<td>Oppel, 1811</td>
</tr>
<tr class="even">
<td>TURTLE_CBAR_0045</td>
<td>AdL_TURTLE_CBAR_0004</td>
<td><a href="https://marineinfo.org/id/dataset/6403" class="uri">https://marineinfo.org/id/dataset/6403</a></td>
<td>TURTLE_CBAR_13-17</td>
<td>HumanObservation</td>
<td>AdL_TURTLE_CBAR_0004</td>
<td>Antonella Arcangeli | Cristina Berardi | Lucilla Giulietti | Claudia Boccardi</td>
<td>absent</td>
<td><a href="urn:lsid:marinespecies.org:taxname:136999" class="uri">urn:lsid:marinespecies.org:taxname:136999</a></td>
<td>Cheloniidae</td>
<td>Animalia</td>
<td>Oppel, 1811</td>
</tr>
<tr class="odd">
<td>TURTLE_CBAR_0045_0001</td>
<td>AdL_TURTLE_CBAR_0005</td>
<td><a href="https://marineinfo.org/id/dataset/6403" class="uri">https://marineinfo.org/id/dataset/6403</a></td>
<td>TURTLE_CBAR_13-17</td>
<td>HumanObservation</td>
<td>AdL_TURTLE_CBAR_0005</td>
<td>Antonella Arcangeli | Cristina Berardi | Lucilla Giulietti | Claudia Boccardi</td>
<td>present</td>
<td><a href="urn:lsid:marinespecies.org:taxname:137205" class="uri">urn:lsid:marinespecies.org:taxname:137205</a></td>
<td>Caretta caretta</td>
<td>Animalia</td>
<td>Linnaeus, 1758</td>
</tr>
<tr class="even">
<td>TURTLE_CBAR_0045_0002</td>
<td>AdL_TURTLE_CBAR_0006</td>
<td><a href="https://marineinfo.org/id/dataset/6403" class="uri">https://marineinfo.org/id/dataset/6403</a></td>
<td>TURTLE_CBAR_13-17</td>
<td>HumanObservation</td>
<td>AdL_TURTLE_CBAR_0006</td>
<td>Antonella Arcangeli | Cristina Berardi | Lucilla Giulietti | Claudia Boccardi</td>
<td>present</td>
<td><a href="urn:lsid:marinespecies.org:taxname:137205" class="uri">urn:lsid:marinespecies.org:taxname:137205</a></td>
<td>Caretta caretta</td>
<td>Animalia</td>
<td>Linnaeus, 1758</td>
</tr>
</tbody>
</table>
<p><strong>extendedMeasurementOrFact (eMoF) extension:</strong></p>
<p>The extendedMeasurementOrFact extension (eMoF) for survey or sighting data contains additional attributes and measurements recorded during the survey, such as those regarding the Research Vessel, environmental conditions, and/ or animal measurements. These attributes are linked to the Occurrence extension using the <code>occurrenceID</code>. The example dataset contain measurements regarding the sampling method; speed and height of the Research Vessel as platform; wind force; sighting distance; as well as the count and developmental stage of the biological entity.</p>
<table>
<colgroup>
<col width="10%" />
<col width="9%" />
<col width="22%" />
<col width="17%" />
<col width="13%" />
<col width="14%" />
<col width="11%" />
</colgroup>
<thead>
<tr class="header">
<th>id</th>
<th>occurrenceID</th>
<th>measurementType</th>
<th>measurementTypeID</th>
<th>measurementValue</th>
<th>measurementUnit</th>
<th>measurementUnitID</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>TURTLE_CBAR_0045</td>
<td>AdL_TURTLE_CBAR_0004</td>
<td>BEAUFORT WIND FORCE</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/WMOCWFBF" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/WMOCWFBF</a></td>
<td>0</td>
<td>Beaufort scale</td>
<td></td>
</tr>
<tr class="even">
<td>TURTLE_CBAR_0045</td>
<td>AdL_TURTLE_CBAR_0004</td>
<td>Platform height</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/AHSLZZ01" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/AHSLZZ01</a></td>
<td>29</td>
<td>Metres</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/ULAA/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/ULAA/</a></td>
</tr>
<tr class="odd">
<td>TURTLE_CBAR_0045</td>
<td>AdL_TURTLE_CBAR_0004</td>
<td>Sampling method</td>
<td><a href="http://vocab.nerc.ac.uk/collection/Q01/current/Q0100003" class="uri">http://vocab.nerc.ac.uk/collection/Q01/current/Q0100003</a></td>
<td>visual observation from ferries</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>TURTLE_CBAR_0045</td>
<td>AdL_TURTLE_CBAR_0004</td>
<td>Speed of measurement platform relative to ground surface {speed over ground} by unspecified GPS system</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/APSAGP01" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/APSAGP01</a></td>
<td>23.291</td>
<td>Knots (nautical miles per hour)</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UKNT/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UKNT/</a></td>
</tr>
<tr class="odd">
<td>TURTLE_CBAR_0045_0001</td>
<td>AdL_TURTLE_CBAR_0005</td>
<td>Development stage of biological entity specified elsewhere</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/LSTAGE01" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/LSTAGE01</a></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>TURTLE_CBAR_0045_0001</td>
<td>AdL_TURTLE_CBAR_0005</td>
<td>Count (in assayed sample) of biological entity specified elsewhere</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/OCOUNT01" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/OCOUNT01</a></td>
<td>20</td>
<td>Metres</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/ULAA/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/ULAA/</a></td>
</tr>
<tr class="odd">
<td>TURTLE_CBAR_0045_0001</td>
<td>AdL_TURTLE_CBAR_0005</td>
<td>Sighting distance</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
<p>In addition to the measurements recorded by the example dataset, other measurements are also possible depending on the scope and aims of the survey project. The example dataset <a href="https://www.marine.csiro.au/ipt/resource?r=csiro_ss199510_reptile">Incidental sea snake and turtle bycatch records from the RV Southern Surveyor voyage SS199510, Gulf of Carpentaria, Australia (Nov 1995)</a> for example, contain information regarding the length and weight of the biological entity as follows:</p>
<p><strong>extendedMeasurementOrFact (eMoF) extension:</strong></p>
<table>
<colgroup>
<col width="6%" />
<col width="9%" />
<col width="6%" />
<col width="8%" />
<col width="27%" />
<col width="8%" />
<col width="8%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>id</th>
<th>measurementID</th>
<th>occurrenceID</th>
<th>measurementType</th>
<th>measurementTypeID</th>
<th>measurementValue</th>
<th>measurementUnit</th>
<th>measurementUnitID</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>SS199510-001</td>
<td>SS199510-001-length</td>
<td>SS199510-001</td>
<td>Length</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/OBSINDLX" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/OBSINDLX</a></td>
<td>1250</td>
<td>Millimetres</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UXMM" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UXMM</a></td>
</tr>
<tr class="even">
<td>SS199510-001</td>
<td>SS199510-001-weight</td>
<td>SS199510-001</td>
<td>Weight</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/SPWGXX01" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/SPWGXX01</a></td>
<td>800</td>
<td>Grams</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UGRM" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UGRM</a></td>
</tr>
<tr class="odd">
<td>SS199510-002</td>
<td>SS199510-002-length</td>
<td>SS199510-002</td>
<td>Length</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/OBSINDLX" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/OBSINDLX</a></td>
<td>1630</td>
<td>Millimetres</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UXMM" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UXMM</a></td>
</tr>
<tr class="even">
<td>SS199510-002</td>
<td>SS199510-002-weight</td>
<td>SS199510-002</td>
<td>Weight</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/SPWGXX01" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/SPWGXX01</a></td>
<td>1477.7</td>
<td>Grams</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UGRM" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UGRM</a></td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="microbes-biomass-and-diversity" class="section level4 hasAnchor" number="4.10.0.9">
<h4><span class="header-section-number">4.10.0.9</span> Microbes biomass and diversity<a href="examples.html#microbes-biomass-and-diversity" class="anchor-section" aria-label="Anchor link to header"></a></h4>
(example coming soon)
<p>
 
</p>
</div>
<div id="phytoplankton-biomass-and-diversity" class="section level4 hasAnchor" number="4.10.0.10">
<h4><span class="header-section-number">4.10.0.10</span> Phytoplankton biomass and diversity<a href="examples.html#phytoplankton-biomass-and-diversity" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>This example deals with encoding phytoplankton observation data, including environmental data, into Darwin Core. Extracts from the actual data set <a href="https://obis.org/dataset/956d618f-91dc-4930-a253-cdf80ddb9371">LifeWatch observatory data: phytoplankton observations by imaging flow cytometry (FlowCam) in the Belgian Part of the North Sea</a>, are used as an example. Phytoplankton data should adhere to guidelines by Martin-Cabrera et al., 2022: <a href="http://dx.doi.org/10.25607/OBP-1742">Best practices and recommendations for plankton imaging data management: Ensuring effective data flow towards European data infrastructures. Version 1.</a>. Specific guidelines for the Occurrence and eMoF tables are documented in this publication, and we emphasize important fields in this example.</p>
<p><strong>Event core:</strong></p>
<p>The Event core contains events at the different levels and are linked together with <code>eventID</code> and <code>parentEventID</code>. In this example,the dataset contains records pointing to the origin, the in-situ sampling position as well as a record referring to the ex-situ collection of living specimens. In this case, the the event type information is provided in <code>type</code>. The recommended practice for providing the countryCode is to use an ISO 3166-1-alpha-2 country code. If additional information regarding licencing is provided, these can be populated under <code>rightsHolder</code> and <code>accessRights</code>. The remaining Event core fields provide location data including <code>datasetID</code> and <code>datasetName</code>, <code>locationID</code>, <code>waterBody</code>, <code>maximumDepthInMeters</code>, <code>minimumDepthInMeters</code>, <code>decimalLongitude</code>, <code>decimalLatitude</code>, <code>coordinateUncertaintyInMeters</code>, <code>geodeticDatum</code> and <code>footprintSRS</code>.</p>
<table style="width:100%;">
<colgroup>
<col width="7%" />
<col width="5%" />
<col width="3%" />
<col width="7%" />
<col width="5%" />
<col width="7%" />
<col width="10%" />
<col width="3%" />
<col width="3%" />
<col width="2%" />
<col width="3%" />
<col width="5%" />
<col width="6%" />
<col width="4%" />
<col width="4%" />
<col width="4%" />
<col width="8%" />
<col width="3%" />
</colgroup>
<thead>
<tr class="header">
<th>eventID</th>
<th>parentEventID</th>
<th>eventRemarks</th>
<th>eventDate</th>
<th>modified</th>
<th>datasetID</th>
<th>datasetName</th>
<th>locationID</th>
<th>waterBody</th>
<th>country</th>
<th>countryCode</th>
<th>minimumDepthInMeters</th>
<th>maximumDepthInMeters</th>
<th>decimalLatitude</th>
<th>decimalLongitude</th>
<th>geodeticDatum</th>
<th>coordinateUncertaintyInMeters</th>
<th>footprintSRS</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>TripNR3242</td>
<td></td>
<td>cruise</td>
<td>2017-05T13:18:00+00:00/2017-05T22:14:00+00:00</td>
<td>2021-10-21 15:52:00</td>
<td><a href="https://marineinfo.org/id/dataset/4688" class="uri">https://marineinfo.org/id/dataset/4688</a></td>
<td>LifeWatch observatory data: phytoplankton observations…</td>
<td></td>
<td>North Sea</td>
<td>Belgium</td>
<td>BE</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>EPSG:4326</td>
<td></td>
<td>EPSG:4326</td>
</tr>
<tr class="even">
<td>TripNR3242TripStationNR16781</td>
<td>TripNR3242</td>
<td>stationVisit</td>
<td>2017-05-08T20:44:00+00:00/2017-05-08T20:55:00+00:00</td>
<td>2021-10-21 15:52:00</td>
<td><a href="https://marineinfo.org/id/dataset/4688" class="uri">https://marineinfo.org/id/dataset/4688</a></td>
<td></td>
<td>JN17_5</td>
<td></td>
<td></td>
<td></td>
<td>0</td>
<td>30.22</td>
<td>51.0131</td>
<td>1.90562</td>
<td>EPSG:4326</td>
<td>1.11</td>
<td>EPSG:4326</td>
</tr>
<tr class="odd">
<td>TripNR3242TripStationNR16781MidasTripActionID105598</td>
<td>TripNR3242TripStationNR16781</td>
<td>sample</td>
<td>2017-05-08T20:50:00+00:00</td>
<td>2021-10-21 15:52:00</td>
<td><a href="https://marineinfo.org/id/dataset/4688" class="uri">https://marineinfo.org/id/dataset/4688</a></td>
<td></td>
<td>JN17_5</td>
<td></td>
<td></td>
<td></td>
<td>0</td>
<td>1</td>
<td>51.01203</td>
<td>1.90217</td>
<td>EPSG:4326</td>
<td>1.11</td>
<td>EPSG:4326</td>
</tr>
<tr class="even">
<td>TripNR3242TripStationNR16781MidasTripActionID105598UW</td>
<td>TripNR3242TripStationNR16781</td>
<td>sample</td>
<td>2017-05-08T20:50:00+00:00</td>
<td>2021-10-21 15:52:00</td>
<td><a href="https://marineinfo.org/id/dataset/4688" class="uri">https://marineinfo.org/id/dataset/4688</a></td>
<td></td>
<td>JN17_5</td>
<td></td>
<td></td>
<td></td>
<td>3</td>
<td>3</td>
<td>51.01203</td>
<td>1.90217</td>
<td>EPSG:4326</td>
<td>1.11</td>
<td>EPSG:4326</td>
</tr>
</tbody>
</table>
<p><strong>Occurrence extension:</strong></p>
<p>The Occurrence extension contains data of each occurrence with an <code>occurrenceID</code> and is linked to the Event core with the <code>eventID</code>. The Occurrence extension should provide information on the <code>basisOfRecord</code> and <code>occurrenceStatus</code>. Scientific names and links to the World Register of Marine Species should be provided under <code>scientificName</code> and <code>scientificNameID</code>, respectively. Recommended best practice is to always use the term of <code>MachineObservation</code> for imaging datasets derived from imaging instruments like this example, Additionally, the fields <code>identifiedBy</code> and <code>identificationVerificationStatus</code> are crucial to indicate whether the data has been validated, and by whom. While not shown in this example, <code>identificationReferences</code> and <code>associatedMedia</code> are also recommended to document the citation or algorithm software used in identification, and the persistent URL of annotated images, respectively. We know this dataset was validated because of information provided in the abstract and the eMoF table below, “identified and counted by image analysis and normalised to a unit volume of water body, validated by human”.</p>
<table style="width:100%;">
<colgroup>
<col width="23%" />
<col width="26%" />
<col width="4%" />
<col width="7%" />
<col width="7%" />
<col width="8%" />
<col width="10%" />
<col width="8%" />
<col width="2%" />
</colgroup>
<thead>
<tr class="header">
<th>eventID</th>
<th>occurrenceID</th>
<th>modified</th>
<th>basisOfRecord</th>
<th>occurrenceStatus</th>
<th>scientificNameID</th>
<th>scientificName</th>
<th>identifiedBy</th>
<th>identificationVerificationStatus</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>TripNR3242TripStationNR16781MidasTripActionID105598</td>
<td>TripNR3242TripStationNR16781MidasTripActionID105598occurenceIDTA_105598_(Pseudo-)pediastrum_5</td>
<td>2021-10-21</td>
<td>MachineObservation</td>
<td>absent</td>
<td><a href="urn:lsid:marinespecies.org:taxname:160560" class="uri">urn:lsid:marinespecies.org:taxname:160560</a></td>
<td>Hydrodictyaceae</td>
<td>Flanders Marine Institute</td>
<td>ValidatedByHuman</td>
</tr>
<tr class="even">
<td>TripNR3242TripStationNR16781MidasTripActionID105598</td>
<td>TripNR3242TripStationNR16781MidasTripActionID105598occurenceIDTA_105598_Actinoptychus senarius_5</td>
<td>2021-10-21</td>
<td>MachineObservation</td>
<td>present</td>
<td><a href="urn:lsid:marinespecies.org:taxname:148948" class="uri">urn:lsid:marinespecies.org:taxname:148948</a></td>
<td>Actinoptychus senarius</td>
<td>Flanders Marine Institute</td>
<td>ValidatedByHuman</td>
</tr>
<tr class="odd">
<td>TripNR3242TripStationNR16781MidasTripActionID105598</td>
<td>TripNR3242TripStationNR16781MidasTripActionID105598occurenceIDTA_105598_Actinoptychus splendens_5</td>
<td>2021-10-21</td>
<td>MachineObservation</td>
<td>present</td>
<td><a href="urn:lsid:marinespecies.org:taxname:148949" class="uri">urn:lsid:marinespecies.org:taxname:148949</a></td>
<td>Actinoptychus splendens</td>
<td>Flanders Marine Institute</td>
<td>ValidatedByHuman</td>
</tr>
<tr class="even">
<td>TripNR3242TripStationNR16781MidasTripActionID105598</td>
<td>TripNR3242TripStationNR16781MidasTripActionID105598occurenceIDTA_105598_Actinoptychus_5</td>
<td>2021-10-21</td>
<td>MachineObservation</td>
<td>present</td>
<td><a href="urn:lsid:marinespecies.org:taxname:148947" class="uri">urn:lsid:marinespecies.org:taxname:148947</a></td>
<td>Actinoptychus</td>
<td>Flanders Marine Institute</td>
<td>ValidatedByHuman</td>
</tr>
</tbody>
</table>
<p><strong>extendedMeasurementOrFact (eMoF) extension:</strong></p>
<p>The eMoF extension contains the environmental and measurement information and data of each occurrence. This extension is also linked to the Event core using the <code>eventID</code>, and linked to the Occurrence extension table using the <code>occurrenceID</code>. The various measurements are populated with <code>measurementID</code>, <code>measurementType</code>, <code>measurementTypeID</code>, <code>measurementUnit</code>, <code>measurementUnitID</code>, <code>measurementValue</code>, <code>measurementValueID</code>, <code>measurementAccuracy</code>, <code>measurementMethod</code>, <code>measurementDeterminedBy</code> and <code>measurementDeterminedDate</code>. In the example dataset, the LifeWatch observatory data was compiled using imaging flow cytometry (FlowCam) to observe and identify phytoplankton in the Belgian Part of the North Sea and recorded a number of measurements including abundance, lifestages, sampling device information as well as environmental measurements such as water temperature, salinity and conductivity with accompanying vocabulary.</p>
<table style="width:100%;">
<colgroup>
<col width="8%" />
<col width="12%" />
<col width="12%" />
<col width="6%" />
<col width="8%" />
<col width="11%" />
<col width="8%" />
<col width="6%" />
<col width="12%" />
<col width="10%" />
</colgroup>
<thead>
<tr class="header">
<th>id</th>
<th>occurrenceID</th>
<th>measurementType</th>
<th>measurementTypeID</th>
<th>measurementValue</th>
<th>measurementValueID</th>
<th>measurementUnit</th>
<th>measurementUnitID</th>
<th>measurementDeterminedBy</th>
<th>measurementMethod</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>TripNR3242</td>
<td></td>
<td>Platform Name</td>
<td><a href="http://vocab.nerc.ac.uk/collection/Q01/current/Q0100001/" class="uri">http://vocab.nerc.ac.uk/collection/Q01/current/Q0100001/</a></td>
<td>Simon Stevin</td>
<td><a href="http://vocab.nerc.ac.uk/collection/C17/current/11SS/" class="uri">http://vocab.nerc.ac.uk/collection/C17/current/11SS/</a></td>
<td></td>
<td></td>
<td></td>
<td>Flanders Marine Institute</td>
</tr>
<tr class="even">
<td>TripNR3242TripStationNR16781MidasTripActionID105598</td>
<td>TripNR3242TripStationNR16781MidasTripActionID105598occurenceIDTA_105598_Actinoptychus_5</td>
<td>Abundance of biological entity specified elsewhere per unit volume of the water body</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL01/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL01/</a></td>
<td>2.24</td>
<td></td>
<td>specimens/L</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UCPL" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UCPL</a></td>
<td>Flanders Marine Institute</td>
<td>identified and counted by image analysis and normalised to a unit volume of water body, validated by human</td>
</tr>
<tr class="odd">
<td>TripNR3242TripStationNR16781MidasTripActionID105598</td>
<td>TripNR3242TripStationNR16781MidasTripActionID105598occurenceIDTA_105598_Actinoptychus senarius_5</td>
<td>Abundance of biological entity specified elsewhere per unit volume of the water body</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL01/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL01/</a></td>
<td>1.12</td>
<td></td>
<td>specimens/L</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UCPL" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UCPL</a></td>
<td>Flanders Marine Institute</td>
<td>identified and counted by image analysis and normalised to a unit volume of water body, validated by human</td>
</tr>
<tr class="even">
<td>TripNR3242TripStationNR16781MidasTripActionID105598</td>
<td>TripNR3242TripStationNR16781MidasTripActionID105598occurenceIDTA_105598_(Pseudo-)pediastrum_5</td>
<td>Lifestage</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/LSTAGE01/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/LSTAGE01/</a></td>
<td>adult</td>
<td><a href="http://vocab.nerc.ac.uk/collection/S11/current/S1116/" class="uri">http://vocab.nerc.ac.uk/collection/S11/current/S1116/</a></td>
<td></td>
<td></td>
<td>Flanders Marine Institute</td>
<td>identified and counted by image analysis and normalised to a unit volume of water body, validated by human</td>
</tr>
<tr class="odd">
<td>TripNR3242TripStationNR16781MidasTripActionID105598</td>
<td>TripNR3242TripStationNR16781MidasTripActionID105598occurenceIDTA_105598_Actinoptychus senarius_5</td>
<td>Lifestage</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/LSTAGE01/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/LSTAGE01/</a></td>
<td>adult</td>
<td><a href="http://vocab.nerc.ac.uk/collection/S11/current/S1116/" class="uri">http://vocab.nerc.ac.uk/collection/S11/current/S1116/</a></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>TripNR3242TripStationNR16781MidasTripActionID105598</td>
<td></td>
<td>Sampling device aperture diameter</td>
<td><a href="http://vocab.nerc.ac.uk/collection/Q01/current/Q0100012/" class="uri">http://vocab.nerc.ac.uk/collection/Q01/current/Q0100012/</a></td>
<td>0.4</td>
<td></td>
<td>meter</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/ULAA/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/ULAA/</a></td>
<td>Flanders Marine Institute</td>
<td></td>
</tr>
<tr class="odd">
<td>TripNR3242TripStationNR16781MidasTripActionID105598</td>
<td></td>
<td>Sampling instrument name</td>
<td><a href="http://vocab.nerc.ac.uk/collection/Q01/current/Q0100002/" class="uri">http://vocab.nerc.ac.uk/collection/Q01/current/Q0100002/</a></td>
<td>Planktonnet Apstein</td>
<td><a href="http://vocab.nerc.ac.uk/collection/L22/current/TOOL0978/" class="uri">http://vocab.nerc.ac.uk/collection/L22/current/TOOL0978/</a></td>
<td></td>
<td></td>
<td>Flanders Marine Institute</td>
<td></td>
</tr>
<tr class="even">
<td>TripNR3242TripStationNR16781MidasTripActionID105598</td>
<td></td>
<td>Sampling net mesh size</td>
<td><a href="http://vocab.nerc.ac.uk/collection/Q01/current/Q0100015/" class="uri">http://vocab.nerc.ac.uk/collection/Q01/current/Q0100015/</a></td>
<td>55</td>
<td></td>
<td>micrometer</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UMIC/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UMIC/</a></td>
<td>Flanders Marine Institute</td>
<td></td>
</tr>
<tr class="odd">
<td>TripNR3242TripStationNR16781MidasTripActionID105598UW</td>
<td></td>
<td>Conductivity of the water body</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/CNDCZZ01/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/CNDCZZ01/</a></td>
<td>3.916</td>
<td></td>
<td>Siemens per metre</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UECA" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UECA</a></td>
<td>Flanders Marine Institute</td>
<td>Electrical conductivity of the water body by thermosalinograph, based on the UnderWaySystem of the ship</td>
</tr>
<tr class="even">
<td>TripNR3242TripStationNR16781MidasTripActionID105598UW</td>
<td></td>
<td>Practical salinity of the water body</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/PSALPR01/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/PSALPR01/</a></td>
<td>34.295</td>
<td></td>
<td>Grams per kilogram</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UGKG/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UGKG/</a></td>
<td>Flanders Marine Institute</td>
<td>Practical salinity of the water body based on water from the UnderWaySystem of the ship</td>
</tr>
<tr class="odd">
<td>TripNR3242TripStationNR16781MidasTripActionID105598UW</td>
<td></td>
<td>Temperature of the water body</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/TEMPPR01/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/TEMPPR01/</a></td>
<td>11.881</td>
<td></td>
<td>Degrees Celsius</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UPAA/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UPAA/</a></td>
<td>Flanders Marine Institute</td>
<td>Temperature of the water body based on water from the UnderWaySystem of the ship</td>
</tr>
</tbody>
</table>
</div>
<div id="seagrass-cover-and-composition" class="section level4 hasAnchor" number="4.10.0.11">
<h4><span class="header-section-number">4.10.0.11</span> Seagrass cover and composition<a href="examples.html#seagrass-cover-and-composition" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The structure of the Event, Occurrence and extendedMeasurementOrFact extensions for Seagrass Cover &amp; Composition is based on community feedback organised through the the Scientific Committee on Oceanic Research (SCOR): <a href="https://scor-int.org/group/158/">Coordinated Global Research Assessment of Seagrass System (C-GRASS)</a>. We acknowledge the work that the C-grass SCOR work group has done to develop a proposed scheme for completing Seagrass related extension files.</p>
<p>Here encode seagrass survey data into Darwin Core according to the ENV-DATA approach and using sections of the actual data set of <a href="http://ipt.iobis.org/caribbeanobis/resource?r=seagrasssurvey_colombia">Seagrass Monitoring at Chengue Bay, Colombia</a> as an example dataset.</p>
<p><strong>Event core:</strong></p>
<p>The Event core table is created by extracting all events and attributes. All events are linked together using <code>eventID</code> and <code>parentEventID</code>. <code>eventDate</code> is populated at the transect level with the recommended format that conforms to ISO 8601-1:2019. <code>habitat</code> is populated as a category or description of the habitat in which the event occurred. Additional <code>fieldNotes</code> can also be provided if applicable. The recommended best practice for <code>countryCode</code> is to use an ISO 3166-1-alpha-2 country code. The remaining Event core fields comprise of location data including <code>maximumDepthInMeters</code>, <code>minimumDepthInMeters</code>, <code>decimalLongitude</code>, <code>decimalLatitude</code>, <code>coordinateUncertaintyInMeters</code>, <code>footprintWKT</code> and <code>footprintSRS</code>. Addtionally in the Event core, it is recommended to further include information regarding <code>license</code>, <code>rightsHolder</code>, <code>bibliographicCitation</code>, <code>institutionID</code>, <code>datasetID</code>, <code>institutionCode</code> and <code>datasetName</code>.</p>
<table>
<colgroup>
<col width="13%" />
<col width="8%" />
<col width="3%" />
<col width="3%" />
<col width="3%" />
<col width="4%" />
<col width="5%" />
<col width="6%" />
<col width="5%" />
<col width="5%" />
<col width="9%" />
<col width="25%" />
<col width="4%" />
</colgroup>
<thead>
<tr class="header">
<th>eventID</th>
<th>parentEventID</th>
<th>eventDate</th>
<th>habitat</th>
<th>fieldNotes</th>
<th>countryCode</th>
<th>minimumDepthInMeters</th>
<th>maximumDepthInMeters</th>
<th>decimalLatitude</th>
<th>decimalLongitude</th>
<th>coordinateUncertaintyInMeters</th>
<th>footprintWKT</th>
<th>footprintSRS</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>USBsg-chengue-pastocoral</td>
<td></td>
<td>2019-05-13</td>
<td>seagrass</td>
<td>no notes</td>
<td>CO</td>
<td>0.8</td>
<td>2</td>
<td>11.32021806</td>
<td>-74.12753684</td>
<td>10</td>
<td>POLYGON ((-74.1273259763024 11.320475512862,-74.1272978004008 11.3201655779439))</td>
<td>EPSG:4326</td>
</tr>
<tr class="even">
<td>USBsg-chengue-pastomanglar</td>
<td></td>
<td>2019-05-14</td>
<td>seagrass</td>
<td>no notes</td>
<td>CO</td>
<td>0.8</td>
<td>0.8</td>
<td>11.31977189</td>
<td>-74.12536879</td>
<td>10</td>
<td>POLYGON ((-74.1253370891273 11.3195001294432,-74.1253337743154 11.3194968146313))</td>
<td>EPSG:4326</td>
</tr>
<tr class="odd">
<td>USBsg-chengue-pastocoral-SquidPopTransect1</td>
<td>USBsg-chengue-pastocoral</td>
<td>2019-05-13</td>
<td>seagrass</td>
<td>no notes</td>
<td>CO</td>
<td>0.8</td>
<td>2</td>
<td>11.32039927</td>
<td>-74.12737404</td>
<td>50</td>
<td>POINT (-74.1273740410759 11.3203992721869)</td>
<td>EPSG:4326</td>
</tr>
<tr class="even">
<td>USBsg-chengue-pastocoral-SquidPopTransect2</td>
<td>USBsg-chengue-pastocoral</td>
<td>2019-05-13</td>
<td>seagrass</td>
<td>no notes</td>
<td>CO</td>
<td>0.8</td>
<td>2</td>
<td>11.32027662</td>
<td>-74.1273989</td>
<td>50</td>
<td>POINT (-74.1273989021655 11.3202766241445)</td>
<td>EPSG:4326</td>
</tr>
</tbody>
</table>
<p><strong>Occurrence extension:</strong></p>
<p>The Occurrence extension table contain data for each occurrence with an <code>occurrenceID</code> and is linked to the Event core with the <code>eventID</code>. This table should provide information on the <code>basisOfRecord</code> and <code>occurrenceStatus</code>. Scientific names and links to the World Register of Marine Species should be provided under <code>scientificName</code> and <code>scientificNameID</code>, respectively. If a species was identified by an expert, the field <code>identifiedBy</code> can be populated. If the species is well-known by another common name, this name can be provided under <code>vernacularName</code>.</p>
<table>
<colgroup>
<col width="24%" />
<col width="19%" />
<col width="9%" />
<col width="9%" />
<col width="23%" />
<col width="13%" />
</colgroup>
<thead>
<tr class="header">
<th>eventID</th>
<th>occurrenceID</th>
<th>basisOfRecord</th>
<th>occurrenceStatus</th>
<th>scientificNameID</th>
<th>scientificName</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>USBsg-chengue-pastocoral</td>
<td>USBsg-chengue-pastocoral-tt</td>
<td>HumanObservation</td>
<td>present</td>
<td><a href="urn:lsid:marinespecies.org:taxname:374720" class="uri">urn:lsid:marinespecies.org:taxname:374720</a></td>
<td>Thalassia testudinum</td>
</tr>
<tr class="even">
<td>USBsg-chengue-pastomanglar</td>
<td>USBsg-chengue-manglar-tt</td>
<td>HumanObservation</td>
<td>present</td>
<td><a href="urn:lsid:marinespecies.org:taxname:374720" class="uri">urn:lsid:marinespecies.org:taxname:374720</a></td>
<td>Thalassia testudinum</td>
</tr>
<tr class="odd">
<td>USBsg-chengue-pastocoral-SquidPopTransect1</td>
<td>USBsg-chengue-pastocoral-fish-001</td>
<td>HumanObservation</td>
<td>present</td>
<td><a href="urn:lsid:marinespecies.org:taxname:158815" class="uri">urn:lsid:marinespecies.org:taxname:158815</a></td>
<td>Halichoeres bivittatus</td>
</tr>
<tr class="even">
<td>USBsg-chengue-pastocoral-SquidPopTransect1</td>
<td>USBsg-chengue-pastocoral-fish-002</td>
<td>HumanObservation</td>
<td>present</td>
<td><a href="urn:lsid:marinespecies.org:taxname:158932" class="uri">urn:lsid:marinespecies.org:taxname:158932</a></td>
<td>Lactophrys triqueter</td>
</tr>
</tbody>
</table>
<p><strong>extendedMeasurementOrFact (eMoF) extension:</strong></p>
<p>The eMoF table contains the measurement information and data of each occurrence. This extension is also linked to the Event core using the <code>eventID</code>, and linked to the Occurrence table using the <code>occurrenceID</code>. The various measurements are populated with <code>measurementType</code>, <code>measurementTypeID</code>, <code>measurementUnit</code>, <code>measurementUnitID</code>, <code>measurementValue</code>, <code>measurementValueID</code>, <code>measurementAccuracy</code>, <code>measurementMethod</code>, <code>measurementDeterminedBy</code> and <code>measurementDeterminedDate</code>. The example dataset of <a href="http://ipt.iobis.org/caribbeanobis/resource?r=seagrasssurvey_colombia">Seagrass Monitoring at Chengue Bay, Colombia</a> recorded a number of measurements and can be used as an example of how to populate the respective fields:</p>
<table style="width:100%;">
<colgroup>
<col width="8%" />
<col width="9%" />
<col width="15%" />
<col width="13%" />
<col width="19%" />
<col width="6%" />
<col width="8%" />
<col width="18%" />
</colgroup>
<thead>
<tr class="header">
<th>eventID</th>
<th>occurrenceID</th>
<th>measurementID</th>
<th>measurementType</th>
<th>measurementTypeID</th>
<th>measurementValue</th>
<th>measurementUnit</th>
<th>measurementUnitID</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>USBsg-chengue-pastocoral</td>
<td>USBsg-chengue-pastocoral-tt</td>
<td>USBsg-chengue-pastocoral-PhyQ01</td>
<td>WaterTemp</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/TEMPMP01/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/TEMPMP01/</a></td>
<td>29.23</td>
<td>Degrees Celsius</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UPAA/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UPAA/</a></td>
</tr>
<tr class="even">
<td>USBsg-chengue-pastocoral</td>
<td>USBsg-chengue-pastocoral-tt</td>
<td>USBsg-chengue-pastocoral-PhyQ02</td>
<td>Salinity</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/SSALSL01/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/SSALSL01/</a></td>
<td>36</td>
<td>Parts per thousand</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UPPT/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UPPT/</a></td>
</tr>
<tr class="odd">
<td>USBsg-chengue-pastocoral</td>
<td>USBsg-chengue-pastocoral-tt</td>
<td>USBsg-chengue-pastocoral-PhyQ03</td>
<td>Dissolved oxygen</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/DOXYSE02/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/DOXYSE02/</a></td>
<td>6.58</td>
<td>Milligrams per litre</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UMGL/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UMGL/</a></td>
</tr>
<tr class="even">
<td>USBsg-chengue-pastocoral</td>
<td>USBsg-chengue-pastocoral-tt</td>
<td>USBsg-chengue-pastocoral-T1C1-shoot-01</td>
<td>Shoot Density</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL02/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL02/</a></td>
<td>128</td>
<td>Number per square metre</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UPMS/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UPMS/</a></td>
</tr>
<tr class="odd">
<td>USBsg-chengue-pastocoral</td>
<td>USBsg-chengue-pastocoral-tt</td>
<td>USBsg-chengue-pastocoral-T1C1-leafLenght-01</td>
<td>Leaf Length</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/OBSMAXLX/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/OBSMAXLX/</a></td>
<td>18</td>
<td>Centimetres</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/ULCM/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/ULCM/</a></td>
</tr>
<tr class="even">
<td>USBsg-chengue-pastocoral</td>
<td>USBsg-chengue-pastocoral-tt</td>
<td>USBsg-chengue-pastocoral-T1N1-DryBiomass</td>
<td>Total Dry Biomass</td>
<td><a href="http://vocab.nerc.ac.uk/collection/S06/current/S0600087/" class="uri">http://vocab.nerc.ac.uk/collection/S06/current/S0600087/</a></td>
<td>0.32055</td>
<td>Grams per square metre</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UGMS/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UGMS/</a></td>
</tr>
<tr class="odd">
<td>USBsg-chengue-pastocoral</td>
<td>USBsg-chengue-pastocoral-tt</td>
<td>USBsg-chengue-pastocoral-T1N1-biomassGL</td>
<td>Dry biomass of green leaves</td>
<td><a href="http://vocab.nerc.ac.uk/collection/S06/current/S0600087/" class="uri">http://vocab.nerc.ac.uk/collection/S06/current/S0600087/</a></td>
<td>0.05575</td>
<td>Grams per square metre</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UGMS/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UGMS/</a></td>
</tr>
<tr class="even">
<td>USBsg-chengue-pastocoral</td>
<td>USBsg-chengue-pastocoral-tt</td>
<td>USBsg-chengue-pastocoral-T1N1-biomassNGL</td>
<td>Dry biomass of non green leaves</td>
<td><a href="http://vocab.nerc.ac.uk/collection/S06/current/S0600087/" class="uri">http://vocab.nerc.ac.uk/collection/S06/current/S0600087/</a></td>
<td>0.1469</td>
<td>Grams per square metre</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UGMS/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UGMS/</a></td>
</tr>
<tr class="odd">
<td>USBsg-chengue-pastocoral</td>
<td>USBsg-chengue-pastocoral-tt</td>
<td>USBsg-chengue-pastocoral-T1N1-biomassSH</td>
<td>Dry biomass of the shoots</td>
<td><a href="http://vocab.nerc.ac.uk/collection/S06/current/S0600087/" class="uri">http://vocab.nerc.ac.uk/collection/S06/current/S0600087/</a></td>
<td>0.07625</td>
<td>Grams per square metre</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UGMS/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UGMS/</a></td>
</tr>
<tr class="even">
<td>USBsg-chengue-pastocoral</td>
<td>USBsg-chengue-pastocoral-tt</td>
<td>USBsg-chengue-pastocoral-T1N2-biomassR</td>
<td>Dry biomass of the roots</td>
<td><a href="http://vocab.nerc.ac.uk/collection/S06/current/S0600087/" class="uri">http://vocab.nerc.ac.uk/collection/S06/current/S0600087/</a></td>
<td>0.0385</td>
<td>Grams per square metre</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UGMS/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UGMS/</a></td>
</tr>
<tr class="odd">
<td>USBsg-chengue-pastocoral</td>
<td>USBsg-chengue-pastocoral-tt</td>
<td>USBsg-chengue-pastocoral-T1N2-biomassRIZ</td>
<td>Dry biomass of the rizome</td>
<td><a href="http://vocab.nerc.ac.uk/collection/S06/current/S0600087/" class="uri">http://vocab.nerc.ac.uk/collection/S06/current/S0600087/</a></td>
<td>0.02725</td>
<td>Grams per square metre</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UGMS/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UGMS/</a></td>
</tr>
<tr class="even">
<td>USBsg-chengue-pastocoral</td>
<td>USBsg-chengue-pastocoral-tt</td>
<td>USBsg-chengue-pastocoral-T1N2-biomassOTH</td>
<td>Dry biomass of other seagrass species</td>
<td><a href="http://vocab.nerc.ac.uk/collection/S06/current/S0600087/" class="uri">http://vocab.nerc.ac.uk/collection/S06/current/S0600087/</a></td>
<td>0</td>
<td>Grams per square metre</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UGMS/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UGMS/</a></td>
</tr>
</tbody>
</table>
</div>
<div id="zooplankton-biomass-and-diversity" class="section level4 hasAnchor" number="4.10.0.12">
<h4><span class="header-section-number">4.10.0.12</span> Zooplankton biomass and diversity<a href="examples.html#zooplankton-biomass-and-diversity" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Here we will encode zooplankton observation and environmental data into Darwin Core. Extracts from the actual dataset <a href="https://obis.org/dataset/afa5b0e8-826d-4433-b698-beb176ef7880">LifeWatch observatory data: zooplankton observations by imaging (ZooScan) in the Belgian Part of the North Sea</a>, are used as an example. As with the phytoplankton example, zooplankton data should also adhere to guidelines by Martin-Cabrera et al., 2022: <a href="http://dx.doi.org/10.25607/OBP-1742">Best practices and recommendations for plankton imaging data management: Ensuring effective data flow towards European data infrastructures. Version 1.</a>. These Best Practices indicate that for imaging data like this example, the fields <code>identificationVerificationStatus</code> and <code>identifiedBy</code> are crucial to know whether the data has been validated, and by whom. <code>identificationReferences</code> is used to document the citation or algorithm software used in identification. When possible, <code>associatedMedia</code> should also be populated with the persistent URL of annotated images.</p>
<p><strong>Event core:</strong></p>
<p>The Event core contains events at the different levels and are linked together with <code>eventID</code> and <code>parentEventID</code>. In this example,the dataset contains records pointing to the origin, the in-situ sampling position as well as a record referring to the ex-situ collection of living specimens. In this case, the the event type information is provided in <code>type</code>. The recommended practice for providing the countryCode is to use an ISO 3166-1-alpha-2 country code. If additional information regarding licencing is provided, these can be populated under <code>rightsHolder</code> and <code>accessRights</code>. The remaining Event core fields provide location data including <code>datasetID</code> and <code>datasetName</code>, <code>locationID</code>, <code>waterBody</code>, <code>maximumDepthInMeters</code>, <code>minimumDepthInMeters</code>, <code>decimalLongitude</code>, <code>decimalLatitude</code>, <code>coordinateUncertaintyInMeters</code>, <code>geodeticDatum</code> and <code>footprintSRS</code>.</p>
<table>
<colgroup>
<col width="8%" />
<col width="7%" />
<col width="5%" />
<col width="8%" />
<col width="8%" />
<col width="6%" />
<col width="7%" />
<col width="4%" />
<col width="8%" />
<col width="3%" />
<col width="4%" />
<col width="5%" />
<col width="6%" />
<col width="4%" />
<col width="6%" />
<col width="3%" />
</colgroup>
<thead>
<tr class="header">
<th>eventID</th>
<th>parentEventID</th>
<th>eventRemarks</th>
<th>eventDate</th>
<th>modified</th>
<th>datasetID</th>
<th>datasetName</th>
<th>locationID</th>
<th>waterBody</th>
<th>country</th>
<th>minimumDepthInMeters</th>
<th>maximumDepthInMeters</th>
<th>decimalLatitude</th>
<th>decimalLongitude</th>
<th>geodeticDatum</th>
<th>footprintSRS</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>TripNR2547</td>
<td></td>
<td>cruise</td>
<td>2013-07-22T06:58:00+00:00/2013-07-22T16:58:00+00:00</td>
<td>2021-06-23 14:54:00</td>
<td><a href="https://marineinfo.org/id/dataset/4687" class="uri">https://marineinfo.org/id/dataset/4687</a></td>
<td>LifeWatch observatory data: zooplankton observations…</td>
<td></td>
<td>Belgian Part of the North Sea</td>
<td>Belgium</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>EPSG:4326</td>
<td>EPSG:4326</td>
</tr>
<tr class="even">
<td>TripNR2547TripStationNR9781</td>
<td>TripNR2547</td>
<td>stationVisit</td>
<td>2013-07-22T07:13:00+00:00/2013-07-22T07:26:00+00:00</td>
<td>2021-06-23 14:54:00</td>
<td><a href="https://marineinfo.org/id/dataset/4687" class="uri">https://marineinfo.org/id/dataset/4687</a></td>
<td>LifeWatch observatory data: zooplankton observations…</td>
<td>130</td>
<td></td>
<td></td>
<td>0</td>
<td>13.4</td>
<td>51.27083333</td>
<td>2.905</td>
<td>EPSG:4326</td>
<td>EPSG:4326</td>
</tr>
<tr class="odd">
<td>TripNR2547TripStationNR9781MidasTripActionID23024</td>
<td>TripNR2547TripStationNR9781</td>
<td>sample</td>
<td>2013-07-22T07:22:00+00:00</td>
<td>2021-06-23 14:54:00</td>
<td><a href="https://marineinfo.org/id/dataset/4687" class="uri">https://marineinfo.org/id/dataset/4687</a></td>
<td>LifeWatch observatory data: zooplankton observations…</td>
<td>130</td>
<td></td>
<td></td>
<td>0</td>
<td>0</td>
<td>51.2687318</td>
<td>2.901797</td>
<td>EPSG:4326</td>
<td>EPSG:4326</td>
</tr>
<tr class="even">
<td>TripNR2547TripStationNR9781MidasTripActionID23024UW</td>
<td>TripNR2547TripStationNR9781</td>
<td>sample</td>
<td>2013-07-22T07:22:00+00:00</td>
<td>2021-06-23 14:54:00</td>
<td><a href="https://marineinfo.org/id/dataset/4687" class="uri">https://marineinfo.org/id/dataset/4687</a></td>
<td>LifeWatch observatory data: zooplankton observations…</td>
<td>130</td>
<td></td>
<td></td>
<td>3</td>
<td>3</td>
<td>51.2687318</td>
<td>2.901797</td>
<td>EPSG:4326</td>
<td>EPSG:4326</td>
</tr>
</tbody>
</table>
<p><strong>Occurrence extension:</strong></p>
<p>The Occurrence extension contains data of each occurrence with an <code>occurrenceID</code> and is linked to the Event core with the <code>eventID</code>. The Occurrence extension should provide information on the <code>basisOfRecord</code> and <code>occurrenceStatus</code>. Scientific names and links to the World Register of Marine Species should be provided under <code>scientificName</code> and <code>scientificNameID</code>, respectively.</p>
<table>
<colgroup>
<col width="15%" />
<col width="13%" />
<col width="6%" />
<col width="10%" />
<col width="12%" />
<col width="9%" />
<col width="7%" />
<col width="7%" />
<col width="7%" />
<col width="7%" />
</colgroup>
<thead>
<tr class="header">
<th>eventID</th>
<th>occurrenceID</th>
<th>modified</th>
<th>basisOfRecord</th>
<th>occurrenceStatus</th>
<th>scientificNameID</th>
<th>scientificName</th>
<th>identifiedBy</th>
<th>identificationVerificationStatus</th>
<th>identificationReferences</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>TripNR2547TripStationNR9781MidasTripActionID23024</td>
<td>TripNR2547TripStationNR9781MidasTripActionID23024occurenceIDTA23024_Amphipoda_sub2_130</td>
<td>2021-06-22</td>
<td>MachineObservation</td>
<td>absent</td>
<td><a href="urn:lsid:marinespecies.org:taxname:1135" class="uri">urn:lsid:marinespecies.org:taxname:1135</a></td>
<td>Amphipoda</td>
<td>Jonas Mortelmans</td>
<td>ValidatedByHuman</td>
<td>RandomForest</td>
</tr>
<tr class="even">
<td>TripNR2547TripStationNR9781MidasTripActionID23024</td>
<td>TripNR2547TripStationNR9781MidasTripActionID23024occurenceIDTA23024_Annelida_sub2_130</td>
<td>2021-06-22</td>
<td>MachineObservation</td>
<td>present</td>
<td><a href="urn:lsid:marinespecies.org:taxname:882" class="uri">urn:lsid:marinespecies.org:taxname:882</a></td>
<td>Annelida</td>
<td>Jonas Mortelmans</td>
<td>ValidatedByHuman</td>
<td>RandomForest</td>
</tr>
<tr class="odd">
<td>TripNR2547TripStationNR9781MidasTripActionID23024</td>
<td>TripNR2547TripStationNR9781MidasTripActionID23024occurenceIDTA23024_Anomura_sub2_130</td>
<td>2021-06-22</td>
<td>MachineObservation</td>
<td>absent</td>
<td><a href="urn:lsid:marinespecies.org:taxname:106671" class="uri">urn:lsid:marinespecies.org:taxname:106671</a></td>
<td>Anomura</td>
<td>Jonas Mortelmans</td>
<td>ValidatedByHuman</td>
<td>RandomForest</td>
</tr>
<tr class="even">
<td>TripNR2547TripStationNR9781MidasTripActionID23024</td>
<td>TripNR2547TripStationNR9781MidasTripActionID23024occurenceIDTA23024_Appendicularia_sub2_130</td>
<td>2021-06-22</td>
<td>MachineObservation</td>
<td>absent</td>
<td><a href="urn:lsid:marinespecies.org:taxname:146421" class="uri">urn:lsid:marinespecies.org:taxname:146421</a></td>
<td>Appendicularia</td>
<td>Jonas Mortelmans</td>
<td>ValidatedByHuman</td>
<td>RandomForest</td>
</tr>
</tbody>
</table>
<p><strong>extendedMeasurementOrFact (eMoF) extension:</strong></p>
<p>The eMoF extension table contains the measurement information and data of each occurrence. This extension is also linked to the Event core using the <code>eventID</code>, and linked to the Occurrence table using the <code>occurrenceID</code>. The various measurements are populated with <code>measurementType</code>, <code>measurementTypeID</code>, <code>measurementUnit</code>, <code>measurementUnitID</code>, <code>measurementValue</code>, <code>measurementValueID</code>, <code>measurementAccuracy</code>, <code>measurementMethod</code>, <code>measurementDeterminedBy</code> and <code>measurementDeterminedDate</code>. The example dataset of <a href="https://obis.org/dataset/afa5b0e8-826d-4433-b698-beb176ef7880">LifeWatch observatory data: zooplankton observations by imaging (ZooScan) in the Belgian Part of the North Sea</a> recorded some ENV-DATA and organism measurements the can be used as an example of how to populate the respective fields, including conductivity of the water body; concentration of chlorophyll-a per unit volume of the water body; sampling instrument name; sampling net mesh size; lifestage of the organism observed; and abundance of the organism observed.</p>
<table>
<colgroup>
<col width="11%" />
<col width="7%" />
<col width="8%" />
<col width="9%" />
<col width="7%" />
<col width="16%" />
<col width="9%" />
<col width="10%" />
<col width="11%" />
<col width="7%" />
</colgroup>
<thead>
<tr class="header">
<th>id</th>
<th>occurrenceID</th>
<th>measurementType</th>
<th>measurementTypeID</th>
<th>measurementValue</th>
<th>measurementValueID</th>
<th>measurementUnit</th>
<th>measurementUnitID</th>
<th>measurementDeterminedBy</th>
<th>measurementMethod</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>TripNR3256TripStationNR17157MidasTripActionID106326</td>
<td></td>
<td>Sampling instrument name</td>
<td><a href="http://vocab.nerc.ac.uk/collection/Q01/current/Q0100002/" class="uri">http://vocab.nerc.ac.uk/collection/Q01/current/Q0100002/</a></td>
<td>Planktonnet WP2</td>
<td>&lt;<a href="http://vocab.nerc.ac.uk/collection/L22/current/TOOL0979/" class="uri">http://vocab.nerc.ac.uk/collection/L22/current/TOOL0979/</a></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>TripNR3256TripStationNR17157MidasTripActionID106326</td>
<td></td>
<td>Sampling net mesh size</td>
<td><a href="http://vocab.nerc.ac.uk/collection/Q01/current/Q0100015/" class="uri">http://vocab.nerc.ac.uk/collection/Q01/current/Q0100015/</a></td>
<td>200</td>
<td></td>
<td>micrometer</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UMIC/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UMIC/</a></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>TripNR3529TripStationNR19242MidasTripActionID109631UW</td>
<td></td>
<td>Conductivity of the water body</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/CNDCZZ01/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/CNDCZZ01/</a></td>
<td>4.05</td>
<td></td>
<td>Siemens per metre</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UECA/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UECA/</a></td>
<td>Flanders Marine Institute</td>
<td>Electrical conductivity of the water body by thermosalinograph, based on the UnderWaySystem of the ship</td>
</tr>
<tr class="even">
<td>TripNR3529TripStationNR19243MidasTripActionID109634</td>
<td></td>
<td>Concentration of chlorophyll-a per unit volume of the water body</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/CPHLHPP1/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/CPHLHPP1/</a></td>
<td>1.42</td>
<td></td>
<td>Micrograms per litre</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UGPL/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UGPL/</a></td>
<td>Flanders Marine Institute</td>
<td>Concentration of chlorophyll-a per unit volume of the water body [particulate &gt;GF/F phase] by filtration, acetone extraction and high performance liquid chromatography (HPLC)</td>
</tr>
<tr class="odd">
<td>TripNR2547TripStationNR9781MidasTripActionID23024</td>
<td>TripNR2547TripStationNR9781MidasTripActionID23024occurenceIDTA23024_Annelida_sub2_130</td>
<td>Lifestage</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/LSTAGE01/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/LSTAGE01/</a></td>
<td>unspecified</td>
<td><a href="http://vocab.nerc.ac.uk/collection/S11/current/S1152/" class="uri">http://vocab.nerc.ac.uk/collection/S11/current/S1152/</a></td>
<td></td>
<td></td>
<td>Flanders Marine Institute</td>
<td>identified and counted by image analysis and normalised to a unit volume of water body, validated by human</td>
</tr>
<tr class="even">
<td>TripNR2547TripStationNR9781MidasTripActionID23024</td>
<td>TripNR2547TripStationNR9781MidasTripActionID23024occurenceIDTA23024_Annelida_sub2_130</td>
<td>Abundance of biological entity specified elsewhere per unit volume of the water body</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL01/" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL01/</a></td>
<td>0.50</td>
<td></td>
<td>specimens/m³</td>
<td><a href="http://vocab.nerc.ac.uk/collection/P06/current/UPMM/" class="uri">http://vocab.nerc.ac.uk/collection/P06/current/UPMM/</a></td>
<td>Flanders Marine Insitute</td>
<td>identified and counted by image analysis and normalised to a unit volume of water body, validated by human</td>
</tr>
</tbody>
</table>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="common_formatissues.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="other_data_types.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["manual.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "none"
}
});
});
</script>

</body>

</html>
