<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>1.5 Frequently Asked Questions | The OBIS manual</title>
  <meta name="description" content="1.5 Frequently Asked Questions | The OBIS manual" />
  <meta name="generator" content="bookdown 0.40 and GitBook 2.6.7" />

  <meta property="og:title" content="1.5 Frequently Asked Questions | The OBIS manual" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="1.5 Frequently Asked Questions | The OBIS manual" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="gethelp.html"/>
<link rel="next" href="linklist.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-JBW0JFMNBJ"></script>
<script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'G-JBW0JFMNBJ');
</script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="/">The OBIS manual</a></li>
<li><a href="https://obis.org/">Return to OBIS.org</a></li>

<li class="divider"></li>
<li class="part"><span><b>Overview</b></span></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#guidelines-on-the-sharing-and-use-of-data-in-obis"><i class="fa fa-check"></i><b>1.1</b> Guidelines on the sharing and use of data in OBIS</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i><b>1.2</b> Acknowledgements</a></li>
<li class="chapter" data-level="1.3" data-path="policy.html"><a href="policy.html"><i class="fa fa-check"></i><b>1.3</b> Data Policy</a></li>
<li class="chapter" data-level="1.4" data-path="gethelp.html"><a href="gethelp.html"><i class="fa fa-check"></i><b>1.4</b> Getting Help in OBIS</a></li>
<li class="chapter" data-level="1.5" data-path="FAQ.html"><a href="FAQ.html"><i class="fa fa-check"></i><b>1.5</b> Frequently Asked Questions</a></li>
<li class="chapter" data-level="1.6" data-path="linklist.html"><a href="linklist.html"><i class="fa fa-check"></i><b>1.6</b> Quick links to resources</a></li>
</ul></li>
<li class="part"><span><b>Contributing data to OBIS</b></span></li>
<li class="chapter" data-level="2" data-path="contribute.html"><a href="contribute.html"><i class="fa fa-check"></i><b>2</b> What can you contribute and how?</a>
<ul>
<li class="chapter" data-level="2.1" data-path="contribute.html"><a href="contribute.html#why-publish-data-to-obis"><i class="fa fa-check"></i><b>2.1</b> Why publish data to OBIS</a></li>
<li class="chapter" data-level="2.2" data-path="contribute.html"><a href="contribute.html#how-to-handle-sensitive-data"><i class="fa fa-check"></i><b>2.2</b> How to handle sensitive data</a></li>
<li class="chapter" data-level="2.3" data-path="data_standards.html"><a href="data_standards.html"><i class="fa fa-check"></i><b>2.3</b> OBIS Data Life Cycle</a></li>
<li class="chapter" data-level="2.4" data-path="data_standards.html"><a href="data_standards.html#biodiversity-data-standards"><i class="fa fa-check"></i><b>2.4</b> Biodiversity data standards</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="darwin_core.html"><a href="darwin_core.html"><i class="fa fa-check"></i><b>2.4.1</b> Darwin Core Terms &amp; Guidelines</a></li>
<li class="chapter" data-level="2.4.2" data-path="data_format.html"><a href="data_format.html"><i class="fa fa-check"></i><b>2.4.2</b> Darwin Core Archive</a></li>
<li class="chapter" data-level="2.4.3" data-path="relational_db.html"><a href="relational_db.html"><i class="fa fa-check"></i><b>2.4.3</b> Relational databases: the underlying framework of OBIS</a></li>
<li class="chapter" data-level="2.4.4" data-path="eml.html"><a href="eml.html"><i class="fa fa-check"></i><b>2.4.4</b> Ecological Metadata Language</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="nodes.html"><a href="nodes.html"><i class="fa fa-check"></i><b>3</b> OBIS Terms of Reference: Nodes, Coordination Groups, Secretariat</a>
<ul>
<li class="chapter" data-level="3.1" data-path="nodes.html"><a href="nodes.html#tor-of-obis-nodes"><i class="fa fa-check"></i><b>3.1</b> TOR of OBIS nodes</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="nodes.html"><a href="nodes.html#how-to-become-an-obis-node"><i class="fa fa-check"></i><b>3.1.1</b> How to become an OBIS node</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="nodes.html"><a href="nodes.html#tor-of-obis-steering-group"><i class="fa fa-check"></i><b>3.2</b> TOR of OBIS Steering Group</a></li>
<li class="chapter" data-level="3.3" data-path="nodes.html"><a href="nodes.html#tor-of-obis-co-chairs"><i class="fa fa-check"></i><b>3.3</b> TOR of OBIS Co-Chairs</a></li>
<li class="chapter" data-level="3.4" data-path="nodes.html"><a href="nodes.html#tor-of-obis-secretariat"><i class="fa fa-check"></i><b>3.4</b> TOR of OBIS Secretariat</a></li>
<li class="chapter" data-level="3.5" data-path="nodes.html"><a href="nodes.html#tor-of-obis-coordination-groups"><i class="fa fa-check"></i><b>3.5</b> TOR of OBIS Coordination Groups</a></li>
<li class="chapter" data-level="3.6" data-path="nodes.html"><a href="nodes.html#tor-of-obis-executive-committee"><i class="fa fa-check"></i><b>3.6</b> TOR of OBIS Executive Committee</a></li>
</ul></li>
<li class="part"><span><b>Data Formatting</b></span></li>
<li class="chapter" data-level="4" data-path="formatting.html"><a href="formatting.html"><i class="fa fa-check"></i><b>4</b> Dataset structure</a>
<ul>
<li class="chapter" data-level="4.1" data-path="formatting.html"><a href="formatting.html#when-to-use-event-core"><i class="fa fa-check"></i><b>4.1</b> When to use Event Core</a></li>
<li class="chapter" data-level="4.2" data-path="formatting.html"><a href="formatting.html#when-to-use-occurrence-core"><i class="fa fa-check"></i><b>4.2</b> When to use Occurrence Core</a></li>
<li class="chapter" data-level="4.3" data-path="formatting.html"><a href="formatting.html#extensions-in-obis"><i class="fa fa-check"></i><b>4.3</b> Extensions in OBIS</a></li>
<li class="chapter" data-level="4.4" data-path="formatting.html"><a href="formatting.html#data-formatting-tools"><i class="fa fa-check"></i><b>4.4</b> Data formatting tools</a></li>
<li class="chapter" data-level="4.5" data-path="identifiers.html"><a href="identifiers.html"><i class="fa fa-check"></i><b>4.5</b> Constructing and using identifier codes</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="identifiers.html"><a href="identifiers.html#eventid"><i class="fa fa-check"></i><b>4.5.1</b> eventID</a></li>
<li class="chapter" data-level="4.5.2" data-path="identifiers.html"><a href="identifiers.html#occurrenceid"><i class="fa fa-check"></i><b>4.5.2</b> occurrenceID</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="checklist.html"><a href="checklist.html"><i class="fa fa-check"></i><b>5</b> Formatting data tables</a>
<ul>
<li class="chapter" data-level="5.1" data-path="checklist.html"><a href="checklist.html#darwin-core-term-checklist-for-obis"><i class="fa fa-check"></i><b>5.1</b> Darwin Core Term Checklist for OBIS</a></li>
<li class="chapter" data-level="5.2" data-path="name_matching.html"><a href="name_matching.html"><i class="fa fa-check"></i><b>5.2</b> Name matching for taxonomic quality control</a></li>
<li class="chapter" data-level="5.3" data-path="format_occurrence.html"><a href="format_occurrence.html"><i class="fa fa-check"></i><b>5.3</b> How to format Occurrence tables</a></li>
<li class="chapter" data-level="5.4" data-path="format_event.html"><a href="format_event.html"><i class="fa fa-check"></i><b>5.4</b> How to format Event tables</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="format_event.html"><a href="format_event.html#populating-parent-and-child-events"><i class="fa fa-check"></i><b>5.4.1</b> Populating parent and child events</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="format_emof.html"><a href="format_emof.html"><i class="fa fa-check"></i><b>5.5</b> How to format extendedMeasurementOrFact tables</a></li>
<li class="chapter" data-level="5.6" data-path="dna_data.html"><a href="dna_data.html"><i class="fa fa-check"></i><b>5.6</b> DNA derived data</a>
<ul>
<li class="chapter" data-level="5.6.1" data-path="dna_data.html"><a href="dna_data.html#compiling-edna-and-metabarcoding-datasets"><i class="fa fa-check"></i><b>5.6.1</b> Compiling eDNA and metabarcoding datasets</a></li>
<li class="chapter" data-level="5.6.2" data-path="dna_data.html"><a href="dna_data.html#unknown-sequences"><i class="fa fa-check"></i><b>5.6.2</b> Unknown sequences</a></li>
<li class="chapter" data-level="5.6.3" data-path="dna_data.html"><a href="dna_data.html#compiling-qpcr-datasets"><i class="fa fa-check"></i><b>5.6.3</b> Compiling qPCR datasets</a></li>
<li class="chapter" data-level="5.6.4" data-path="dna_data.html"><a href="dna_data.html#obis-bioinformatics-pipeline"><i class="fa fa-check"></i><b>5.6.4</b> OBIS Bioinformatics Pipeline</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="vocabulary.html"><a href="vocabulary.html"><i class="fa fa-check"></i><b>6</b> Standardizing terms &amp; vocabularies</a>
<ul>
<li class="chapter" data-level="6.1" data-path="vocabulary.html"><a href="vocabulary.html#map-data-fields-to-darwin-core"><i class="fa fa-check"></i><b>6.1</b> Map data fields to Darwin Core</a></li>
<li class="chapter" data-level="6.2" data-path="vocabulary.html"><a href="vocabulary.html#map-emof-measurement-identifiers-to-preferred-vocabulary"><i class="fa fa-check"></i><b>6.2</b> Map eMoF measurement identifiers to preferred vocabulary</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="vocabulary.html"><a href="vocabulary.html#populate-measurementunitid"><i class="fa fa-check"></i><b>6.2.1</b> Populate measurementUnitID</a></li>
<li class="chapter" data-level="6.2.2" data-path="vocabulary.html"><a href="vocabulary.html#populate-measurementvalueid"><i class="fa fa-check"></i><b>6.2.2</b> Populate measurementValueID</a></li>
<li class="chapter" data-level="6.2.3" data-path="vocabulary.html"><a href="vocabulary.html#populate-measurementtypeid"><i class="fa fa-check"></i><b>6.2.3</b> Populate measurementTypeID</a></li>
<li class="chapter" data-level="6.2.4" data-path="vocabulary.html"><a href="vocabulary.html#request-new-vocabulary-terms"><i class="fa fa-check"></i><b>6.2.4</b> Request new vocabulary terms</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="common_formatissues.html"><a href="common_formatissues.html"><i class="fa fa-check"></i><b>7</b> Other Important Data Formatting Steps</a>
<ul>
<li class="chapter" data-level="7.1" data-path="common_formatissues.html"><a href="common_formatissues.html#temporal-dates-and-times"><i class="fa fa-check"></i><b>7.1</b> Temporal: Dates and times</a></li>
<li class="chapter" data-level="7.2" data-path="common_formatissues.html"><a href="common_formatissues.html#historical-data"><i class="fa fa-check"></i><b>7.2</b> Historical data</a></li>
<li class="chapter" data-level="7.3" data-path="common_formatissues.html"><a href="common_formatissues.html#spatial"><i class="fa fa-check"></i><b>7.3</b> Spatial</a></li>
<li class="chapter" data-level="7.4" data-path="common_formatissues.html"><a href="common_formatissues.html#missing-required-fields"><i class="fa fa-check"></i><b>7.4</b> Missing required fields</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="examples.html"><a href="examples.html"><i class="fa fa-check"></i><b>8</b> Dataset Examples: ENV-DATA and DNA derived data</a>
<ul>
<li class="chapter" data-level="8.1" data-path="other_data_types.html"><a href="other_data_types.html"><i class="fa fa-check"></i><b>8.1</b> Other data types</a>
<ul>
<li class="chapter" data-level="8.1.1" data-path="other_data_types.html"><a href="other_data_types.html#multimedia-data-acoustic-imaging"><i class="fa fa-check"></i><b>8.1.1</b> Multimedia data (Acoustic, Imaging)</a></li>
<li class="chapter" data-level="8.1.2" data-path="other_data_types.html"><a href="other_data_types.html#habitat-data"><i class="fa fa-check"></i><b>8.1.2</b> Habitat data</a></li>
<li class="chapter" data-level="8.1.3" data-path="other_data_types.html"><a href="other_data_types.html#tracking-data"><i class="fa fa-check"></i><b>8.1.3</b> Tracking data</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Ensuring Data Quality</b></span></li>
<li class="chapter" data-level="9" data-path="data_qc.html"><a href="data_qc.html"><i class="fa fa-check"></i><b>9</b> Data quality control</a>
<ul>
<li class="chapter" data-level="9.1" data-path="data_qc.html"><a href="data_qc.html#why-are-records-dropped"><i class="fa fa-check"></i><b>9.1</b> Why are records dropped?</a></li>
<li class="chapter" data-level="9.2" data-path="data_qc.html"><a href="data_qc.html#how-to-conduct-quality-control"><i class="fa fa-check"></i><b>9.2</b> How to conduct Quality Control</a></li>
<li class="chapter" data-level="9.3" data-path="dataquality.html"><a href="dataquality.html"><i class="fa fa-check"></i><b>9.3</b> Data quality flags</a></li>
<li class="chapter" data-level="9.4" data-path="mofreports.html"><a href="mofreports.html"><i class="fa fa-check"></i><b>9.4</b> How To Use MoF Report and Tool</a></li>
<li class="chapter" data-level="9.5" data-path="lifewatch_qc.html"><a href="lifewatch_qc.html"><i class="fa fa-check"></i><b>9.5</b> Geographic and data format quality control</a></li>
<li class="chapter" data-level="9.6" data-path="common_qc.html"><a href="common_qc.html"><i class="fa fa-check"></i><b>9.6</b> Common Quality Control issues</a>
<ul>
<li class="chapter" data-level="9.6.1" data-path="common_qc.html"><a href="common_qc.html#uncertain-temporal-range"><i class="fa fa-check"></i><b>9.6.1</b> Uncertain temporal range</a></li>
<li class="chapter" data-level="9.6.2" data-path="common_qc.html"><a href="common_qc.html#uncertain-geolocation"><i class="fa fa-check"></i><b>9.6.2</b> Uncertain geolocation</a></li>
<li class="chapter" data-level="9.6.3" data-path="common_qc.html"><a href="common_qc.html#how-to-use-obis-map-tool"><i class="fa fa-check"></i><b>9.6.3</b> How to use OBIS Map Tool</a></li>
<li class="chapter" data-level="9.6.4" data-path="common_qc.html"><a href="common_qc.html#using-getty-thesaurus-google-maps-to-obtain-locality-coordinates"><i class="fa fa-check"></i><b>9.6.4</b> Using Getty Thesaurus &amp; Google Maps to obtain locality coordinates</a></li>
<li class="chapter" data-level="9.6.5" data-path="common_qc.html"><a href="common_qc.html#how-to-use-marine-regions-gazetteer-tool"><i class="fa fa-check"></i><b>9.6.5</b> How to use Marine Regions Gazetteer tool</a></li>
<li class="chapter" data-level="9.6.6" data-path="common_qc.html"><a href="common_qc.html#low-confidence-taxonomic-identification"><i class="fa fa-check"></i><b>9.6.6</b> Low confidence taxonomic identification</a></li>
<li class="chapter" data-level="9.6.7" data-path="common_qc.html"><a href="common_qc.html#uncertain-measurements"><i class="fa fa-check"></i><b>9.6.7</b> Uncertain measurements</a></li>
<li class="chapter" data-level="9.6.8" data-path="common_qc.html"><a href="common_qc.html#non-marine-species"><i class="fa fa-check"></i><b>9.6.8</b> Non-marine species</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Publishing Data</b></span></li>
<li class="chapter" data-level="10" data-path="data_publication.html"><a href="data_publication.html"><i class="fa fa-check"></i><b>10</b> Data publication and sharing</a>
<ul>
<li class="chapter" data-level="10.1" data-path="data_publication.html"><a href="data_publication.html#licenses"><i class="fa fa-check"></i><b>10.1</b> Licenses</a></li>
<li class="chapter" data-level="10.2" data-path="ipt.html"><a href="ipt.html"><i class="fa fa-check"></i><b>10.2</b> IPT: Integrated Publishing Toolkit</a></li>
<li class="chapter" data-level="10.3" data-path="ipt_admin.html"><a href="ipt_admin.html"><i class="fa fa-check"></i><b>10.3</b> IPT Admin Responsibilities</a></li>
<li class="chapter" data-level="10.4" data-path="data_sharing.html"><a href="data_sharing.html"><i class="fa fa-check"></i><b>10.4</b> Maintaining and sharing published data</a>
<ul>
<li class="chapter" data-level="10.4.1" data-path="data_sharing.html"><a href="data_sharing.html#adding-a-doi-to-datasets"><i class="fa fa-check"></i><b>10.4.1</b> Adding a DOI to datasets</a></li>
<li class="chapter" data-level="10.4.2" data-path="data_sharing.html"><a href="data_sharing.html#user-tracking"><i class="fa fa-check"></i><b>10.4.2</b> User tracking</a></li>
</ul></li>
<li class="chapter" data-level="10.5" data-path="data_sharing.html"><a href="data_sharing.html#update-your-data-in-obis"><i class="fa fa-check"></i><b>10.5</b> Update your data in OBIS</a></li>
<li class="chapter" data-level="10.6" data-path="data_sharing.html"><a href="data_sharing.html#publish-obis-data-to-gbif"><i class="fa fa-check"></i><b>10.6</b> Publish OBIS data to GBIF</a></li>
</ul></li>
<li class="part"><span><b>Access Data from OBIS</b></span></li>
<li class="chapter" data-level="11" data-path="access.html"><a href="access.html"><i class="fa fa-check"></i><b>11</b> Data access</a>
<ul>
<li class="chapter" data-level="11.1" data-path="access.html"><a href="access.html#obis-homepage-and-dataset-pages"><i class="fa fa-check"></i><b>11.1</b> OBIS Homepage and dataset pages</a></li>
<li class="chapter" data-level="11.2" data-path="access.html"><a href="access.html#mapper"><i class="fa fa-check"></i><b>11.2</b> Mapper</a></li>
<li class="chapter" data-level="11.3" data-path="access.html"><a href="access.html#r-package"><i class="fa fa-check"></i><b>11.3</b> R package</a></li>
<li class="chapter" data-level="11.4" data-path="access.html"><a href="access.html#api"><i class="fa fa-check"></i><b>11.4</b> API</a></li>
<li class="chapter" data-level="11.5" data-path="access.html"><a href="access.html#full-exports"><i class="fa fa-check"></i><b>11.5</b> Full exports</a></li>
<li class="chapter" data-level="11.6" data-path="access.html"><a href="access.html#finding-your-own-data-in-obis"><i class="fa fa-check"></i><b>11.6</b> Finding your own data in OBIS</a></li>
<li class="chapter" data-level="11.7" data-path="access.html"><a href="access.html#how-to-contact-data-provider"><i class="fa fa-check"></i><b>11.7</b> How to contact data provider</a></li>
<li class="chapter" data-level="11.8" data-path="access.html"><a href="access.html#interpreting-downloaded-files-from-obis"><i class="fa fa-check"></i><b>11.8</b> Interpreting downloaded files from OBIS</a></li>
<li class="chapter" data-level="11.9" data-path="citing.html"><a href="citing.html"><i class="fa fa-check"></i><b>11.9</b> Citing Data from OBIS</a></li>
</ul></li>
<li class="part"><span><b>Data Visualization and Analysis</b></span></li>
<li class="chapter" data-level="12" data-path="dataviz.html"><a href="dataviz.html"><i class="fa fa-check"></i><b>12</b> Data Visualization</a>
<ul>
<li class="chapter" data-level="12.1" data-path="dataviz.html"><a href="dataviz.html#example-notebooks-using-data-from-obis"><i class="fa fa-check"></i><b>12.1</b> Example notebooks using data from OBIS</a></li>
<li class="chapter" data-level="12.2" data-path="dataviz.html"><a href="dataviz.html#obisindicators-calculating-visualizing-spatial-biodiversity-using-data-from-obis"><i class="fa fa-check"></i><b>12.2</b> obisindicators: calculating &amp; visualizing spatial biodiversity using data from OBIS</a></li>
</ul></li>
<li class="part"><span><b>Additional Resources</b></span></li>
<li class="chapter" data-level="13" data-path="other_resources.html"><a href="other_resources.html"><i class="fa fa-check"></i><b>13</b> Other Resources</a>
<ul>
<li class="chapter" data-level="13.1" data-path="other_resources.html"><a href="other_resources.html#mbon-pole-to-pole-tutorial"><i class="fa fa-check"></i><b>13.1</b> MBON Pole to Pole Tutorial</a></li>
<li class="chapter" data-level="13.2" data-path="other_resources.html"><a href="other_resources.html#ioos-darwin-core-guide"><i class="fa fa-check"></i><b>13.2</b> IOOS Darwin Core Guide</a></li>
<li class="chapter" data-level="13.3" data-path="other_resources.html"><a href="other_resources.html#emodnet-biology"><i class="fa fa-check"></i><b>13.3</b> EMODnet Biology</a></li>
<li class="chapter" data-level="13.4" data-path="other_resources.html"><a href="other_resources.html#template-generators"><i class="fa fa-check"></i><b>13.4</b> Template Generators</a></li>
<li class="chapter" data-level="13.5" data-path="other_resources.html"><a href="other_resources.html#bionomia"><i class="fa fa-check"></i><b>13.5</b> Bionomia</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The OBIS manual</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="frequently-asked-questions" class="section level2 hasAnchor" number="1.5">
<h2><span class="header-section-number">1.5</span> Frequently Asked Questions<a href="FAQ.html#frequently-asked-questions" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><strong>Content</strong></p>
<ul>
<li><a href="FAQ.html#general">General</a></li>
<li><a href="FAQ.html#darwin-core">Darwin Core</a></li>
<li><a href="FAQ.html#formatting-data">Formatting data</a></li>
<li><a href="FAQ.html#vocabulary">Vocabulary</a></li>
<li><a href="FAQ.html#tools">Tools</a></li>
<li><a href="FAQ.html#quality-control">Quality Control</a></li>
<li><a href="FAQ.html#publishing">Publishing</a></li>
<li><a href="FAQ.html#accessing-data-in-obis">Data Access</a></li>
</ul>
<div id="general" class="section level4 hasAnchor" number="1.5.0.1">
<h4><span class="header-section-number">1.5.0.1</span> General<a href="FAQ.html#general" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>
<a href="contribute.html">I have data and want to publish to OBIS - what do I do?</a>
</li>
<li>
<a href="contribute.html#why-publish-data-to-obis">Why is it important to share and format data?</a>
</li>
<li>
<a href="contribute.html#how-to-handle-sensitive-data">How do I handle sensitive data?</a>
</li>
<li>
<a href="https://github.com/iobis/manual">Where can I make suggestions for improvements on this Manual?</a>
</li>
<li>
<a href="https://youtube.com/playlist?list=PLlgUwSvpCFS4TS7ZN0fhByj_3EBZ5lXbF">Where can I find OBIS related training videos?</a>
</li>
<li>
<a href="nodes.html">What are the responsibilities of OBIS node managers?</a>
</li>
<li>
<a href="https://github.com/iobis/obis-network-datasets/">Where can I find marine datasets linked to the OBIS network by the GBIF registry, that now require endorising?</a>
</li>
<li>
<details>
<summary>
I am an OBIS node and need to update the contact information listed for my node on the OBIS website, how do I do that?
</summary>
<p>To update OBIS Node contact information listed at <a href="https://obis.org/contact/" class="uri">https://obis.org/contact/</a>, you can edit the information here <a href="https://metadata.obis.org/" class="uri">https://metadata.obis.org/</a> and signing in with your OceanExpert credentials.</p>
</details>
</li>
</ul>
</div>
<div id="darwin-core" class="section level4 hasAnchor" number="1.5.0.2">
<h4><span class="header-section-number">1.5.0.2</span> Darwin Core<a href="FAQ.html#darwin-core" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>
<a href="darwin_core.html">Where can I learn about “Darwin Core”?</a>
</li>
<li>
<a href="relational_db.html">I am having trouble understanding how Core and Extension tables relate to each other</a>
</li>
<li>
<a href="relational_db.html#how-to-avoid-redundancy">How does the OBIS format avoid redundancy in data</a>
</li>
<li>
<a href="formatting.html#extensions-in-obis">How are extension tables (e.g. eMOF, occurrence) linked with the core table?</a>
</li>
<li>
<a href="formatting.html#dataset-structure">What is the difference between Occurence Core and Event Core?</a>
</li>
<li>
<a href="identifiers.html#introduction-to-identifiers">What is the difference between eventID, occurrenceID, organismID, taxonID, scientificNameID, recordNumber, materialEntityID, materialSampleID, and catalogNumber?</a>
</li>
<li>
<details>
<summary>
What is the difference between <code>individualCount</code>, and <code>organismQuantity</code>, and which one should I use?
</summary>
<p>The DwC term <code>individualCount</code> is used simply for documenting the <em>number of individuals</em> present at the time the occurrence happened. Conversely, <code>organismQuantity</code> is used to record <em>any type</em> of quantity related to an organism or occurrence. <code>organismQuantityType</code> may include the number of individuals, the % cover, biomass, the cover-abundance on the Braun-Blanquet Scale, etc. While both DwC terms can be placed in the Occurrence table, OBIS recommends also populating them in the eMoF table because of the standardization of terms and because you can link to sampling events and factual sampling information.</p>
</details>
</li>
</ul>
</div>
<div id="formatting-data" class="section level4 hasAnchor" number="1.5.0.3">
<h4><span class="header-section-number">1.5.0.3</span> Formatting Data<a href="FAQ.html#formatting-data" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>
<a href="checklist.html">Is there a checklist of all required Darwin Core fields for OBIS?</a>
</li>
<li>
<a href="data_standards.html">How does data flow in OBIS?</a>
</li>
<li>
<a href="common_formatissues.html#missing-required-fields">What should I do if I do not have the data for required fields by OBIS?</a>
</li>
<li>
<a href="identifiers.html#eventid">How do I construct an eventID?</a>
</li>
<li>
<a href="identifiers.html#occurrenceid">How do I construct occurrenceID?</a>
</li>
<li>
<a href="format_occurrence.html">What data goes into Occurrence core (or extension) and how do I set up this file?</a>
</li>
<li>
<a href="format_event.html">How do I set up an Event core table?</a>
</li>
<li>
<a href="format_event.html#populating-parent-and-child-events">How should I populate parent and child event information? Do I have to populate information for child events if the parent event already contains that information?</a>
</li>
<li>
<details>
<summary>
Do I have to provide decimalLatitude and decimalLongitude for the Event <i>and</i> Occurrence tables?
</summary>
The answer may depend on your dataset structure, but generally, no. If you have Event core, then you do not need to repeat location information in the Occurrence table (but you can if you’d like). If you are using Occurrence core, then location information must be provided in the Occurrence table.
</details>
</li>
<li>
<a href="format_emof.html">What data goes into extendedMeasurementOrFact (eMoF) and how do I set it up?</a>
</li>
<li>
<details>
<summary>
Can I have multiple eMoF tables, one for Event and one for Occurrence measurements?
</summary>
Yes, you can create multiple eMoF tables if it is simpler for you to separate Event and Occurrence measurement data. However, you must ensure all tables include the correct identifiers, i.e. eventID for measurements related to events, and occurrenceID <em>and</em> eventID for measurements associated with occurrences. When you add multiple tables of the same type in the IPT, they will be treated as if they were a single table, so there is no issue if you would like to do this.
</details>
</li>
<li>
<a href="common_formatissues.html#temporal-dates-and-times">How do I format dates?</a>
</li>
<li>
<a href="common_formatissues.html#historical-data"> How do I handle historical data?</a>
</li>
<li>
<a href="common_formatissues.html#converting-coordinates">How do I convert coordinates to decimal degrees?</a>
</li>
<li>
<a href="common_formatissues.html#geographical-format-conversion">How do I convert different geographical formats to WGS84?</a>
</li>
<li>
<a href="other_data_types.html#multimedia-data-acoustic-imaging">How do I compile acoustic, imaging, or other multimedia data for OBIS?</a>
</li>
<li>
<a href="other_data_types.html#habitat-data">How do I compile habitat data for OBIS?</a>
</li>
<li>
<a href="other_data_types.html#tracking-data">How do I compile tracking data for OBIS?</a>
</li>
<li>
<a href="dna_data.html">How do I compile DNA and genetic data for OBIS?</a>
</li>
<li>
<details>
<summary>
What does sampleSize and organismQuantity refer to for genetic data?
</summary>
<p>For genetic data, <code>sampleSizeValue</code> and <code>organismQuantity</code> do <strong>not</strong> refer to the amount sampled nor the number of organisms. Instead these fields are related to either 1) the number of sequence reads for eDNA data or 2) the number of droplets/partitions for qPCR data. See <a href="dna_data#.html">DNA data guidelines</a> for more details.</p>
</details>
</li>
<li>
<details>
<summary>
How do I document occurrences from unknown species, those new to science, or those with temporary names? e.g. Eurythenes sp. DISCOLL.PAP.JC165.674
</summary>
<p>Occurrences unknown or new to science should be documented according to recommendations by <a href="https://www.frontiersin.org/articles/10.3389/fmars.2021.620702/full">Horton et al. 2021</a>. You should populate the <code>scientificName</code> field with the genus, and in <code>identificationQualifer</code> provide the ON sign ‘sp.’. However you must also indicate the reason why species-level identification is unavailable. To do this, supplement ‘sp.’ with either stet. (stetit) or indet. (indeterminabilis). If neither of these are applicable, (e.g. for undescribed new species), add a unique taxon identifier code after ‘sp.’ to <code>identificationQualifer</code>. For example Eurythenes sp. DISCOLL.PAP.JC165.674.</p>
Please avoid simple alphanumeric codes (i.e. Eurythenes sp. 1, Eurythenes sp. A). Similar to creating <code>eventIDs</code> or <code>occurrenceIDs</code>, you should strive to provide more complex and globally unique identifier. Identifiers could be constructed by combining higher taxonomic information with information related to a collection, institution, museum or collection code, sample number or museum accession number, expedition, dive number, or timestamp. This ensures namestrings will remain unique within a larger repositories like OBIS. It is also recommended to include these temporary names on specimen labels for physical specimens.
</details>
</li>
</ul>
</div>
<div id="vocabulary" class="section level4 hasAnchor" number="1.5.0.4">
<h4><span class="header-section-number">1.5.0.4</span> Vocabulary<a href="FAQ.html#vocabulary" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>
<a href="vocabulary.html#map-emof-measurement-identifiers-to-preferred-bodc-vocabulary">How do I map Measurement or Fact terms in OBIS with preferred BODC vocabulary?</a>
</li>
<li>
<a href="vocabulary.html#requesting-new-vocabulary-terms">I can’t find a suitable vocabulary, what do I do? How do I request a new vocabulary term?</a>
</li>
<li>
<details>
<summary>
Should I use taxon-specific P01 codes to populate for measurementTypeID? e.g. <link><a href="http://vocab.nerc.ac.uk/collection/P01/current/A15985A1" class="uri">http://vocab.nerc.ac.uk/collection/P01/current/A15985A1</a></link>
</summary>
No. You should never use taxon-specific P01 codes. This is because the taxa are already identified in the Occurrence table, in the fields <code>scientificName</code> and <code>scientificNameID</code>.
</details>
</li>
<li>
<a href="vocabulary.html#map-data-fields-to-darwin-core">How should I match raw data fields with Darwin Core terminology?</a>
</li>
</ul>
</div>
<div id="tools" class="section level4 hasAnchor" number="1.5.0.5">
<h4><span class="header-section-number">1.5.0.5</span> Tools<a href="FAQ.html#tools" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>
<a href="name_matching.html">How do I use the WoRMS taxon match tool?</a>
</li>
<li>
<a href="name_matching.html#how-to-fetch-a-full-classification-for-a-list-of-species-from-worms">Can I fetch a full classification for a list of species from WoRMS?</a>
</li>
<li>
<a href="name_matching.html#what-to-do-with-non-matching-names">What do I do if my scientificName does not return a match from WoRMS?</a>
</li>
<li>
<details>
<summary>
Can scientificNameID be populated with an identifier (e.g. WoRMS LSID) representing an unaccepted taxon name?
</summary>
<p>Yes. The identifier in <code>scientificNameID</code> should always correspond with the name that is in the <code>scientificName</code> field, even if the name is an unaccepted name in WoRMS. For example, the species name “Holothuria mammiculata” was provided, but this name is unaccepted in WoRMS. The accepted name is “Holothuria (Stauropora) pervicax Selenka, 1867”. In this case <code>scientificNameID</code> should correspond to the original name with LSID <a href="urn:lsid:marinespecies.org:taxname:529968" class="uri">urn:lsid:marinespecies.org:taxname:529968</a> because the ID must correlate with the name as recorded in <code>scientificName</code>.</p>
</details>
</li>
<li>
<a href="https://sequence.obis.org/">Where can I find DNA sequences published in OBIS?</a>
</li>
<li>
<details>
<summary>
Is there a template generator I can use to help create my Event, Occurrence, and eMoF tables?
</summary>
<p>Yes. There is an <a href="https://www.nordatanet.no/aen/template-generator/config%3DDarwin%20Core">Excel template generator</a> developed by Luke Marsden &amp; Olaf Schneider as part of the Nansen Legacy project. Note this template generator is aimed at GBIF users, so make to account for and include required OBIS terms.</p>
<p>There is also this <a href="https://zenodo.org/record/6453921#.Y9KsQkHMKmU">Excel to Darwin Core macro tool</a> developed by GBIF Norway you can use to help generate templates.</p>
</details>
</li>
<li>
<a href="common_qc.html#uncertain-geolocation">How do I georeference locations, including text-based descriptions?</a>
</li>
</ul>
</div>
<div id="quality-control" class="section level4 hasAnchor" number="1.5.0.6">
<h4><span class="header-section-number">1.5.0.6</span> Quality Control<a href="FAQ.html#quality-control" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li><a href="data_qc.html#how-to-conduct-quality-control">How do I do data quality control?</a></li>
<li><a href="dataquality.html">What are the OBIS quality control flags?</a></li>
<li><a href="data_qc.html#why-are-records-dropped">Why are certain records dropped in OBIS?</a></li>
<li>What do I do when I am uncertain about the:
<ul>
<li><a href="common_qc.html#uncertain-temporal-range">Temporal range of a dataset OR eventDate</a></li>
<li><a href="common_qc.html#uncertain-geolocation">Geospatial location</a></li>
<li><a href="common_qc.html#uncertain-taxonomic-information">Taxonomic identification</a></li>
<li><a href="common_qc.html#individualcount">Individual count</a></li>
</ul></li>
<li><a href="common_qc.html#non-marine-species">What do I do with freshwater species that are part of my marine dataset?</a></li>
</ul>
</div>
<div id="publishing" class="section level4 hasAnchor" number="1.5.0.7">
<h4><span class="header-section-number">1.5.0.7</span> Publishing<a href="FAQ.html#publishing" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li><a href="data_publication.html">How do I add my data to the OBIS database?</a></li>
<li><a href="eml.html#metadata-sections">What metadata do I have to provide? Where? How?</a></li>
<li><a href="data_publication.html#licenses">How do you know which license to choose?</a></li>
<li><a href="ipt.html#how-to-access-the-ipt">How do I access the IPT?</a></li>
<li><a href="ipt.html#create-your-resource-on-the-ipt">How do I use the IPT?</a></li>
<li><a href="ipt_admin.html">Are there instructions for IPT administrators?</a></li>
<li><a href="data_sharing.html#adding-a-doi-to-datasets">How do I add DOI to my dataset?</a></li>
<li><a href="data_sharing.html#simultaneous-publishing-to-gbif">How do I publish to both GBIF and OBIS?</a></li>
<li><a href="data_sharing.html#update-your-data-in-obis">How do I update my already published dataset?</a></li>
</ul>
</div>
<div id="accessing-data-in-obis" class="section level4 hasAnchor" number="1.5.0.8">
<h4><span class="header-section-number">1.5.0.8</span> Accessing data in OBIS<a href="FAQ.html#accessing-data-in-obis" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ul>
<li>
<details>
<summary>
Can I get (or publish) other types of information from OBIS, besides occurences (e.g. bio-eco variables, DNA data, abiotic measurements, etc.)?
</summary>
Yes! OBIS allows access to not only taxa occurrences, but also all kinds of measurement data including DNA data (see <a href="contribute.html">Contribute</a> for a list of data accepted by OBIS). This data is recorded in the (Extended) Measurement Or Fact and DNA-Derived Data extension tables, respectively. To access this data when downloading, be sure to select MoF or DNA from the dropdown Dataset Type menu on the <a href="https://obis.org/datasets">advanced dataset serach page</a>. If you are obtaining data from the OBIS Mapper, make sure to check the box for MeasurementOrFact and/or DNADerivedData from the Extensions toggle section. When using the R package robis, use the <code>mof=TRUE</code> and/or <code>dna=TRUE</code> arguments. More data filter options will be added in the future.
</details>
</li>
<li>
<a href="access.html#obis-homepage-and-dataset-pages">How do I download data from OBIS?</a>
</li>
<li>
<details>
<summary>
How do I load the full (.csv) export of OBIS data?
</summary>
<p>Loading the entire OBIS dataset uses <em>a lot</em> of memory and is probably not feasible on most desktop computers. You have a few potential options depending on the use case: i) process the data in smaller batches, or ii) load the dataset into a local database such as SQLite and use SQL queries to analyze the data</p>
Otherwise, we recommend you use the parquet download which is available <a href="https://obis.org/data/access/">here</a>, instead of the CSV. Then in R, you can use the <a href="https://arrow.apache.org/docs/r/"><code>arrow</code></a> package to work with parquet files. We also have a short tutorial on working with parquet files in R <a href="https://resources.obis.org/tutorials/arrow-obis/">here</a>, with an example application of this approach <a href="https://iobis.github.io/notebook-diversity-indicators/">here</a> (see first code block).
</details>
</li>
<li>
<a href="access.html#r-package">How can I use R to access OBIS data?</a>
</li>
<li>
<a href="access.html#api">How do I use the OBIS API to fetch and filter data?</a>
</li>
<li>
<a href="access.html#api">How do I contact the data provider?</a>
</li>
<li>
<a href="citing.html">How can I cite OBIS datasets and downloads?</a>
</li>
<li>
<a href="access.html#interpreting-downloaded-files-from-obis">What are the definitions of the field names in the downloads generated by OBIS?</a>
</li>
<li>
<details>
<summary>
How do I obtain a taxon checklist for an area?
</summary>
<p>There are a few possible ways to obtain a taxon checklist for a given area. We will obtain a checklist of species in the Albain EEZ as an example. To do this we will create a bounding box around our area of interest, and then apply filters to simplify the geometry.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="FAQ.html#cb6-1" tabindex="-1"></a>  <span class="fu">library</span>(mregions)</span>
<span id="cb6-2"><a href="FAQ.html#cb6-2" tabindex="-1"></a>  <span class="fu">library</span>(dplyr)</span>
<span id="cb6-3"><a href="FAQ.html#cb6-3" tabindex="-1"></a>  <span class="fu">library</span>(robis)</span>
<span id="cb6-4"><a href="FAQ.html#cb6-4" tabindex="-1"></a>  <span class="fu">library</span>(sf)</span>
<span id="cb6-5"><a href="FAQ.html#cb6-5" tabindex="-1"></a>  <span class="co">#obtain Albanian EEZ as sf</span></span>
<span id="cb6-6"><a href="FAQ.html#cb6-6" tabindex="-1"></a>  geom <span class="ot">&lt;-</span> <span class="fu">mr_shp</span>(<span class="at">key =</span> <span class="st">&quot;MarineRegions:eez&quot;</span>, <span class="at">filter =</span> <span class="st">&quot;Albanian Exclusive Economic Zone&quot;</span>, <span class="at">maxFeatures =</span> <span class="cn">NULL</span>)</span>
<span id="cb6-7"><a href="FAQ.html#cb6-7" tabindex="-1"></a>  <span class="co">#get WKT for the bounding box</span></span>
<span id="cb6-8"><a href="FAQ.html#cb6-8" tabindex="-1"></a>  wkt <span class="ot">&lt;-</span> <span class="fu">st_as_text</span>(<span class="fu">st_as_sfc</span>(<span class="fu">st_bbox</span>(geom)), <span class="at">digits =</span> <span class="dv">6</span>)</span>
<span id="cb6-9"><a href="FAQ.html#cb6-9" tabindex="-1"></a>  <span class="co">#fetch occurrences for bounding box</span></span>
<span id="cb6-10"><a href="FAQ.html#cb6-10" tabindex="-1"></a>  occ <span class="ot">&lt;-</span> <span class="fu">occurrence</span>(<span class="at">geometry =</span> wkt) <span class="sc">%&gt;%</span></span>
<span id="cb6-11"><a href="FAQ.html#cb6-11" tabindex="-1"></a>    <span class="fu">st_as_sf</span>(<span class="at">coords =</span> <span class="fu">c</span>(<span class="st">&quot;decimalLongitude&quot;</span>, <span class="st">&quot;decimalLatitude&quot;</span>), <span class="at">crs =</span> <span class="dv">4326</span>)</span>
<span id="cb6-12"><a href="FAQ.html#cb6-12" tabindex="-1"></a>  <span class="co">#filter using geometry</span></span>
<span id="cb6-13"><a href="FAQ.html#cb6-13" tabindex="-1"></a>  occ_filtered <span class="ot">&lt;-</span> occ <span class="sc">%&gt;%</span></span>
<span id="cb6-14"><a href="FAQ.html#cb6-14" tabindex="-1"></a>    <span class="fu">filter</span>(<span class="fu">st_intersects</span>(geometry, geom, <span class="at">sparse =</span> <span class="cn">FALSE</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb6-15"><a href="FAQ.html#cb6-15" tabindex="-1"></a>    <span class="fu">as_tibble</span>() <span class="sc">%&gt;%</span></span>
<span id="cb6-16"><a href="FAQ.html#cb6-16" tabindex="-1"></a>    <span class="fu">select</span>(<span class="sc">-</span>geometry)</span>
<span id="cb6-17"><a href="FAQ.html#cb6-17" tabindex="-1"></a>  <span class="co">#get taxa</span></span>
<span id="cb6-18"><a href="FAQ.html#cb6-18" tabindex="-1"></a>  alb_taxa <span class="ot">&lt;-</span> occ_filtered <span class="sc">%&gt;%</span></span>
<span id="cb6-19"><a href="FAQ.html#cb6-19" tabindex="-1"></a>    <span class="fu">group_by</span>(phylum, class, order, family, genus, species, scientificName) <span class="sc">%&gt;%</span></span>
<span id="cb6-20"><a href="FAQ.html#cb6-20" tabindex="-1"></a>    <span class="fu">summarize</span>(<span class="at">records =</span> <span class="fu">n</span>())</span></code></pre></div>
</details>
</li>
<li>
<details>
<summary>
The dates look unusual in the download file. What are these, how do I convert them, and/or how do I obtain separate elements from them (e.g. month)?
</summary>
<p>The values in <code>date_start</code>, <code>date_mid</code>, and <code>date_end</code> are unix timestamps which have been calculated from the ISO date in the <code>eventDate</code> column. We can convert these numerical values to dates using the formula below.</p>
<pre class="excel"><code>=(E2/86400000)+DATE(1970,1,1)</code></pre>
<p>If, when you apply this formula, you still see numbers, you will need to set the cell formatting to Date. Once you have dates, you can obtain, e.g. months for seasonal analyses using:</p>
<pre class="excel"><code>=MONTH(H2)</code></pre>
<p>You can also use <a href="https://www.unixtimestamp.com/">this tool</a> to convert timestamps.</p>
</details>
</li>
<li>
<details>
<summary>
How do I filter by or obtain trait information for OBIS data (e.g. all benthic organisms)?
</summary>
<p>Currently, it is not possible to filter OBIS data by trait. To do this, we recommend using the traits database of the <a href="https://www.marinespecies.org/traits/aphia.php?p=attributes">World Register of Marine Species</a>. For example, searching by “functional group”, you can specify benthos, plankton, nekton, etc.</p>
</details>
</li>
</ul>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="gethelp.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="linklist.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["manual.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "none"
}
});
});
</script>

</body>

</html>
