<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5.2 Name matching for taxonomic quality control | The OBIS manual</title>
  <meta name="description" content="5.2 Name matching for taxonomic quality control | The OBIS manual" />
  <meta name="generator" content="bookdown 0.40 and GitBook 2.6.7" />

  <meta property="og:title" content="5.2 Name matching for taxonomic quality control | The OBIS manual" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5.2 Name matching for taxonomic quality control | The OBIS manual" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="checklist.html"/>
<link rel="next" href="format_occurrence.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-JBW0JFMNBJ"></script>
<script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'G-JBW0JFMNBJ');
</script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="/">The OBIS manual</a></li>
<li><a href="https://obis.org/">Return to OBIS.org</a></li>

<li class="divider"></li>
<li class="part"><span><b>Overview</b></span></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#guidelines-on-the-sharing-and-use-of-data-in-obis"><i class="fa fa-check"></i><b>1.1</b> Guidelines on the sharing and use of data in OBIS</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i><b>1.2</b> Acknowledgements</a></li>
<li class="chapter" data-level="1.3" data-path="policy.html"><a href="policy.html"><i class="fa fa-check"></i><b>1.3</b> Data Policy</a></li>
<li class="chapter" data-level="1.4" data-path="gethelp.html"><a href="gethelp.html"><i class="fa fa-check"></i><b>1.4</b> Getting Help in OBIS</a></li>
<li class="chapter" data-level="1.5" data-path="FAQ.html"><a href="FAQ.html"><i class="fa fa-check"></i><b>1.5</b> Frequently Asked Questions</a></li>
<li class="chapter" data-level="1.6" data-path="linklist.html"><a href="linklist.html"><i class="fa fa-check"></i><b>1.6</b> Quick links to resources</a></li>
</ul></li>
<li class="part"><span><b>Contributing data to OBIS</b></span></li>
<li class="chapter" data-level="2" data-path="contribute.html"><a href="contribute.html"><i class="fa fa-check"></i><b>2</b> What can you contribute and how?</a>
<ul>
<li class="chapter" data-level="2.1" data-path="contribute.html"><a href="contribute.html#why-publish-data-to-obis"><i class="fa fa-check"></i><b>2.1</b> Why publish data to OBIS</a></li>
<li class="chapter" data-level="2.2" data-path="contribute.html"><a href="contribute.html#how-to-handle-sensitive-data"><i class="fa fa-check"></i><b>2.2</b> How to handle sensitive data</a></li>
<li class="chapter" data-level="2.3" data-path="data_standards.html"><a href="data_standards.html"><i class="fa fa-check"></i><b>2.3</b> OBIS Data Life Cycle</a></li>
<li class="chapter" data-level="2.4" data-path="data_standards.html"><a href="data_standards.html#biodiversity-data-standards"><i class="fa fa-check"></i><b>2.4</b> Biodiversity data standards</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="darwin_core.html"><a href="darwin_core.html"><i class="fa fa-check"></i><b>2.4.1</b> Darwin Core Terms &amp; Guidelines</a></li>
<li class="chapter" data-level="2.4.2" data-path="data_format.html"><a href="data_format.html"><i class="fa fa-check"></i><b>2.4.2</b> Darwin Core Archive</a></li>
<li class="chapter" data-level="2.4.3" data-path="relational_db.html"><a href="relational_db.html"><i class="fa fa-check"></i><b>2.4.3</b> Relational databases: the underlying framework of OBIS</a></li>
<li class="chapter" data-level="2.4.4" data-path="eml.html"><a href="eml.html"><i class="fa fa-check"></i><b>2.4.4</b> Ecological Metadata Language</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="nodes.html"><a href="nodes.html"><i class="fa fa-check"></i><b>3</b> OBIS Terms of Reference: Nodes, Coordination Groups, Secretariat</a>
<ul>
<li class="chapter" data-level="3.1" data-path="nodes.html"><a href="nodes.html#tor-of-obis-nodes"><i class="fa fa-check"></i><b>3.1</b> TOR of OBIS nodes</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="nodes.html"><a href="nodes.html#how-to-become-an-obis-node"><i class="fa fa-check"></i><b>3.1.1</b> How to become an OBIS node</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="nodes.html"><a href="nodes.html#tor-of-obis-steering-group"><i class="fa fa-check"></i><b>3.2</b> TOR of OBIS Steering Group</a></li>
<li class="chapter" data-level="3.3" data-path="nodes.html"><a href="nodes.html#tor-of-obis-co-chairs"><i class="fa fa-check"></i><b>3.3</b> TOR of OBIS Co-Chairs</a></li>
<li class="chapter" data-level="3.4" data-path="nodes.html"><a href="nodes.html#tor-of-obis-secretariat"><i class="fa fa-check"></i><b>3.4</b> TOR of OBIS Secretariat</a></li>
<li class="chapter" data-level="3.5" data-path="nodes.html"><a href="nodes.html#tor-of-obis-coordination-groups"><i class="fa fa-check"></i><b>3.5</b> TOR of OBIS Coordination Groups</a></li>
<li class="chapter" data-level="3.6" data-path="nodes.html"><a href="nodes.html#tor-of-obis-executive-committee"><i class="fa fa-check"></i><b>3.6</b> TOR of OBIS Executive Committee</a></li>
</ul></li>
<li class="part"><span><b>Data Formatting</b></span></li>
<li class="chapter" data-level="4" data-path="formatting.html"><a href="formatting.html"><i class="fa fa-check"></i><b>4</b> Dataset structure</a>
<ul>
<li class="chapter" data-level="4.1" data-path="formatting.html"><a href="formatting.html#when-to-use-event-core"><i class="fa fa-check"></i><b>4.1</b> When to use Event Core</a></li>
<li class="chapter" data-level="4.2" data-path="formatting.html"><a href="formatting.html#when-to-use-occurrence-core"><i class="fa fa-check"></i><b>4.2</b> When to use Occurrence Core</a></li>
<li class="chapter" data-level="4.3" data-path="formatting.html"><a href="formatting.html#extensions-in-obis"><i class="fa fa-check"></i><b>4.3</b> Extensions in OBIS</a></li>
<li class="chapter" data-level="4.4" data-path="formatting.html"><a href="formatting.html#data-formatting-tools"><i class="fa fa-check"></i><b>4.4</b> Data formatting tools</a></li>
<li class="chapter" data-level="4.5" data-path="identifiers.html"><a href="identifiers.html"><i class="fa fa-check"></i><b>4.5</b> Constructing and using identifier codes</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="identifiers.html"><a href="identifiers.html#eventid"><i class="fa fa-check"></i><b>4.5.1</b> eventID</a></li>
<li class="chapter" data-level="4.5.2" data-path="identifiers.html"><a href="identifiers.html#occurrenceid"><i class="fa fa-check"></i><b>4.5.2</b> occurrenceID</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="checklist.html"><a href="checklist.html"><i class="fa fa-check"></i><b>5</b> Formatting data tables</a>
<ul>
<li class="chapter" data-level="5.1" data-path="checklist.html"><a href="checklist.html#darwin-core-term-checklist-for-obis"><i class="fa fa-check"></i><b>5.1</b> Darwin Core Term Checklist for OBIS</a></li>
<li class="chapter" data-level="5.2" data-path="name_matching.html"><a href="name_matching.html"><i class="fa fa-check"></i><b>5.2</b> Name matching for taxonomic quality control</a></li>
<li class="chapter" data-level="5.3" data-path="format_occurrence.html"><a href="format_occurrence.html"><i class="fa fa-check"></i><b>5.3</b> How to format Occurrence tables</a></li>
<li class="chapter" data-level="5.4" data-path="format_event.html"><a href="format_event.html"><i class="fa fa-check"></i><b>5.4</b> How to format Event tables</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="format_event.html"><a href="format_event.html#populating-parent-and-child-events"><i class="fa fa-check"></i><b>5.4.1</b> Populating parent and child events</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="format_emof.html"><a href="format_emof.html"><i class="fa fa-check"></i><b>5.5</b> How to format extendedMeasurementOrFact tables</a></li>
<li class="chapter" data-level="5.6" data-path="dna_data.html"><a href="dna_data.html"><i class="fa fa-check"></i><b>5.6</b> DNA derived data</a>
<ul>
<li class="chapter" data-level="5.6.1" data-path="dna_data.html"><a href="dna_data.html#compiling-edna-and-metabarcoding-datasets"><i class="fa fa-check"></i><b>5.6.1</b> Compiling eDNA and metabarcoding datasets</a></li>
<li class="chapter" data-level="5.6.2" data-path="dna_data.html"><a href="dna_data.html#unknown-sequences"><i class="fa fa-check"></i><b>5.6.2</b> Unknown sequences</a></li>
<li class="chapter" data-level="5.6.3" data-path="dna_data.html"><a href="dna_data.html#compiling-qpcr-datasets"><i class="fa fa-check"></i><b>5.6.3</b> Compiling qPCR datasets</a></li>
<li class="chapter" data-level="5.6.4" data-path="dna_data.html"><a href="dna_data.html#obis-bioinformatics-pipeline"><i class="fa fa-check"></i><b>5.6.4</b> OBIS Bioinformatics Pipeline</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="vocabulary.html"><a href="vocabulary.html"><i class="fa fa-check"></i><b>6</b> Standardizing terms &amp; vocabularies</a>
<ul>
<li class="chapter" data-level="6.1" data-path="vocabulary.html"><a href="vocabulary.html#map-data-fields-to-darwin-core"><i class="fa fa-check"></i><b>6.1</b> Map data fields to Darwin Core</a></li>
<li class="chapter" data-level="6.2" data-path="vocabulary.html"><a href="vocabulary.html#map-emof-measurement-identifiers-to-preferred-vocabulary"><i class="fa fa-check"></i><b>6.2</b> Map eMoF measurement identifiers to preferred vocabulary</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="vocabulary.html"><a href="vocabulary.html#populate-measurementunitid"><i class="fa fa-check"></i><b>6.2.1</b> Populate measurementUnitID</a></li>
<li class="chapter" data-level="6.2.2" data-path="vocabulary.html"><a href="vocabulary.html#populate-measurementvalueid"><i class="fa fa-check"></i><b>6.2.2</b> Populate measurementValueID</a></li>
<li class="chapter" data-level="6.2.3" data-path="vocabulary.html"><a href="vocabulary.html#populate-measurementtypeid"><i class="fa fa-check"></i><b>6.2.3</b> Populate measurementTypeID</a></li>
<li class="chapter" data-level="6.2.4" data-path="vocabulary.html"><a href="vocabulary.html#request-new-vocabulary-terms"><i class="fa fa-check"></i><b>6.2.4</b> Request new vocabulary terms</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="common_formatissues.html"><a href="common_formatissues.html"><i class="fa fa-check"></i><b>7</b> Other Important Data Formatting Steps</a>
<ul>
<li class="chapter" data-level="7.1" data-path="common_formatissues.html"><a href="common_formatissues.html#temporal-dates-and-times"><i class="fa fa-check"></i><b>7.1</b> Temporal: Dates and times</a></li>
<li class="chapter" data-level="7.2" data-path="common_formatissues.html"><a href="common_formatissues.html#historical-data"><i class="fa fa-check"></i><b>7.2</b> Historical data</a></li>
<li class="chapter" data-level="7.3" data-path="common_formatissues.html"><a href="common_formatissues.html#spatial"><i class="fa fa-check"></i><b>7.3</b> Spatial</a></li>
<li class="chapter" data-level="7.4" data-path="common_formatissues.html"><a href="common_formatissues.html#missing-required-fields"><i class="fa fa-check"></i><b>7.4</b> Missing required fields</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="examples.html"><a href="examples.html"><i class="fa fa-check"></i><b>8</b> Dataset Examples: ENV-DATA</a>
<ul>
<li class="chapter" data-level="8.1" data-path="other_data_types.html"><a href="other_data_types.html"><i class="fa fa-check"></i><b>8.1</b> Multimedia data (Acoustic, Imaging)</a></li>
<li class="chapter" data-level="8.2" data-path="other_data_types.html"><a href="other_data_types.html#habitat-data"><i class="fa fa-check"></i><b>8.2</b> Habitat data</a></li>
<li class="chapter" data-level="8.3" data-path="other_data_types.html"><a href="other_data_types.html#tracking-data"><i class="fa fa-check"></i><b>8.3</b> Tracking data</a></li>
</ul></li>
<li class="part"><span><b>Ensuring Data Quality</b></span></li>
<li class="chapter" data-level="9" data-path="data_qc.html"><a href="data_qc.html"><i class="fa fa-check"></i><b>9</b> Data quality control</a>
<ul>
<li class="chapter" data-level="9.1" data-path="data_qc.html"><a href="data_qc.html#why-are-records-dropped"><i class="fa fa-check"></i><b>9.1</b> Why are records dropped?</a></li>
<li class="chapter" data-level="9.2" data-path="data_qc.html"><a href="data_qc.html#how-to-conduct-quality-control"><i class="fa fa-check"></i><b>9.2</b> How to conduct Quality Control</a></li>
<li class="chapter" data-level="9.3" data-path="dataquality.html"><a href="dataquality.html"><i class="fa fa-check"></i><b>9.3</b> Data quality flags</a></li>
<li class="chapter" data-level="9.4" data-path="mofreports.html"><a href="mofreports.html"><i class="fa fa-check"></i><b>9.4</b> How To Use MoF Report and Tool</a></li>
<li class="chapter" data-level="9.5" data-path="lifewatch_qc.html"><a href="lifewatch_qc.html"><i class="fa fa-check"></i><b>9.5</b> Geographic and data format quality control</a></li>
<li class="chapter" data-level="9.6" data-path="common_qc.html"><a href="common_qc.html"><i class="fa fa-check"></i><b>9.6</b> Common Quality Control issues</a>
<ul>
<li class="chapter" data-level="9.6.1" data-path="common_qc.html"><a href="common_qc.html#uncertain-temporal-range"><i class="fa fa-check"></i><b>9.6.1</b> Uncertain temporal range</a></li>
<li class="chapter" data-level="9.6.2" data-path="common_qc.html"><a href="common_qc.html#uncertain-geolocation"><i class="fa fa-check"></i><b>9.6.2</b> Uncertain geolocation</a></li>
<li class="chapter" data-level="9.6.3" data-path="common_qc.html"><a href="common_qc.html#how-to-use-obis-map-tool"><i class="fa fa-check"></i><b>9.6.3</b> How to use OBIS Map Tool</a></li>
<li class="chapter" data-level="9.6.4" data-path="common_qc.html"><a href="common_qc.html#using-getty-thesaurus-google-maps-to-obtain-locality-coordinates"><i class="fa fa-check"></i><b>9.6.4</b> Using Getty Thesaurus &amp; Google Maps to obtain locality coordinates</a></li>
<li class="chapter" data-level="9.6.5" data-path="common_qc.html"><a href="common_qc.html#how-to-use-marine-regions-gazetteer-tool"><i class="fa fa-check"></i><b>9.6.5</b> How to use Marine Regions Gazetteer tool</a></li>
<li class="chapter" data-level="9.6.6" data-path="common_qc.html"><a href="common_qc.html#low-confidence-taxonomic-identification"><i class="fa fa-check"></i><b>9.6.6</b> Low confidence taxonomic identification</a></li>
<li class="chapter" data-level="9.6.7" data-path="common_qc.html"><a href="common_qc.html#uncertain-measurements"><i class="fa fa-check"></i><b>9.6.7</b> Uncertain measurements</a></li>
<li class="chapter" data-level="9.6.8" data-path="common_qc.html"><a href="common_qc.html#non-marine-species"><i class="fa fa-check"></i><b>9.6.8</b> Non-marine species</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Publishing Data</b></span></li>
<li class="chapter" data-level="10" data-path="data_publication.html"><a href="data_publication.html"><i class="fa fa-check"></i><b>10</b> Data publication and sharing</a>
<ul>
<li class="chapter" data-level="10.1" data-path="data_publication.html"><a href="data_publication.html#licenses"><i class="fa fa-check"></i><b>10.1</b> Licenses</a></li>
<li class="chapter" data-level="10.2" data-path="ipt.html"><a href="ipt.html"><i class="fa fa-check"></i><b>10.2</b> IPT: Integrated Publishing Toolkit</a></li>
<li class="chapter" data-level="10.3" data-path="ipt_admin.html"><a href="ipt_admin.html"><i class="fa fa-check"></i><b>10.3</b> IPT Admin Responsibilities</a></li>
<li class="chapter" data-level="10.4" data-path="data_sharing.html"><a href="data_sharing.html"><i class="fa fa-check"></i><b>10.4</b> Maintaining and sharing published data</a>
<ul>
<li class="chapter" data-level="10.4.1" data-path="data_sharing.html"><a href="data_sharing.html#adding-a-doi-to-datasets"><i class="fa fa-check"></i><b>10.4.1</b> Adding a DOI to datasets</a></li>
<li class="chapter" data-level="10.4.2" data-path="data_sharing.html"><a href="data_sharing.html#user-tracking"><i class="fa fa-check"></i><b>10.4.2</b> User tracking</a></li>
</ul></li>
<li class="chapter" data-level="10.5" data-path="data_sharing.html"><a href="data_sharing.html#update-your-data-in-obis"><i class="fa fa-check"></i><b>10.5</b> Update your data in OBIS</a></li>
<li class="chapter" data-level="10.6" data-path="data_sharing.html"><a href="data_sharing.html#publish-obis-data-to-gbif"><i class="fa fa-check"></i><b>10.6</b> Publish OBIS data to GBIF</a></li>
</ul></li>
<li class="part"><span><b>Access Data from OBIS</b></span></li>
<li class="chapter" data-level="11" data-path="access.html"><a href="access.html"><i class="fa fa-check"></i><b>11</b> Data access</a>
<ul>
<li class="chapter" data-level="11.1" data-path="access.html"><a href="access.html#obis-homepage-and-dataset-pages"><i class="fa fa-check"></i><b>11.1</b> OBIS Homepage and dataset pages</a></li>
<li class="chapter" data-level="11.2" data-path="access.html"><a href="access.html#mapper"><i class="fa fa-check"></i><b>11.2</b> Mapper</a></li>
<li class="chapter" data-level="11.3" data-path="access.html"><a href="access.html#r-package"><i class="fa fa-check"></i><b>11.3</b> R package</a></li>
<li class="chapter" data-level="11.4" data-path="access.html"><a href="access.html#api"><i class="fa fa-check"></i><b>11.4</b> API</a></li>
<li class="chapter" data-level="11.5" data-path="access.html"><a href="access.html#full-exports"><i class="fa fa-check"></i><b>11.5</b> Full exports</a></li>
<li class="chapter" data-level="11.6" data-path="access.html"><a href="access.html#finding-your-own-data-in-obis"><i class="fa fa-check"></i><b>11.6</b> Finding your own data in OBIS</a></li>
<li class="chapter" data-level="11.7" data-path="access.html"><a href="access.html#how-to-contact-data-provider"><i class="fa fa-check"></i><b>11.7</b> How to contact data provider</a></li>
<li class="chapter" data-level="11.8" data-path="access.html"><a href="access.html#interpreting-downloaded-files-from-obis"><i class="fa fa-check"></i><b>11.8</b> Interpreting downloaded files from OBIS</a></li>
<li class="chapter" data-level="11.9" data-path="citing.html"><a href="citing.html"><i class="fa fa-check"></i><b>11.9</b> Citing Data from OBIS</a></li>
</ul></li>
<li class="part"><span><b>Data Visualization and Analysis</b></span></li>
<li class="chapter" data-level="12" data-path="dataviz.html"><a href="dataviz.html"><i class="fa fa-check"></i><b>12</b> Data Visualization</a>
<ul>
<li class="chapter" data-level="12.1" data-path="dataviz.html"><a href="dataviz.html#example-notebooks-using-data-from-obis"><i class="fa fa-check"></i><b>12.1</b> Example notebooks using data from OBIS</a></li>
<li class="chapter" data-level="12.2" data-path="dataviz.html"><a href="dataviz.html#obisindicators-calculating-visualizing-spatial-biodiversity-using-data-from-obis"><i class="fa fa-check"></i><b>12.2</b> obisindicators: calculating &amp; visualizing spatial biodiversity using data from OBIS</a></li>
</ul></li>
<li class="part"><span><b>Additional Resources</b></span></li>
<li class="chapter" data-level="13" data-path="other_resources.html"><a href="other_resources.html"><i class="fa fa-check"></i><b>13</b> Other Resources</a>
<ul>
<li class="chapter" data-level="13.1" data-path="other_resources.html"><a href="other_resources.html#mbon-pole-to-pole-tutorial"><i class="fa fa-check"></i><b>13.1</b> MBON Pole to Pole Tutorial</a></li>
<li class="chapter" data-level="13.2" data-path="other_resources.html"><a href="other_resources.html#ioos-darwin-core-guide"><i class="fa fa-check"></i><b>13.2</b> IOOS Darwin Core Guide</a></li>
<li class="chapter" data-level="13.3" data-path="other_resources.html"><a href="other_resources.html#emodnet-biology"><i class="fa fa-check"></i><b>13.3</b> EMODnet Biology</a></li>
<li class="chapter" data-level="13.4" data-path="other_resources.html"><a href="other_resources.html#template-generators"><i class="fa fa-check"></i><b>13.4</b> Template Generators</a></li>
<li class="chapter" data-level="13.5" data-path="other_resources.html"><a href="other_resources.html#bionomia"><i class="fa fa-check"></i><b>13.5</b> Bionomia</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The OBIS manual</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="name-matching-for-taxonomic-quality-control" class="section level2 hasAnchor" number="5.2">
<h2><span class="header-section-number">5.2</span> Name matching for taxonomic quality control<a href="name_matching.html#name-matching-for-taxonomic-quality-control" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>OBIS requires all your specimens to be classified and matched against an authoritative taxonomic register. This effectively attaches unique stable identifiers (and digitally traceable) to each of your species. Meaning, if a taxonomic ranking or a species name changes in the future, there will be no question as to which species your dataset is actually referring to. Matching to registers also helps to avoid misspelled or unused terms.</p>
<p>OBIS currently accepts identifiers from <strong>three</strong> authoritative lists:</p>
<ul>
<li><a href="https://www.marinespecies.org/">World Register Marine Species (WoRMS)</a> LSIDs</li>
<li><a href="https://www.itis.gov/">Integrated Taxonomic Information System (ITIS)</a> TSNs</li>
<li><a href="https://biodiversitygenomics.net/projects/bold/">Barcode of Life Data Systems (BOLD)</a> and <a href="https://www.ncbi.nlm.nih.gov/">NCBI</a> identifiers</li>
</ul>
<p>The identifiers (LSID, TSN, ID) from these registers will be used to populate the <code>scientificNameID</code> field. OBIS can accept other LSIDS besides WoRMS, as long as they are mapped in WoRMS. If you would like to include multiple identifiers, please use a concatenated list where each register is clearly identified (e.g. <a href="urn:lsid:itis.gov:itis_tsn:12345" class="uri">urn:lsid:itis.gov:itis_tsn:12345</a>, NCBI:12345, BOLD:12345).</p>
<blockquote>
<p><strong>Note</strong>
You should prioritize using LSIDs because they are unique identifiers that indicate the authority the ID comes from.WoRMS LSIDs are also the taxonomic backbone that OBIS relies on, as it is built on marine systems and is linked to the other taxonomic authoritative lists.</p>
</blockquote>
<p>You can also use the <a href="https://www.irmng.org/aphia.php?p=search">Interim Register of Marine and Nonmarine Genera (IRMNG)</a> to <a href="common_qc.html#non-marine-species">distinguish marine genera from freshwater genera</a>.</p>
<div id="taxon-matching-workflow" class="section level3 unlisted unnumbered hasAnchor">
<h3>Taxon Matching Workflow<a href="name_matching.html#taxon-matching-workflow" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The OBIS node managers have agreed to match all the scientific names in their datasets according to the following Name Matching workflow:</p>
<div class="float">
<img src="images/WoRMS-taxaMatch-workflow.png" style="width:70.0%" alt="Workflow for matching a list of taxon names to WoRMS" />
<div class="figcaption">Workflow for matching a list of taxon names to WoRMS</div>
</div>
<div id="step-1-match-with-worms" class="section level4 hasAnchor" number="5.2.0.1">
<h4><span class="header-section-number">5.2.0.1</span> Step 1: Match with WoRMS<a href="name_matching.html#step-1-match-with-worms" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The procedure for matching to WoRMS and then attaching successful matches back to your data can be simplified to:</p>
<ul>
<li>Prepare a file (.csv, .txt, .xlsx, etc.) with the list of your specimens/taxa</li>
<li>Upload the file to WoRMS taxon match tool
<ul>
<li>Check relevant boxes</li>
</ul></li>
<li>Review returned file</li>
<li>Resolve any ambiguous matches</li>
<li>Download file and identify data to include in your Occurrence data table for OBIS
<ul>
<li>LSIDs, taxonomic fields, etc.</li>
</ul></li>
<li>Attach LSIDs back to your data using e.g.:
<ul>
<li>R (merge)</li>
<li>Excel (vlookup)</li>
</ul></li>
</ul>
<p>The taxon match tool of the World Register of Marine Species (WoRMS) is an automatic way to download the taxonomic information about your occurrence records, without having to look for each name in the site. It is available at <a href="http://www.marinespecies.org/aphia.php?p=match">http://www.marinespecies.org/aphia.php?p=match</a>. The WoRMS taxon match will compare your taxon list to the taxa available in WoRMS. The following video demonstrates the basic steps for using the WoRMS Taxon match.</p>
<p><iframe width="560" height="315"
src="https://www.youtube.com/embed/yZKwtr14JVM"
frameborder="0"
allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
allowfullscreen></iframe></p>
<p>The taxon match takes into account exact matches and fuzzy matches. Fuzzy matches include possible spelling variations of a name available in WoRMS. WoRMS also identifies ambiguous matches, indicating that several potential matching options are available (e.g. homonyms). You can check these ambiguous matches and select the correct one, based on e.g., the general group information (a sponge dataset) or the authority. If this would be impossible with the available information (e.g., missing authority or very diverse dataset), then you need to contact the data provider for clarification. Watch the video below for a demonstration on how to resolve ambiguous or fuzzy matches.</p>
<p><iframe width="560" height="315"
src="https://www.youtube.com/embed/yZKwtr14JVM"
frameborder="0"
allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
allowfullscreen></iframe></p>
<p>For performance reasons, the limit is set to 1,500 rows for the taxon match tool. Larger files can be sent to <a href="mailto:info@marinespecies.org">info@marinespecies.org</a> and will be returned as quickly as possible. In case you have recorded a taxon that is not registered in WoRMS (e.g., newly discovered species), you should contact them so the database can be updated.</p>
<p>After matching, the tool will return you a file with the AphiaIDs, LSIDs, valid names, authorities, classification, and any other output you have selected.</p>
<blockquote>
<p><strong>Note</strong>
The WoRMS LSID is used for <code>DwC:scientificNameID</code>.</p>
</blockquote>
<p>A complete online manual is available at <a href="http://www.marinespecies.org/tutorial/taxonmatch.php">http://www.marinespecies.org/tutorial/taxonmatch.php</a>. You can attach IDs obtained from WoRMS back to your own data using <a href="https://support.microsoft.com/en-us/office/vlookup-function-0bbc8083-26fe-4963-8ab8-93a18ad188a1">Excel’s vlookup function</a>. R script to do this is shown below.</p>
<p><strong>R script for attaching Taxon Lists to ID Lists:</strong></p>
<p>If you are familiar enough with R, you can use the <a href="https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/merge"><code>merge</code></a> function to attach the two lists to your data. We provide a short example of how to use this function below.</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="name_matching.html#cb10-1" tabindex="-1"></a><span class="co">#Generate example data table with species occurences, for this example we will only have one column with the scientificName</span></span>
<span id="cb10-2"><a href="name_matching.html#cb10-2" tabindex="-1"></a>data<span class="ot">&lt;-</span><span class="fu">data.frame</span>(<span class="at">scientificName=</span><span class="fu">c</span>(<span class="st">&quot;Thunnus thynnus&quot;</span>, <span class="st">&quot;Rhincodon typus&quot;</span>, <span class="st">&quot;Luidia maculata&quot;</span>,<span class="st">&quot;Ginglymostoma cirratum&quot;</span>))</span>
<span id="cb10-3"><a href="name_matching.html#cb10-3" tabindex="-1"></a></span>
<span id="cb10-4"><a href="name_matching.html#cb10-4" tabindex="-1"></a><span class="co"># this would be your matched file from WoRMS, but for example we are generating a simple list with the species names and LSIDs</span></span>
<span id="cb10-5"><a href="name_matching.html#cb10-5" tabindex="-1"></a>lsids<span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">scientificName=</span><span class="fu">c</span>(<span class="st">&quot;Ginglymostoma cirratum&quot;</span>,<span class="st">&quot;Luidia maculata&quot;</span>,<span class="st">&quot;Thunnus thynnus&quot;</span>, <span class="st">&quot;Rhincodon typus&quot;</span>),</span>
<span id="cb10-6"><a href="name_matching.html#cb10-6" tabindex="-1"></a> <span class="at">LSID =</span> <span class="fu">c</span>(<span class="st">&quot;urn:lsid:marinespecies.org:taxname:105846&quot;</span>, <span class="st">&quot;urn:lsid:marinespecies.org:taxname:213112&quot;</span>,<span class="st">&quot;urn:lsid:marinespecies.org:taxname:127029&quot;</span>,<span class="st">&quot;urn:lsid:marinespecies.org:taxname:105847&quot;</span>))</span>
<span id="cb10-7"><a href="name_matching.html#cb10-7" tabindex="-1"></a></span>
<span id="cb10-8"><a href="name_matching.html#cb10-8" tabindex="-1"></a><span class="co">#merge data frames together</span></span>
<span id="cb10-9"><a href="name_matching.html#cb10-9" tabindex="-1"></a>matched_data<span class="ot">&lt;-</span><span class="fu">merge</span>(data, lsids, <span class="at">by =</span> <span class="st">&quot;scientificName&quot;</span>)</span>
<span id="cb10-10"><a href="name_matching.html#cb10-10" tabindex="-1"></a>matched_data</span></code></pre></div>
</div>
<div id="how-to-fetch-a-full-classification-for-a-list-of-species-from-worms" class="section level4 hasAnchor" number="5.2.0.2">
<h4><span class="header-section-number">5.2.0.2</span> How to fetch a full classification for a list of species from WoRMS?<a href="name_matching.html#how-to-fetch-a-full-classification-for-a-list-of-species-from-worms" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>When setting up your WoRMS taxon match, to obtain the full classification for your list of species, simply check the box labeled “Classification”. This will add classification output in addition to the requested identifiers to your taxon match file, including Kingdom, Phylum, Class, Order, Family, Genus, Subgenus, Species, and Subspecies.</p>
<div class="float">
<img src="images/WoRMS_classification.png" style="width:70.0%" alt="WoRMS classification box" />
<div class="figcaption">WoRMS classification box</div>
</div>
</div>
<div id="what-to-do-with-non-matching-names" class="section level4 hasAnchor" number="5.2.0.3">
<h4><span class="header-section-number">5.2.0.3</span> What to do with non-matching names?<a href="name_matching.html#what-to-do-with-non-matching-names" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>If your scientificName does not find an exact match to the WoRMS database, you may get an <strong>ambiguous</strong> match. According to WoRMS guidelines, ambiguous matches can be marked as one of the following:</p>
<ul>
<li>phonetic</li>
<li>near_1</li>
<li>near_2</li>
<li>near_3</li>
<li>match_quarantine</li>
<li>match_deleted</li>
</ul>
<p>See <a href="https://www.marinespecies.org/tutorial_taxonmatch.php" class="uri">https://www.marinespecies.org/tutorial_taxonmatch.php</a> for definitions of each of these terms.</p>
<p>In each of these cases, WoRMS will try to suggest a species to match your uncertain taxon. Take care to ensure the correct species name is selected given the information in the original documentation. This is especially true for near_2 or near_3 matches. When checking a potential matched name, we recommend referencing the authority and higher taxonomic levels of a given suggestion when possible. For example, if you know the ambiguous species is a sponge, but one of the suggestions is for a mammal, you know that is not the correct name.</p>
<p>Sometimes the species name as is it was originally documented is a name that is not accepted in WoRMS (i.e. Status = <span style="color: red;">unaccepted</span>). In these cases <code>scientificNameID</code> should reflect the name in the <code>scientificName</code> field, even if this name is <em>not</em> accepted. If you or the data provider can confirm with certainty that the name in <code>scientificName</code> can be changed to an accepted name, then <code>verbatimIdentification</code> <em>must</em> be populated with the scientific name as it was originally documented. In general, it is good practice to populate <code>verbatimIdentification</code> with the original taxon identification.</p>
<div class="float">
<img src="images/WoRMS-resolve-ambiguousMatches.png" style="width:70.0%" alt="Example of choices from an abiguous match" />
<div class="figcaption">Example of choices from an abiguous match</div>
</div>
<p>In cases where no match can be found, WoRMS will indicate none. For these cases you should follow these steps:</p>
<ul>
<li>Ensure the name was entered correctly and any other information (e.g., authority, year, identification qualifiers) are included in separate columns, not the same cell as the name, as this can sometimes prevent matching</li>
<li>Match with <a href="https://www.lifewatch.be/data-services/">LifeWatch</a> or another register (see Step 2 below)</li>
<li>Check that the species <a href="common_qc.html#non-marine-species">is marine</a></li>
</ul>
<p>If a scientific name does not appear in any register, you should contact the original data provider, where possible, to confirm taxonomic spelling, authority, and obtain any original description documents, then attempt to match again. If even after this there are no matches, please contact the WoRMS data management team at <a href="mailto:info@marinespecies.org" class="email">info@marinespecies.org</a> to see if the taxon should be added to the WoRMS register.</p>
</div>
<div id="step-2-check-name-in-other-registers" class="section level4 hasAnchor" number="5.2.0.4">
<h4><span class="header-section-number">5.2.0.4</span> Step 2: Check name in other registers<a href="name_matching.html#step-2-check-name-in-other-registers" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>If you do not find a match with WoRMS, you should next check other registers. The <a href="http://www.lifewatch.be/data-services/">LifeWatch taxon match</a> compares your taxon list to multiple taxonomic standards. Matching with multiple registers gives an indication of the correct spelling of a name, regardless of its environment. If a name would not appear in any of the registers, this could indicate a mistake in the scientific name and the name should go back to the provider for additional checking/verification.</p>
<p>Contrary to the WoRMS taxon match, when several matching options are available, the LifeWatch taxon match only mentions “no exact match found, multiple possibilities” instead of listing the available options. If multiple options are available, these should be looked up and matched manually.</p>
<p>Currently, this web service matches the scientific names with the following taxonomic registers:</p>
<ul>
<li>World Register of Marine Species – WoRMS</li>
<li>Catalogue of Life – CoL</li>
<li>Integrated Taxonomic Information System – ITIS</li>
<li>Pan-European Species-directories Infrastructure – PESI</li>
<li>Index Fungorum – IF</li>
<li>International Plant Names Index – IPNI</li>
<li>Global Names Index - GNI</li>
<li>Paleobiology Database - PaleoDB</li>
</ul>
</div>
<div id="step-3-is-taxon-marine" class="section level4 hasAnchor" number="5.2.0.5">
<h4><span class="header-section-number">5.2.0.5</span> Step 3: Is taxon marine?<a href="name_matching.html#step-3-is-taxon-marine" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The Interim Register of Marine and Non-marine Genera (IRMNG) matching services are available through <a href="http://www.irmng.org/">http://www.irmng.org/</a>, as well as through the <a href="http://www.lifewatch.be/data-services/">LifeWatch taxon match</a>. This service allows you to search for a genus (or other taxonomic rank when you uncheck the “genera” box) to check if it is known to be marine, brackish, freshwater, or terrestrial. You can find this information in the row labeled “Environment”. If the taxa is marine, you may have to contact the WoRMS data management team (<a href="mailto:info@marinespecies.org" class="email">info@marinespecies.org</a>) to have the taxon added to the WoRMS register (note you may have to provide supporting information confirming taxonomic and marine status).</p>
</div>
</div>
<div id="r-packages-for-taxon-matching" class="section level3 unlisted unnumbered hasAnchor">
<h3>R packages for taxon matching<a href="name_matching.html#r-packages-for-taxon-matching" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>If you are familiar with R, you may use the <a href="https://github.com/iobis/obistools#taxon-matching">obistools</a> function <code>match_taxa</code> to conduct taxon matching for your dataset. There is also a WoRMS package called <a href="https://cran.r-project.org/web/packages/worrms/index.html">worrms</a> that has a function called <code>wm_records_taxamatch</code> you can use to conduct taxon matching.</p>
<p>The output will be the same as that from the WoRMS tool, so you should check ambiguous matches as described above, confirming with other registers as necessary.</p>
</div>
<div id="taxon-match-tools-overview" class="section level3 unlisted unnumbered hasAnchor">
<h3>Taxon Match Tools Overview<a href="name_matching.html#taxon-match-tools-overview" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>See the table below for a summary of the different tools available.</p>
<table>
<colgroup>
<col width="22%" />
<col width="42%" />
<col width="34%" />
</colgroup>
<thead>
<tr class="header">
<th>Tool</th>
<th>Advantage</th>
<th>Disadvantage</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>WoRMS taxon match</td>
<td>Accessible online, Does not require coding knowledge</td>
<td>Requires rematch information back to your data</td>
</tr>
<tr class="even">
<td>obistools::match_taxa</td>
<td>Produces same output as WoRMS taxon match, Already in R so easier to merge back with data</td>
<td>Requires knowledge of R or python</td>
</tr>
<tr class="odd">
<td>worrms::wm_records_taxamatch</td>
<td>Outputs all WoRMS matching information</td>
<td>Outputs a tibble for each taxa name specified; Requires knowledge of R or python</td>
</tr>
</tbody>
</table>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="checklist.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="format_occurrence.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["manual.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "none"
}
});
});
</script>

</body>

</html>
